{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "1945fb7a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: memote in c:\\users\\pauli\\anaconda3\\lib\\site-packages (0.14.0)\n",
      "Requirement already satisfied: click in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (8.0.4)\n",
      "Requirement already satisfied: click-configfile in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (0.2.3)\n",
      "Requirement already satisfied: click-log in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (0.4.0)\n",
      "Requirement already satisfied: six in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.16.0)\n",
      "Requirement already satisfied: future in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (0.18.3)\n",
      "Requirement already satisfied: pytest>=4.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (7.4.0)\n",
      "Requirement already satisfied: gitpython in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (3.1.40)\n",
      "Requirement already satisfied: pandas in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.5.3)\n",
      "Requirement already satisfied: Jinja2 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (3.1.2)\n",
      "Requirement already satisfied: cookiecutter in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.7.3)\n",
      "Requirement already satisfied: cobra>=0.17 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (0.27.0)\n",
      "Requirement already satisfied: ruamel.yaml>=0.15 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (0.17.21)\n",
      "Requirement already satisfied: travis-encrypt<1.2.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.1.2)\n",
      "Requirement already satisfied: sympy in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.12)\n",
      "Requirement already satisfied: sqlalchemy in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.4.39)\n",
      "Requirement already satisfied: requests in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (2.31.0)\n",
      "Requirement already satisfied: numpydoc in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.5.0)\n",
      "Requirement already satisfied: pylru in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.2.1)\n",
      "Requirement already satisfied: goodtables~=2.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (2.5.4)\n",
      "Requirement already satisfied: depinfo in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from memote) (1.7.0)\n",
      "Requirement already satisfied: appdirs~=1.4 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (1.4.4)\n",
      "Requirement already satisfied: diskcache~=5.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (5.6.3)\n",
      "Requirement already satisfied: httpx~=0.24 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (0.25.0)\n",
      "Requirement already satisfied: importlib-resources in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (6.0.1)\n",
      "Requirement already satisfied: numpy>=1.13 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (1.24.3)\n",
      "Requirement already satisfied: optlang~=1.5 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (1.7.0)\n",
      "Requirement already satisfied: pydantic>=1.6 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (2.3.0)\n",
      "Requirement already satisfied: python-libsbml~=5.19 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (5.20.1)\n",
      "Requirement already satisfied: rich>=8.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (13.5.3)\n",
      "Requirement already satisfied: swiglpk in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cobra>=0.17->memote) (5.0.8)\n",
      "Requirement already satisfied: click-default-group in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from goodtables~=2.0->memote) (1.2.4)\n",
      "Requirement already satisfied: simpleeval>=0.9 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from goodtables~=2.0->memote) (0.9.13)\n",
      "Requirement already satisfied: statistics>=1.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from goodtables~=2.0->memote) (1.0.3.5)\n",
      "Requirement already satisfied: tabulator>=1.40 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from goodtables~=2.0->memote) (1.53.5)\n",
      "Requirement already satisfied: tableschema>=1.16.4 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from goodtables~=2.0->memote) (1.20.2)\n",
      "Requirement already satisfied: datapackage>=1.10 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from goodtables~=2.0->memote) (1.15.2)\n",
      "Requirement already satisfied: colorama in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from click->memote) (0.4.6)\n",
      "Requirement already satisfied: python-dateutil>=2.8.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from pandas->memote) (2.8.2)\n",
      "Requirement already satisfied: pytz>=2020.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from pandas->memote) (2022.7)\n",
      "Requirement already satisfied: iniconfig in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from pytest>=4.0->memote) (1.1.1)\n",
      "Requirement already satisfied: packaging in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from pytest>=4.0->memote) (23.0)\n",
      "Requirement already satisfied: pluggy<2.0,>=0.12 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from pytest>=4.0->memote) (1.0.0)\n",
      "Requirement already satisfied: charset-normalizer<4,>=2 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from requests->memote) (2.0.4)\n",
      "Requirement already satisfied: idna<4,>=2.5 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from requests->memote) (3.4)\n",
      "Requirement already satisfied: urllib3<3,>=1.21.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from requests->memote) (1.26.16)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from requests->memote) (2023.7.22)\n",
      "Requirement already satisfied: cryptography>=2.3 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from travis-encrypt<1.2.0->memote) (41.0.2)\n",
      "Requirement already satisfied: PyYAML>=3.12 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from travis-encrypt<1.2.0->memote) (6.0)\n",
      "Requirement already satisfied: pyperclip==1.6.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from travis-encrypt<1.2.0->memote) (1.6.0)\n",
      "Requirement already satisfied: configparser>=3.5.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from click-configfile->memote) (6.0.0)\n",
      "Requirement already satisfied: binaryornot>=0.4.4 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cookiecutter->memote) (0.4.4)\n",
      "Requirement already satisfied: poyo>=0.5.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cookiecutter->memote) (0.5.0)\n",
      "Requirement already satisfied: jinja2-time>=0.2.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cookiecutter->memote) (0.2.0)\n",
      "Requirement already satisfied: python-slugify>=4.0.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cookiecutter->memote) (5.0.2)\n",
      "Requirement already satisfied: MarkupSafe>=2.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from Jinja2->memote) (2.1.1)\n",
      "Requirement already satisfied: gitdb<5,>=4.0.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from gitpython->memote) (4.0.11)\n",
      "Requirement already satisfied: sphinx>=4.2 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from numpydoc->memote) (5.0.2)\n",
      "Requirement already satisfied: greenlet!=0.4.17 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sqlalchemy->memote) (2.0.1)\n",
      "Requirement already satisfied: mpmath>=0.19 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sympy->memote) (1.3.0)\n",
      "Requirement already satisfied: chardet>=3.0.2 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from binaryornot>=0.4.4->cookiecutter->memote) (4.0.0)\n",
      "Requirement already satisfied: cffi>=1.12 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cryptography>=2.3->travis-encrypt<1.2.0->memote) (1.15.1)\n",
      "Requirement already satisfied: jsonschema>=2.5 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from datapackage>=1.10->goodtables~=2.0->memote) (4.17.3)\n",
      "Requirement already satisfied: unicodecsv>=0.14 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from datapackage>=1.10->goodtables~=2.0->memote) (0.14.1)\n",
      "Requirement already satisfied: jsonpointer>=1.10 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from datapackage>=1.10->goodtables~=2.0->memote) (2.1)\n",
      "Requirement already satisfied: smmap<6,>=3.0.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from gitdb<5,>=4.0.1->gitpython->memote) (5.0.1)\n",
      "Requirement already satisfied: httpcore<0.19.0,>=0.18.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from httpx~=0.24->cobra>=0.17->memote) (0.18.0)\n",
      "Requirement already satisfied: sniffio in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from httpx~=0.24->cobra>=0.17->memote) (1.2.0)\n",
      "Requirement already satisfied: arrow in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from jinja2-time>=0.2.0->cookiecutter->memote) (1.2.3)\n",
      "Requirement already satisfied: annotated-types>=0.4.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from pydantic>=1.6->cobra>=0.17->memote) (0.5.0)\n",
      "Requirement already satisfied: pydantic-core==2.6.3 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from pydantic>=1.6->cobra>=0.17->memote) (2.6.3)\n",
      "Requirement already satisfied: typing-extensions>=4.6.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from pydantic>=1.6->cobra>=0.17->memote) (4.7.1)\n",
      "Requirement already satisfied: text-unidecode>=1.3 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from python-slugify>=4.0.0->cookiecutter->memote) (1.3)\n",
      "Requirement already satisfied: markdown-it-py>=2.2.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from rich>=8.0->cobra>=0.17->memote) (2.2.0)\n",
      "Requirement already satisfied: pygments<3.0.0,>=2.13.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from rich>=8.0->cobra>=0.17->memote) (2.15.1)\n",
      "Requirement already satisfied: sphinxcontrib-applehelp in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (1.0.2)\n",
      "Requirement already satisfied: sphinxcontrib-devhelp in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (1.0.2)\n",
      "Requirement already satisfied: sphinxcontrib-jsmath in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (1.0.1)\n",
      "Requirement already satisfied: sphinxcontrib-htmlhelp>=2.0.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (2.0.0)\n",
      "Requirement already satisfied: sphinxcontrib-serializinghtml>=1.1.5 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (1.1.5)\n",
      "Requirement already satisfied: sphinxcontrib-qthelp in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (1.0.3)\n",
      "Requirement already satisfied: docutils<0.19,>=0.14 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (0.18.1)\n",
      "Requirement already satisfied: snowballstemmer>=1.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (2.2.0)\n",
      "Requirement already satisfied: babel>=1.3 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (2.11.0)\n",
      "Requirement already satisfied: alabaster<0.8,>=0.7 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (0.7.12)\n",
      "Requirement already satisfied: imagesize in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from sphinx>=4.2->numpydoc->memote) (1.4.1)\n",
      "Requirement already satisfied: cached-property>=1.5 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tableschema>=1.16.4->goodtables~=2.0->memote) (1.5.2)\n",
      "Requirement already satisfied: isodate>=0.5.4 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tableschema>=1.16.4->goodtables~=2.0->memote) (0.6.1)\n",
      "Requirement already satisfied: rfc3986>=1.1.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tableschema>=1.16.4->goodtables~=2.0->memote) (2.0.0)\n",
      "Requirement already satisfied: boto3>=1.9 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tabulator>=1.40->goodtables~=2.0->memote) (1.24.28)\n",
      "Requirement already satisfied: ijson>=3.0.3 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tabulator>=1.40->goodtables~=2.0->memote) (3.2.3)\n",
      "Requirement already satisfied: jsonlines>=1.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tabulator>=1.40->goodtables~=2.0->memote) (4.0.0)\n",
      "Requirement already satisfied: linear-tsv>=1.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tabulator>=1.40->goodtables~=2.0->memote) (1.1.0)\n",
      "Requirement already satisfied: xlrd>=1.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tabulator>=1.40->goodtables~=2.0->memote) (2.0.1)\n",
      "Requirement already satisfied: openpyxl>=2.6 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from tabulator>=1.40->goodtables~=2.0->memote) (3.0.10)\n",
      "Requirement already satisfied: botocore<1.28.0,>=1.27.28 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from boto3>=1.9->tabulator>=1.40->goodtables~=2.0->memote) (1.27.59)\n",
      "Requirement already satisfied: jmespath<2.0.0,>=0.7.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from boto3>=1.9->tabulator>=1.40->goodtables~=2.0->memote) (0.10.0)\n",
      "Requirement already satisfied: s3transfer<0.7.0,>=0.6.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from boto3>=1.9->tabulator>=1.40->goodtables~=2.0->memote) (0.6.0)\n",
      "Requirement already satisfied: pycparser in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from cffi>=1.12->cryptography>=2.3->travis-encrypt<1.2.0->memote) (2.21)\n",
      "Requirement already satisfied: anyio<5.0,>=3.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from httpcore<0.19.0,>=0.18.0->httpx~=0.24->cobra>=0.17->memote) (3.5.0)\n",
      "Requirement already satisfied: h11<0.15,>=0.13 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from httpcore<0.19.0,>=0.18.0->httpx~=0.24->cobra>=0.17->memote) (0.14.0)\n",
      "Requirement already satisfied: attrs>=19.2.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from jsonlines>=1.1->tabulator>=1.40->goodtables~=2.0->memote) (22.1.0)\n",
      "Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from jsonschema>=2.5->datapackage>=1.10->goodtables~=2.0->memote) (0.18.0)\n",
      "Requirement already satisfied: mdurl~=0.1 in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from markdown-it-py>=2.2.0->rich>=8.0->cobra>=0.17->memote) (0.1.0)\n",
      "Requirement already satisfied: et_xmlfile in c:\\users\\pauli\\anaconda3\\lib\\site-packages (from openpyxl>=2.6->tabulator>=1.40->goodtables~=2.0->memote) (1.1.0)\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install memote"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "8f260dd8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[1m============================= test session starts =============================\u001b[0m\n",
      "platform win32 -- Python 3.11.4, pytest-7.4.0, pluggy-1.0.0 -- C:\\Users\\pauli\\anaconda3\\python.exe\n",
      "cachedir: .pytest_cache\n",
      "rootdir: C:\\Users\\pauli\n",
      "plugins: anyio-3.5.0\n",
      "\u001b[1mcollecting ... \u001b[0mcollected 146 items / 1 skipped\n",
      "\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_presence \u001b[31mFAILED\u001b[0m\u001b[31m [  0%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_presence \u001b[31mFAILED\u001b[0m\u001b[31m [  1%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_presence \u001b[31mFAILED\u001b[0m\u001b[31m [  2%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[pubchem.compound] \u001b[31mFAILED\u001b[0m\u001b[31m [  2%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[kegg.compound] \u001b[31mFAILED\u001b[0m\u001b[31m [  3%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[seed.compound] \u001b[31mFAILED\u001b[0m\u001b[31m [  4%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[inchikey] \u001b[31mFAILED\u001b[0m\u001b[31m [  4%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[inchi] \u001b[31mFAILED\u001b[0m\u001b[31m [  5%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[chebi] \u001b[31mFAILED\u001b[0m\u001b[31m [  6%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[hmdb] \u001b[31mFAILED\u001b[0m\u001b[31m [  6%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[reactome] \u001b[31mFAILED\u001b[0m\u001b[31m [  7%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[metanetx.chemical] \u001b[31mFAILED\u001b[0m\u001b[31m [  8%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[bigg.metabolite] \u001b[31mFAILED\u001b[0m\u001b[31m [  8%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_overview[biocyc] \u001b[31mFAILED\u001b[0m\u001b[31m [  9%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[rhea] \u001b[31mFAILED\u001b[0m\u001b[31m [ 10%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[kegg.reaction] \u001b[31mFAILED\u001b[0m\u001b[31m [ 10%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[seed.reaction] \u001b[31mFAILED\u001b[0m\u001b[31m [ 11%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[metanetx.reaction] \u001b[31mFAILED\u001b[0m\u001b[31m [ 12%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[bigg.reaction] \u001b[31mFAILED\u001b[0m\u001b[31m [ 13%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[reactome] \u001b[31mFAILED\u001b[0m\u001b[31m [ 13%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[ec-code] \u001b[31mFAILED\u001b[0m\u001b[31m [ 14%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[brenda] \u001b[31mFAILED\u001b[0m\u001b[31m [ 15%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_overview[biocyc] \u001b[31mFAILED\u001b[0m\u001b[31m [ 15%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[refseq] \u001b[31mFAILED\u001b[0m\u001b[31m [ 16%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[uniprot] \u001b[31mFAILED\u001b[0m\u001b[31m [ 17%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[ecogene] \u001b[31mFAILED\u001b[0m\u001b[31m [ 17%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[kegg.genes] \u001b[31mFAILED\u001b[0m\u001b[31m [ 18%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[ncbigi] \u001b[31mFAILED\u001b[0m\u001b[31m [ 19%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[ncbigene] \u001b[31mFAILED\u001b[0m\u001b[31m [ 19%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[ncbiprotein] \u001b[31mFAILED\u001b[0m\u001b[31m [ 20%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[ccds] \u001b[31mFAILED\u001b[0m\u001b[31m [ 21%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[hprd] \u001b[31mFAILED\u001b[0m\u001b[31m [ 21%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_overview[asap] \u001b[31mFAILED\u001b[0m\u001b[31m [ 22%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[pubchem.compound] \u001b[31mFAILED\u001b[0m\u001b[31m [ 23%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[kegg.compound] \u001b[31mFAILED\u001b[0m\u001b[31m [ 23%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[seed.compound] \u001b[31mFAILED\u001b[0m\u001b[31m [ 24%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[inchikey] \u001b[31mFAILED\u001b[0m\u001b[31m [ 25%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[inchi] \u001b[31mFAILED\u001b[0m\u001b[31m [ 26%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[chebi] \u001b[31mFAILED\u001b[0m\u001b[31m [ 26%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[hmdb] \u001b[31mFAILED\u001b[0m\u001b[31m [ 27%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[reactome] \u001b[31mFAILED\u001b[0m\u001b[31m [ 28%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[metanetx.chemical] \u001b[31mFAILED\u001b[0m\u001b[31m [ 28%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[bigg.metabolite] \u001b[31mFAILED\u001b[0m\u001b[31m [ 29%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_annotation_wrong_ids[biocyc] \u001b[31mFAILED\u001b[0m\u001b[31m [ 30%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[rhea] \u001b[31mFAILED\u001b[0m\u001b[31m [ 30%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[kegg.reaction] \u001b[31mFAILED\u001b[0m\u001b[31m [ 31%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[seed.reaction] \u001b[31mFAILED\u001b[0m\u001b[31m [ 32%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[metanetx.reaction] \u001b[31mFAILED\u001b[0m\u001b[31m [ 32%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[bigg.reaction] \u001b[31mFAILED\u001b[0m\u001b[31m [ 33%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[reactome] \u001b[31mFAILED\u001b[0m\u001b[31m [ 34%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[ec-code] \u001b[31mFAILED\u001b[0m\u001b[31m [ 34%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[brenda] \u001b[31mFAILED\u001b[0m\u001b[31m [ 35%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_annotation_wrong_ids[biocyc] \u001b[31mFAILED\u001b[0m\u001b[31m [ 36%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[refseq] \u001b[31mFAILED\u001b[0m\u001b[31m [ 36%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[uniprot] \u001b[31mFAILED\u001b[0m\u001b[31m [ 37%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[ecogene] \u001b[31mFAILED\u001b[0m\u001b[31m [ 38%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[kegg.genes] \u001b[31mFAILED\u001b[0m\u001b[31m [ 39%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[ncbigi] \u001b[31mFAILED\u001b[0m\u001b[31m [ 39%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[ncbigene] \u001b[31mFAILED\u001b[0m\u001b[31m [ 40%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[ncbiprotein] \u001b[31mFAILED\u001b[0m\u001b[31m [ 41%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[ccds] \u001b[31mFAILED\u001b[0m\u001b[31m [ 41%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[hprd] \u001b[31mFAILED\u001b[0m\u001b[31m [ 42%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_gene_product_annotation_wrong_ids[asap] \u001b[31mFAILED\u001b[0m\u001b[31m [ 43%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_metabolite_id_namespace_consistency \u001b[32mPASSED\u001b[0m\u001b[31m [ 43%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::test_reaction_id_namespace_consistency \u001b[32mPASSED\u001b[0m\u001b[31m [ 44%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_model_id_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 45%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_genes_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 45%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_reactions_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 46%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_metabolites_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 47%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_metabolites_formula_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 47%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_metabolites_charge_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 48%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_gene_protein_reaction_rule_presence \u001b[31mFAILED\u001b[0m\u001b[31m [ 49%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_ngam_presence \u001b[31mFAILED\u001b[0m\u001b[31m [ 50%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_metabolic_coverage \u001b[32mPASSED\u001b[0m\u001b[31m [ 50%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_compartments_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 51%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_protein_complex_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 52%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_pure_metabolic_reactions \u001b[32mPASSED\u001b[0m\u001b[31m [ 52%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_constrained_pure_metabolic_reactions \u001b[32mPASSED\u001b[0m\u001b[31m [ 53%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_transport_reactions \u001b[32mPASSED\u001b[0m\u001b[31m [ 54%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_constrained_transport_reactions \u001b[32mPASSED\u001b[0m\u001b[31m [ 54%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_transport_reaction_gpr_presence \u001b[31mFAILED\u001b[0m\u001b[31m [ 55%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_reversible_oxygen_reactions \u001b[32mPASSED\u001b[0m\u001b[31m [ 56%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_unique_metabolites \u001b[32mPASSED\u001b[0m\u001b[31m [ 56%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_duplicate_metabolites_in_compartments \u001b[32mPASSED\u001b[0m\u001b[31m [ 57%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_reactions_with_partially_identical_annotations \u001b[32mPASSED\u001b[0m\u001b[31m [ 58%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_duplicate_reactions \u001b[32mPASSED\u001b[0m\u001b[31m [ 58%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_reactions_with_identical_genes \u001b[31mFAILED\u001b[0m\u001b[31m [ 59%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::test_find_medium_metabolites \u001b[31mFAILED\u001b[0m\u001b[31m [ 60%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_biomass_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 60%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_biomass_consistency[biomass_SC4_bal] \u001b[31mFAILED\u001b[0m\u001b[31m [ 61%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_biomass_default_production[biomass_SC4_bal] \u001b[32mPASSED\u001b[0m\u001b[31m [ 62%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_biomass_open_production[biomass_SC4_bal] \u001b[32mPASSED\u001b[0m\u001b[31m [ 63%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_biomass_precursors_default_production[biomass_SC4_bal] \u001b[32mPASSED\u001b[0m\u001b[31m [ 63%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_biomass_precursors_open_production[biomass_SC4_bal] \u001b[32mPASSED\u001b[0m\u001b[31m [ 64%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_gam_in_biomass[biomass_SC4_bal] \u001b[31mFAILED\u001b[0m\u001b[31m [ 65%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_fast_growth_default[biomass_SC4_bal] \u001b[31mFAILED\u001b[0m\u001b[31m [ 65%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_direct_metabolites_in_biomass[biomass_SC4_bal] \u001b[32mPASSED\u001b[0m\u001b[31m [ 66%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::test_essential_precursors_not_in_biomass[biomass_SC4_bal] \u001b[31mFAILED\u001b[0m\u001b[31m [ 67%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_stoichiometric_consistency \u001b[32mPASSED\u001b[0m\u001b[31m [ 67%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_unconserved_metabolites \u001b[32mPASSED\u001b[0m\u001b[31m [ 68%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_inconsistent_min_stoichiometry \u001b[32mPASSED\u001b[0m\u001b[31m [ 69%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM3] \u001b[33mSKIPPED\u001b[0mn the\n",
      "model.)\u001b[31m                                                                  [ 69%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM63] \u001b[33mSKIPPED\u001b[0min the\n",
      "model.)\u001b[31m                                                                  [ 70%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM51] \u001b[33mSKIPPED\u001b[0min the\n",
      "model.)\u001b[31m                                                                  [ 71%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM121] \u001b[33mSKIPPED\u001b[0m in the\n",
      "model.)\u001b[31m                                                                  [ 71%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM423] \u001b[33mSKIPPED\u001b[0m in the\n",
      "model.)\u001b[31m                                                                  [ 72%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM6] \u001b[33mSKIPPED\u001b[0mn the\n",
      "model.)\u001b[31m                                                                  [ 73%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM10] \u001b[33mSKIPPED\u001b[0min the\n",
      "model.)\u001b[31m                                                                  [ 73%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM38] \u001b[33mSKIPPED\u001b[0min the\n",
      "model.)\u001b[31m                                                                  [ 74%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM208] \u001b[33mSKIPPED\u001b[0m in the\n",
      "model.)\u001b[31m                                                                  [ 75%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM191] \u001b[33mSKIPPED\u001b[0m in the\n",
      "model.)\u001b[31m                                                                  [ 76%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM223] \u001b[33mSKIPPED\u001b[0m in the\n",
      "model.)\u001b[31m                                                                  [ 76%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM7517] \u001b[33mSKIPPED\u001b[0md in\n",
      "the model.)\u001b[31m                                                              [ 77%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM12233] \u001b[33mSKIPPED\u001b[0mnd in\n",
      "the model.)\u001b[31m                                                              [ 78%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM558] \u001b[33mSKIPPED\u001b[0m in the\n",
      "model.)\u001b[31m                                                                  [ 78%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM21] \u001b[33mSKIPPED\u001b[0min the\n",
      "model.)\u001b[31m                                                                  [ 79%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_detect_energy_generating_cycles[MNXM89557] \u001b[33mSKIPPED\u001b[0mnd in\n",
      "the model.)\u001b[31m                                                              [ 80%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_reaction_charge_balance \u001b[31mFAILED\u001b[0m\u001b[31m [ 80%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_reaction_mass_balance \u001b[31mFAILED\u001b[0m\u001b[31m [ 81%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_blocked_reactions \u001b[31mFAILED\u001b[0m\u001b[31m [ 82%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_find_stoichiometrically_balanced_cycles \u001b[31mFAILED\u001b[0m\u001b[31m [ 82%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_find_orphans \u001b[31mFAILED\u001b[0m\u001b[31m [ 83%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_find_deadends \u001b[31mFAILED\u001b[0m\u001b[31m [ 84%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_find_disconnected \u001b[32mPASSED\u001b[0m\u001b[31m [ 84%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_find_metabolites_not_produced_with_open_bounds \u001b[31mFAILED\u001b[0m\u001b[31m [ 85%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_find_metabolites_not_consumed_with_open_bounds \u001b[31mFAILED\u001b[0m\u001b[31m [ 86%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::test_find_reactions_unbounded_flux_default_condition \u001b[31mFAILED\u001b[0m\u001b[31m [ 86%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_essentiality.py::test_gene_essentiality_from_data_qualitative[experiment0] \u001b[33mSKIPPED\u001b[0memote\\suite\\tests\\test_essentiality.py:33)\u001b[31m [ 87%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_growth.py::test_growth_from_data_qualitative[experiment0] \u001b[33mSKIPPED\u001b[0m_qualitative\n",
      "at C:\\Users\\pauli\\anaconda3\\Lib\\site-\n",
      "packages\\memote\\suite\\tests\\test_growth.py:33)\u001b[31m                           [ 88%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py::test_absolute_extreme_coefficient_ratio \u001b[32mPASSED\u001b[0m\u001b[31m [ 89%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py::test_number_independent_conservation_relations \u001b[32mPASSED\u001b[0m\u001b[31m [ 89%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py::test_matrix_rank \u001b[32mPASSED\u001b[0m\u001b[31m [ 90%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py::test_degrees_of_freedom \u001b[32mPASSED\u001b[0m\u001b[31m [ 91%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbml.py::test_sbml_level \u001b[32mPASSED\u001b[0m\u001b[31m [ 91%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbml.py::test_fbc_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 92%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_metabolite_sbo_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 93%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_reaction_sbo_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 93%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_gene_sbo_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 94%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_metabolic_reaction_specific_sbo_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 95%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_transport_reaction_specific_sbo_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 95%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_metabolite_specific_sbo_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 96%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_gene_specific_sbo_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 97%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_exchange_specific_sbo_presence \u001b[32mPASSED\u001b[0m\u001b[31m [ 97%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_demand_specific_sbo_presence \u001b[33mSKIPPED\u001b[0m\u001b[31m [ 98%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_sink_specific_sbo_presence \u001b[33mSKIPPED\u001b[0m\u001b[31m [ 99%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::test_biomass_specific_sbo_presence \u001b[31mFAILED\u001b[0m\u001b[31m [100%]\u001b[0m\n",
      "\n",
      "================================== FAILURES ===================================\n",
      "\u001b[31m\u001b[1m_____________________ test_metabolite_annotation_presence _____________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:57: in test_metabolite_annotation_presence\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 1061 metabolites (100.00%) lack any form of annotation:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c, 13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________________ test_reaction_annotation_presence ______________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:87: in test_reaction_annotation_presence\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 1266 reactions (100.00%) lack any form of annotation:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m____________________ test_gene_product_annotation_presence ____________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:116: in test_gene_product_annotation_presence\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 750 genes (100.00%) lack any form of         annotation:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m____________ test_metabolite_annotation_overview[pubchem.compound] ____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     pubchem.compound:         10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c,\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________ test_metabolite_annotation_overview[kegg.compound] ______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     kegg.compound:         10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c,\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________ test_metabolite_annotation_overview[seed.compound] ______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     seed.compound:         10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c,\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_metabolite_annotation_overview[inchikey] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for inchikey:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c, 13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_metabolite_annotation_overview[inchi] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for inchi:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c, 13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_metabolite_annotation_overview[chebi] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for chebi:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c, 13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m__________________ test_metabolite_annotation_overview[hmdb] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for hmdb:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c, 13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_metabolite_annotation_overview[reactome] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for reactome:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c, 13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m___________ test_metabolite_annotation_overview[metanetx.chemical] ____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     metanetx.chemical:         10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c,\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m____________ test_metabolite_annotation_overview[bigg.metabolite] _____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     bigg.metabolite:         10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c,\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_metabolite_annotation_overview[biocyc] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:169: in test_metabolite_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1061 metabolites (100.00%) lack annotation for biocyc:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     10fthf_c, 10fthf_m, 12dgr_SC_c, 13BDglcn_c, 13BDglcn_e, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1061 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1061 = len(['10fthf_c', '10fthf_m', '12dgr_SC_c', '13BDglcn_c', '13BDglcn_e', '13dampp_c', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m___________________ test_reaction_annotation_overview[rhea] ___________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for rhea:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________ test_reaction_annotation_overview[kegg.reaction] _______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     kegg.reaction:         13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________ test_reaction_annotation_overview[seed.reaction] _______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     seed.reaction:         13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m____________ test_reaction_annotation_overview[metanetx.reaction] _____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     metanetx.reaction:         13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________ test_reaction_annotation_overview[bigg.reaction] _______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for\u001b[0m\n",
      "\u001b[1m\u001b[31mE     bigg.reaction:         13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_reaction_annotation_overview[reactome] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for reactome:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_reaction_annotation_overview[ec-code] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for ec-code:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m__________________ test_reaction_annotation_overview[brenda] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for brenda:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m__________________ test_reaction_annotation_overview[biocyc] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:222: in test_reaction_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 1266 reactions (100.00%) lack annotation for biocyc:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     13BGH, 13BGHe, 13GS, 2DDA7Ptm, 2DHPtm, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1266 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1266 = len(['13BGH', '13BGHe', '13GS', '2DDA7Ptm', '2DHPtm', '2MCITtm', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_gene_product_annotation_overview[refseq] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for refseq:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_gene_product_annotation_overview[uniprot] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for uniprot:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_gene_product_annotation_overview[ecogene] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for ecogene:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________ test_gene_product_annotation_overview[kegg.genes] ______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for kegg.genes:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_gene_product_annotation_overview[ncbigi] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for ncbigi:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_gene_product_annotation_overview[ncbigene] _______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for ncbigene:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________ test_gene_product_annotation_overview[ncbiprotein] ______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for ncbiprotein:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_gene_product_annotation_overview[ccds] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for ccds:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_gene_product_annotation_overview[hprd] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for hprd:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_gene_product_annotation_overview[asap] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:275: in test_gene_product_annotation_overview\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The following 750 genes (100.00%) lack annotation for asap:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     Q0045, Q0080, Q0085, Q0105, Q0130, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 750 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 750 = len(['Q0045', 'Q0080', 'Q0085', 'Q0105', 'Q0130', 'Q0250', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m___________ test_metabolite_annotation_wrong_ids[pubchem.compound] ____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the pubchem.compound database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________ test_metabolite_annotation_wrong_ids[kegg.compound] _____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the kegg.compound database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________ test_metabolite_annotation_wrong_ids[seed.compound] _____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the seed.compound database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_metabolite_annotation_wrong_ids[inchikey] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the inchikey database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_metabolite_annotation_wrong_ids[inchi] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the inchi database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_metabolite_annotation_wrong_ids[chebi] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the chebi database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_metabolite_annotation_wrong_ids[hmdb] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the hmdb database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_metabolite_annotation_wrong_ids[reactome] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the reactome database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m___________ test_metabolite_annotation_wrong_ids[metanetx.chemical] ___________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the metanetx.chemical\u001b[0m\n",
      "\u001b[1m\u001b[31mE     database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m____________ test_metabolite_annotation_wrong_ids[bigg.metabolite] ____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the bigg.metabolite database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_metabolite_annotation_wrong_ids[biocyc] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:320: in test_metabolite_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no metabolite annotations for the biocyc database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m__________________ test_reaction_annotation_wrong_ids[rhea] ___________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the rhea database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________ test_reaction_annotation_wrong_ids[kegg.reaction] ______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the kegg.reaction database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________ test_reaction_annotation_wrong_ids[seed.reaction] ______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the seed.reaction database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m____________ test_reaction_annotation_wrong_ids[metanetx.reaction] ____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the metanetx.reaction database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________ test_reaction_annotation_wrong_ids[bigg.reaction] ______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the bigg.reaction database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_reaction_annotation_wrong_ids[reactome] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the reactome database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_reaction_annotation_wrong_ids[ec-code] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the ec-code database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_reaction_annotation_wrong_ids[brenda] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the brenda database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________ test_reaction_annotation_wrong_ids[biocyc] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:379: in test_reaction_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no reaction annotations for the biocyc database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_gene_product_annotation_wrong_ids[refseq] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the refseq database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_gene_product_annotation_wrong_ids[uniprot] _______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the uniprot database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_gene_product_annotation_wrong_ids[ecogene] _______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the ecogene database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________ test_gene_product_annotation_wrong_ids[kegg.genes] ______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the kegg.genes database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_______________ test_gene_product_annotation_wrong_ids[ncbigi] ________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the ncbigi database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________ test_gene_product_annotation_wrong_ids[ncbigene] _______________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the ncbigene database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________ test_gene_product_annotation_wrong_ids[ncbiprotein] _____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the ncbiprotein database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_gene_product_annotation_wrong_ids[ccds] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the ccds database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_gene_product_annotation_wrong_ids[hprd] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the hprd database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________ test_gene_product_annotation_wrong_ids[asap] _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py\u001b[0m:438: in test_gene_product_annotation_wrong_ids\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(total) > \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][db]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are no gene annotations for the asap database.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0 > 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 0 = len([])\u001b[0m\n",
      "\u001b[31m\u001b[1m__________________ test_gene_protein_reaction_rule_presence ___________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py\u001b[0m:209: in test_gene_protein_reaction_rule_presence\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are a total of 340 reactions (26.86%) without GPR:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     POLYAO, AABTN, 4ABUTNtm, FA180ACPtm, FAO240p, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 340 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 340 = len(['POLYAO', 'AABTN', '4ABUTNtm', 'FA180ACPtm', 'FAO240p', 'G16MTg', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________________________ test_ngam_presence ______________________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py\u001b[0m:238: in test_ngam_presence\n",
      "    ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m] = get_ids(basic.find_ngam(model))\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\support\\basic.py\u001b[0m:127: in find_ngam\n",
      "    helpers.find_met_in_model(model, \u001b[33m\"\u001b[39;49;00m\u001b[33mMNXM2\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m, id_of_main_compartment)[\u001b[94m0\u001b[39;49;00m],\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\support\\helpers.py\u001b[0m:919: in find_met_in_model\n",
      "    \u001b[94mraise\u001b[39;49;00m \u001b[96mRuntimeError\u001b[39;49;00m(\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   RuntimeError: It was not possible to uniquely identify a single metabolite in compartment c that corresponds to the following MetaNetX identifier: MNXM2.Instead these candidates were found: ['h2o_c', 'oh1_c'].Check that metabolite compartment tags are correct. Consider switching to a namespace scheme where identifiers are truly unique.\u001b[0m\n",
      "---------------------------- Captured stdout call -----------------------------\n",
      "compartment_id = None?\n",
      "compartment_id = None?\n",
      "\u001b[31m\u001b[1m____________________ test_transport_reaction_gpr_presence _____________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py\u001b[0m:514: in test_transport_reaction_gpr_presence\n",
      "    \u001b[94massert\u001b[39;49;00m ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmetric\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m] < \u001b[94m0.2\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are a total of 212 transport reactions (71.38% of all\u001b[0m\n",
      "\u001b[1m\u001b[31mE     transport reactions) without GPR:         AKGt2n, ACtm, XANt,\u001b[0m\n",
      "\u001b[1m\u001b[31mE     4ABUTNtm, MELIBt2, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0.7138047138047138 < 0.2\u001b[0m\n",
      "\u001b[31m\u001b[1m__________________ test_find_reactions_with_identical_genes ___________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py\u001b[0m:714: in test_find_reactions_with_identical_genes\n",
      "    \u001b[94massert\u001b[39;49;00m num_dup == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: Based only on equal genes there are 132 different groups of\u001b[0m\n",
      "\u001b[1m\u001b[31mE     identical reactions which corresponds to a total of 418\u001b[0m\n",
      "\u001b[1m\u001b[31mE     duplicated reactions in the model.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 418 == 0\u001b[0m\n",
      "\u001b[31m\u001b[1m________________________ test_find_medium_metabolites _________________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py\u001b[0m:735: in test_find_medium_metabolites\n",
      "    ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmetric\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m] = \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) / num_ex\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   TypeError: unsupported operand type(s) for /: 'int' and 'list'\u001b[0m\n",
      "---------------------------- Captured stderr call -----------------------------\n",
      "Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "------------------------------ Captured log call ------------------------------\n",
      "\u001b[32mINFO    \u001b[0m cobra.medium.boundary_types:boundary_types.py:68 Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "\u001b[32mINFO    \u001b[0m cobra.medium.boundary_types:boundary_types.py:68 Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "\u001b[31m\u001b[1m__________________ test_biomass_consistency[biomass_SC4_bal] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py\u001b[0m:129: in test_biomass_consistency\n",
      "    \u001b[94massert\u001b[39;49;00m outcome, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][reaction_id]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: The component molar mass of the biomass reaction biomass_SC4_bal sums\u001b[0m\n",
      "\u001b[1m\u001b[31mE     up to 0.9631894889925042             which is outside of the 1e-03\u001b[0m\n",
      "\u001b[1m\u001b[31mE     margin from 1 mmol / g[CDW] / h.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert False\u001b[0m\n",
      "\u001b[31m\u001b[1m____________________ test_gam_in_biomass[biomass_SC4_bal] _____________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py\u001b[0m:379: in test_gam_in_biomass\n",
      "    \u001b[94massert\u001b[39;49;00m outcome, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][reaction_id]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: No, biomass_SC4_bal does not contain a term for growth-associated\u001b[0m\n",
      "\u001b[1m\u001b[31mE     maintenance.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert False\u001b[0m\n",
      "\u001b[31m\u001b[1m__________________ test_fast_growth_default[biomass_SC4_bal] __________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py\u001b[0m:433: in test_fast_growth_default\n",
      "    \u001b[94massert\u001b[39;49;00m outcome, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][reaction_id]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: Using the biomass reaction biomass_SC4_bal and when the model is\u001b[0m\n",
      "\u001b[1m\u001b[31mE     simulated on             the provided default medium the growth rate\u001b[0m\n",
      "\u001b[1m\u001b[31mE     is *lower* than that             of the fastest bacteria. This is to\u001b[0m\n",
      "\u001b[1m\u001b[31mE     be expected for             a majority of organisms.\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert False\u001b[0m\n",
      "\u001b[31m\u001b[1m__________ test_essential_precursors_not_in_biomass[biomass_SC4_bal] __________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py\u001b[0m:573: in test_essential_precursors_not_in_biomass\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][reaction_id]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m][reaction_id]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: biomass_SC4_bal lacks a total of 35 essential metabolites\u001b[0m\n",
      "\u001b[1m\u001b[31mE     (83.33% of all biomass precursors). Specifically         these are:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     ['MNXM94', 'MNXM55', 'MNXM134', 'MNXM76', 'MNXM61', 'MNXM97',\u001b[0m\n",
      "\u001b[1m\u001b[31mE     'MNXM53', 'MNXM114', 'MNXM42', 'MNXM142', 'MNXM37', 'MNXM89557',\u001b[0m\n",
      "\u001b[1m\u001b[31mE     'MNXM231', 'MNXM70', 'MNXM78', 'MNXM199', 'MNXM140', 'MNXM32',\u001b[0m\n",
      "\u001b[1m\u001b[31mE     'MNXM147', 'datp_c', 'dctp_c', 'dttp_c', 'dgtp_c', 'gtp_c', 'ctp_c',\u001b[0m\n",
      "\u001b[1m\u001b[31mE     'utp_c', 'nad_c', 'nadp_c', 'amet_c', 'fad_c', 'pydx5p_c', 'coa_c',\u001b[0m\n",
      "\u001b[1m\u001b[31mE     'thmpp_c', 'fmn_c', 'MNXM2'].\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 35 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 35 = len(['MNXM94', 'MNXM55', 'MNXM134', 'MNXM76', 'MNXM61', 'MNXM97', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m________________________ test_reaction_charge_balance _________________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:207: in test_reaction_charge_balance\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 20 (1.74%) reactions are charge unbalanced with at\u001b[0m\n",
      "\u001b[1m\u001b[31mE     least one of the metabolites not having a charge or the overall\u001b[0m\n",
      "\u001b[1m\u001b[31mE     charge not equal to 0: ASPOcm, AGAT_SC, MTHFD2, MTHFD, GTPCI, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 20 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 20 = len(['ASPOcm', 'AGAT_SC', 'MTHFD2', 'MTHFD', 'GTPCI', 'BTSr', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m_________________________ test_reaction_mass_balance __________________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:242: in test_reaction_mass_balance\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 31 (2.70%) reactions are mass unbalanced with at least\u001b[0m\n",
      "\u001b[1m\u001b[31mE     one of the metabolites not having a formula or the overall mass not\u001b[0m\n",
      "\u001b[1m\u001b[31mE     equal to 0: MIP2CS324_SC, ASPOcm, AGAT_SC, MTHFD2, MIP2CS126_SC, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 31 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 31 = len(['MIP2CS324_SC', 'ASPOcm', 'AGAT_SC', 'MTHFD2', 'MIP2CS126_SC', 'MTHFD', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m___________________________ test_blocked_reactions ____________________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:273: in test_blocked_reactions\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are 522 (41.23%) blocked reactions in         the model:\u001b[0m\n",
      "\u001b[1m\u001b[31mE     2MCITtm, 2OHPH_5tn, 2OMPH_5tm, 2OMPH_5tn, 34HPPOR, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 522 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 522 = len(['2MCITtm', '2OHPH_5tn', '2OMPH_5tm', '2OMPH_5tn', '34HPPOR', '34HPPt2p', ...])\u001b[0m\n",
      "---------------------------- Captured stderr call -----------------------------\n",
      "Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "------------------------------ Captured log call ------------------------------\n",
      "\u001b[32mINFO    \u001b[0m cobra.medium.boundary_types:boundary_types.py:68 Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "\u001b[31m\u001b[1m________________ test_find_stoichiometrically_balanced_cycles _________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:299: in test_find_stoichiometrically_balanced_cycles\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: There are 64 (5.06%) reactions         which participate in SBC in the\u001b[0m\n",
      "\u001b[1m\u001b[31mE     model: 34HPPt2m, 4HGLSDm, ACONT, ACONTm, ADK1, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 64 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 64 = len(['34HPPt2m', '4HGLSDm', 'ACONT', 'ACONTm', 'ADK1', 'ADK3', ...])\u001b[0m\n",
      "\u001b[31m\u001b[1m______________________________ test_find_orphans ______________________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:326: in test_find_orphans\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 71 (6.69%) metabolites are not produced by any reaction\u001b[0m\n",
      "\u001b[1m\u001b[31mE     of the model: 1p3h5c_c, 35ccmp_c, 35cdamp_c, 35cgmp_c, 35cimp_c, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 71 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 71 = len(['1p3h5c_c', '35ccmp_c', '35cdamp_c', '35cgmp_c', '35cimp_c', '4gudbd_c', ...])\u001b[0m\n",
      "---------------------------- Captured stderr call -----------------------------\n",
      "Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "------------------------------ Captured log call ------------------------------\n",
      "\u001b[32mINFO    \u001b[0m cobra.medium.boundary_types:boundary_types.py:68 Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "\u001b[31m\u001b[1m_____________________________ test_find_deadends ______________________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:352: in test_find_deadends\n",
      "    \u001b[94massert\u001b[39;49;00m ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m] == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 85 (8.01%) metabolites are not consumed by any reaction\u001b[0m\n",
      "\u001b[1m\u001b[31mE     of the model: 13dampp_c, 2dr5p_c, 2hhxdal_c, 4gudbutn_c, 4hthr_c, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert ['13dampp_c', '2dr5p_c', '2hhxdal_c', '4gudbutn_c', '4hthr_c', '5mta_c', ...] == 0\u001b[0m\n",
      "---------------------------- Captured stderr call -----------------------------\n",
      "Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "------------------------------ Captured log call ------------------------------\n",
      "\u001b[32mINFO    \u001b[0m cobra.medium.boundary_types:boundary_types.py:68 Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "\u001b[31m\u001b[1m_____________ test_find_metabolites_not_produced_with_open_bounds _____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:408: in test_find_metabolites_not_produced_with_open_bounds\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 339 (31.95%) metabolites cannot be produced in complete\u001b[0m\n",
      "\u001b[1m\u001b[31mE     medium: 1p3h5c_c, 1p3h5c_m, 2dr5p_c, 2mcit_c, 2mcit_m, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 339 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 339 = len(['1p3h5c_c', '1p3h5c_m', '2dr5p_c', '2mcit_c', '2mcit_m', '2ohph_5_c', ...])\u001b[0m\n",
      "---------------------------- Captured stderr call -----------------------------\n",
      "Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "------------------------------ Captured log call ------------------------------\n",
      "\u001b[32mINFO    \u001b[0m cobra.medium.boundary_types:boundary_types.py:68 Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "\u001b[31m\u001b[1m_____________ test_find_metabolites_not_consumed_with_open_bounds _____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:438: in test_find_metabolites_not_consumed_with_open_bounds\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 473 (44.58%) metabolites cannot be consumed in complete\u001b[0m\n",
      "\u001b[1m\u001b[31mE     medium: 10fthf_c, 10fthf_m, 13dampp_c, 1p3h5c_c, 1p3h5c_m, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 473 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 473 = len(['10fthf_c', '10fthf_m', '13dampp_c', '1p3h5c_c', '1p3h5c_m', '25dhpp_c', ...])\u001b[0m\n",
      "---------------------------- Captured stderr call -----------------------------\n",
      "Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "------------------------------ Captured log call ------------------------------\n",
      "\u001b[32mINFO    \u001b[0m cobra.medium.boundary_types:boundary_types.py:68 Compartment `e` sounds like an external compartment. Using this one without counting boundary reactions.\n",
      "\u001b[31m\u001b[1m____________ test_find_reactions_unbounded_flux_default_condition _____________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py\u001b[0m:473: in test_find_reactions_unbounded_flux_default_condition\n",
      "    \u001b[94massert\u001b[39;49;00m ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmetric\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m] <= \u001b[94m0.1\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError:  A fraction of 16.71% of the non-blocked reactions (in total 64\u001b[0m\n",
      "\u001b[1m\u001b[31mE     reactions) can carry unbounded flux in the default model\u001b[0m\n",
      "\u001b[1m\u001b[31mE     condition. Unbounded reactions may be involved in\u001b[0m\n",
      "\u001b[1m\u001b[31mE     thermodynamically infeasible cycles: 34HPPt2m, 4HGLSDm, ACONT, ACONTm,\u001b[0m\n",
      "\u001b[1m\u001b[31mE     ADK1, ...\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 0.1671018276762402 <= 0.1\u001b[0m\n",
      "\u001b[31m\u001b[1m_____________________ test_biomass_specific_sbo_presence ______________________\u001b[0m\n",
      "\u001b[1m\u001b[31mC:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py\u001b[0m:447: in test_biomass_specific_sbo_presence\n",
      "    \u001b[94massert\u001b[39;49;00m \u001b[96mlen\u001b[39;49;00m(ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mdata\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]) == \u001b[94m0\u001b[39;49;00m, ann[\u001b[33m\"\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m\"\u001b[39;49;00m]\u001b[90m\u001b[39;49;00m\n",
      "\u001b[1m\u001b[31mE   AssertionError: A total of 1 biomass reactions (100.00% of all biomass reactions)\u001b[0m\n",
      "\u001b[1m\u001b[31mE     lack annotation with the SBO term \"SBO:0000629\" for         'biomass\u001b[0m\n",
      "\u001b[1m\u001b[31mE     production': biomass_SC4_bal\u001b[0m\n",
      "\u001b[1m\u001b[31mE   assert 1 == 0\u001b[0m\n",
      "\u001b[1m\u001b[31mE    +  where 1 = len(['biomass_SC4_bal'])\u001b[0m\n",
      "\u001b[33m============================== warnings summary ===============================\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: anyio\n",
      "    self._mark_plugins_for_rewrite(hook)\n",
      "\n",
      "-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html\n",
      "\u001b[36m\u001b[1m=========================== short test summary info ===========================\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_presence\u001b[0m - AssertionError: A total of 1061 metabolites (100.00%) lack any form of anno...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_presence\u001b[0m - AssertionError: A total of 1266 reactions (100.00%) lack any form of annota...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_presence\u001b[0m - AssertionError: A total of 750 genes (100.00%) lack any form of         ann...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[pubchem.compound]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[kegg.compound]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[seed.compound]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchikey]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchi]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[chebi]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[hmdb]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[metanetx.chemical]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[bigg.metabolite]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[rhea]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[kegg.reaction]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[seed.reaction]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[metanetx.reaction]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[bigg.reaction]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[ec-code]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[brenda]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[refseq]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for refseq:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[uniprot]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for uniprot:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ecogene]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ecogene:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[kegg.genes]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for kegg....\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigi]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ncbigi:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigene]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ncbig...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbiprotein]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ncbip...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ccds]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ccds:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[hprd]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for hprd:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[asap]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for asap:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[pubchem.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the pubchem.compoun...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[kegg.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the kegg.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[seed.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the seed.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchikey]\u001b[0m - AssertionError: There are no metabolite annotations for the inchikey database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchi]\u001b[0m - AssertionError: There are no metabolite annotations for the inchi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[chebi]\u001b[0m - AssertionError: There are no metabolite annotations for the chebi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[hmdb]\u001b[0m - AssertionError: There are no metabolite annotations for the hmdb database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no metabolite annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[metanetx.chemical]\u001b[0m - AssertionError: There are no metabolite annotations for the metanetx.chemical\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[bigg.metabolite]\u001b[0m - AssertionError: There are no metabolite annotations for the bigg.metabolite...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no metabolite annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[rhea]\u001b[0m - AssertionError: There are no reaction annotations for the rhea database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[kegg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the kegg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[seed.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the seed.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[metanetx.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the metanetx.reaction...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[bigg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the bigg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no reaction annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[ec-code]\u001b[0m - AssertionError: There are no reaction annotations for the ec-code database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[brenda]\u001b[0m - AssertionError: There are no reaction annotations for the brenda database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no reaction annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[refseq]\u001b[0m - AssertionError: There are no gene annotations for the refseq database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[uniprot]\u001b[0m - AssertionError: There are no gene annotations for the uniprot database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ecogene]\u001b[0m - AssertionError: There are no gene annotations for the ecogene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[kegg.genes]\u001b[0m - AssertionError: There are no gene annotations for the kegg.genes database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigi]\u001b[0m - AssertionError: There are no gene annotations for the ncbigi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigene]\u001b[0m - AssertionError: There are no gene annotations for the ncbigene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbiprotein]\u001b[0m - AssertionError: There are no gene annotations for the ncbiprotein database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ccds]\u001b[0m - AssertionError: There are no gene annotations for the ccds database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[hprd]\u001b[0m - AssertionError: There are no gene annotations for the hprd database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[asap]\u001b[0m - AssertionError: There are no gene annotations for the asap database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_gene_protein_reaction_rule_presence\u001b[0m - AssertionError: There are a total of 340 reactions (26.86%) without GPR:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_ngam_presence\u001b[0m - RuntimeError: It was not possible to uniquely identify a single metabolite ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_transport_reaction_gpr_presence\u001b[0m - AssertionError: There are a total of 212 transport reactions (71.38% of all\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_reactions_with_identical_genes\u001b[0m - AssertionError: Based only on equal genes there are 132 different groups of\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_medium_metabolites\u001b[0m - TypeError: unsupported operand type(s) for /: 'int' and 'list'\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_biomass_consistency[biomass_SC4_bal]\u001b[0m - AssertionError: The component molar mass of the biomass reaction biomass_SC...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_gam_in_biomass[biomass_SC4_bal]\u001b[0m - AssertionError: No, biomass_SC4_bal does not contain a term for growth-asso...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_fast_growth_default[biomass_SC4_bal]\u001b[0m - AssertionError: Using the biomass reaction biomass_SC4_bal and when the mod...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_essential_precursors_not_in_biomass[biomass_SC4_bal]\u001b[0m - AssertionError: biomass_SC4_bal lacks a total of 35 essential metabolites\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_charge_balance\u001b[0m - AssertionError: A total of 20 (1.74%) reactions are charge unbalanced with at\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_mass_balance\u001b[0m - AssertionError: A total of 31 (2.70%) reactions are mass unbalanced with at...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_blocked_reactions\u001b[0m - AssertionError: There are 522 (41.23%) blocked reactions in         the model:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_stoichiometrically_balanced_cycles\u001b[0m - AssertionError: There are 64 (5.06%) reactions         which participate in...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_orphans\u001b[0m - AssertionError: A total of 71 (6.69%) metabolites are not produced by any r...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_deadends\u001b[0m - AssertionError: A total of 85 (8.01%) metabolites are not consumed by any r...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_produced_with_open_bounds\u001b[0m - AssertionError: A total of 339 (31.95%) metabolites cannot be produced in c...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_consumed_with_open_bounds\u001b[0m - AssertionError: A total of 473 (44.58%) metabolites cannot be consumed in c...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_reactions_unbounded_flux_default_condition\u001b[0m - AssertionError:  A fraction of 16.71% of the non-blocked reactions (in tota...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_biomass_specific_sbo_presence\u001b[0m - AssertionError: A total of 1 biomass reactions (100.00% of all biomass reac...\n",
      "\u001b[31m====== \u001b[31m\u001b[1m82 failed\u001b[0m, \u001b[32m44 passed\u001b[0m, \u001b[33m21 skipped\u001b[0m, \u001b[33m1 warning\u001b[0m\u001b[31m in 145.65s (0:02:25)\u001b[0m\u001b[31m =======\u001b[0m\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Checking `git` installation.\n",
      "warning: We highly recommend keeping your model in a git repository. It allows you to track changes and to easily collaborate with others via online platforms such as https://github.com.\n",
      "The current solver interface glpk doesn't support setting the optimality tolerance.\n",
      "Storing result in 'result.json.gz'.\n"
     ]
    }
   ],
   "source": [
    "!memote run iND750.xml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "id": "017f4e51",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing problem data to 'C:\\Users\\pauli\\AppData\\Local\\Temp\\tmpuae5w8gs.glpk'..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The current solver interface glpk doesn't support setting the optimality tolerance.\n",
      "warning: Model does not contain SBML fbc package information.\n",
      "The current solver interface glpk doesn't support setting the optimality tolerance.\n",
      "warning: SBML package 'layout' not supported by cobrapy, information is not parsed\n",
      "warning: SBML package 'render' not supported by cobrapy, information is not parsed\n",
      "warning: #metaid_M_1_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_2_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_3_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_4_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_5_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_6_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_7_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_8_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_9_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_10_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_11_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_12_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_13_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_14_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_15_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_16_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_17_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_18_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_19_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_20_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_21_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_22_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_23_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_24_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_25_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_26_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_27_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_28_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_29 \"1,3-beta-D-glucan\">\n",
      "warning: #metaid_M_30_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_31_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_32_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_33_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_34_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_35_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_36_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_37_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_38 \"1-phosphatidyl-1D-myo-inositols\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_39 \"1-phosphatidyl-1D-myo-inositol 3,5-bisphosphates\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_40 \"1-phosphatidyl-1D-myo-inositol 3-phosphates\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_41 \"1-phosphatidyl-1D-myo-inositol 4,5-bisphosphates\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_42 \"1-phosphatidyl-1D-myo-inositol 4-phosphates\">\n",
      "warning: #metaid_M_44_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_45_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_46_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_47_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_48_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_49_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_50_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_51_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_52_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_53_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_54_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_55_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_56_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_57_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_58_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_59_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_60_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_61_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_62_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_63_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_64_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_65_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_66_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_67_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_68_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_69_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_70_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_71_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_75_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_76_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_78_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_79_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_80_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_81_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_82_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_83_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_84_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_85_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_86_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_87_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_88_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_89_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_90_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_91_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_92_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_93_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_94_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_95_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_96_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_97_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_98_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_99_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_107_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_108_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_109_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_110_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_111_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_112_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_113_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_114_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_115_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_126_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_128_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_129_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_130_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_131_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_132_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_133 \"3-oxodecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_134 \"3-oxodecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_135 \"3-oxododecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_136 \"3-oxododecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_137 \"3-oxohexadecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_138 \"3-oxohexadecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_139 \"3-oxohexanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_140 \"3-oxohexanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_141 \"3-oxostearoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_142 \"3-oxostearoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_143 \"3-oxooctanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_144 \"3-oxooctanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_145 \"3-oxotetradecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_146 \"3-oxotetradecanoyl-[acp]\">\n",
      "warning: #metaid_M_147_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_148_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_149_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_150_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_151_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_152_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_153_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_154_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_155_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_156_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_157_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_158_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_159_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_160_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_161_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_162_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_163_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_164_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_165_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_166_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_167_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_168_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_169_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_170_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_171_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_172_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_173_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_174_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_175_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_176_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_177_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_178_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_179_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_180_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_181_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_182_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_183_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_184_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_185_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_186_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_187_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_188_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_189_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_190_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_191_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_192_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_193_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_194_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_195_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_196_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_197_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_198_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_199_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_200_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_201_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_202_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_203_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_204_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_205_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_206_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_207_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_208_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_209 \"acetoacetyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_210 \"acetoacetyl-[acp]\">\n",
      "warning: #metaid_M_211_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_212 \"acetyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_213 \"acetyl-[acp]\">\n",
      "warning: #metaid_M_214_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_215_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_216_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_218 \"acyl-carrier protein\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_219 \"acyl-carrier protein\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_220 \"acyl-sn-glycerol 3-phosphates\">\n",
      "warning: #metaid_M_223_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_224_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_225_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_226_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_227_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_228_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_229_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_230_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_231_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_232_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_233_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_234_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_235_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_236_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_237_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_238_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_239_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_240_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_241_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_242_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_243_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_244 \"alpha-D-fucosides\">\n",
      "warning: #metaid_M_245_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_246_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_247_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_248_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_249_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_250_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_251_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_252_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_253_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_254_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_255_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_256_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_257_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_258_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_259_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_260_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_261_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_262_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_263_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_264_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_265_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_266_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_267_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_268_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_269_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_270_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_271_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_273_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_275_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_276_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_277_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_278 \"butyryl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_279 \"butyryl-[acp]\">\n",
      "warning: #metaid_M_280_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_281_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_282_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_285_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_287_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_297_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_298 \"cardiolipin\">\n",
      "warning: #metaid_M_299_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_300_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_301_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_302_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_303_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_304 \"CDP-diacylglycerol\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_305 \"CDP-diacylglycerol\">\n",
      "warning: #metaid_M_306_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_307 \"chitin\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_308 \"chitosan\">\n",
      "warning: #metaid_M_309_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_310_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_311_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_312_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_313_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_314_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_315_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_316_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_317_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_318_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_319_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_320_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_321_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_322_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_323_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_324_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_325_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_326_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_327_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_328_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_329_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_330_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_331_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_332_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_333_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_334_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_336_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_337_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_338_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_339_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_340_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_341_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_342_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_343_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_344_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_345_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_346_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_347_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_348_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_349_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_350_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_351_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_352_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_353_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_354_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_355_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_356_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_357_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_358_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_359_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_360_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_361_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_362_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_363_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_364_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_365_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_366_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_367_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_368_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_369_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_370_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_371_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_372_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_373_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_374_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_375_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_376 \"decanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_377 \"decanoyl-[acp]\">\n",
      "warning: #metaid_M_378_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_379_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_380_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_381_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_382_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_383_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_384_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_385_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_386_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_387_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_388_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_389_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_390_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_391_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_392_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_393_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_394 \"diglycerides\">\n",
      "warning: #metaid_M_396_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_397_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_398_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_399_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_400_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_401_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_402_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_403 \"dipeptide\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_404 \"dipeptide\">\n",
      "warning: #metaid_M_405_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_406 \"dodecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_407 \"dodecanoyl-[acp]\">\n",
      "warning: #metaid_M_408_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_409_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_410_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_411 \"dolichyl phosphate\">\n",
      "warning: #metaid_M_412_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_413_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_414_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_415_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_416_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_417_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_418_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_419_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_420_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_421_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_422_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_423_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_424_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_425_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_426_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_427 \"5,7,24(28)-ergostatrienol\">\n",
      "warning: #metaid_M_428_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_429_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_430 \"ergosterol ester\">\n",
      "warning: #metaid_M_431_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_432_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_433_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_434_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_435_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_436_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_437_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_438_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_439_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_440_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_441_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_445_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_446_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_447_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_448_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_449_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_450_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_451_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_452_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_453_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_454_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_455_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_456_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_457_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_458_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_459_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_460_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_461_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_462_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_463_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_464_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_465_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_466_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_467_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_468_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_469_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_470_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_471_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_472_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_473_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_474_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_475_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_476_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_477_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_478_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_479_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_480_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_483_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_484_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_486_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_487 \"glycylpeptide\">\n",
      "warning: #metaid_M_488_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_489_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_490_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_491_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_494_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_495_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_496_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_497_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_498_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_499_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_500_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_501_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_502_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_503_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_504_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_505_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_508 \"hexadecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_509 \"hexadecanoyl-[acp]\">\n",
      "warning: #metaid_M_510_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_511 \"hexanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_512 \"hexanoyl-[acp]\">\n",
      "warning: #metaid_M_513_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_514_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_515_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_516_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_517_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_518_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_519_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_520_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_521_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_522_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_523_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_524_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_525_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_526_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_527_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_528_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_529_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_530_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_531_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_536_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_541_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_542_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_543_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_544_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_545_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_546_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_547_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_548_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_549_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_550_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_551_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_552_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_553_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_554_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_555_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_556 \"L-alanyl-tRNA(Ala)\">\n",
      "warning: #metaid_M_557_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_558 \"L-arginyl-tRNA(Arg)\">\n",
      "warning: #metaid_M_559_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_560 \"L-asparaginyl-tRNA(Asn)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_561 \"L-asparaginyl-tRNA(Asn)\">\n",
      "warning: #metaid_M_562_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_563_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_564_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_565 \"L-aspartyl-tRNA(Asp)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_566 \"L-aspartyl-tRNA(Asp)\">\n",
      "warning: #metaid_M_567_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_568_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_569_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_570_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_571 \"L-cysteinyl-tRNA(Cys)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_572 \"L-cysteinyl-tRNA(Cys)\">\n",
      "warning: #metaid_M_573_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_574_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_575_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_576_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_577_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_578_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_579 \"L-glutaminyl-tRNA(Gln)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_580 \"L-glutamyl-tRNA(Glu)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_581 \"L-glutamyl-tRNA(Glu)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_582 \"glycyl-tRNA(Gly)\">\n",
      "warning: #metaid_M_583_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_584_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_585_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_586 \"L-histidyl-tRNA(His)\">\n",
      "warning: #metaid_M_587_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_588_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_589_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_590 \"L-isoleucyl-tRNA(Ile)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_591 \"L-isoleucyl-tRNA(Ile)\">\n",
      "warning: #metaid_M_592_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_593_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_594_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_595 \"L-leucyl-tRNA(Leu)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_596 \"L-leucyl-tRNA(Leu)\">\n",
      "warning: #metaid_M_597_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_598_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_599 \"L-lysyl-tRNA(Lys)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_600 \"L-lysyl-tRNA(Lys)\">\n",
      "warning: #metaid_M_601_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_602 \"L-methionyl-tRNA(Met)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_603 \"L-methionyl-tRNA(Met)\">\n",
      "warning: #metaid_M_604_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_605_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_606_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_607 \"L-phenylalanyl-tRNA(Phe)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_608 \"L-phenylalanyl-tRNA(Phe)\">\n",
      "warning: #metaid_M_609_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_610_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_611 \"L-prolyl-tRNA(Pro)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_612 \"L-prolyl-tRNA(Pro)\">\n",
      "warning: #metaid_M_613_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_614_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_615 \"L-seryl-tRNA(Ser)\">\n",
      "warning: #metaid_M_616_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_617_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_618 \"L-threonyl-tRNA(Thr)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_619 \"L-threonyl-tRNA(Thr)\">\n",
      "warning: #metaid_M_620_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_621_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_622 \"L-tryptophanyl-tRNA(Trp)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_623 \"L-tryptophanyl-tRNA(Trp)\">\n",
      "warning: #metaid_M_624_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_625 \"L-tyrosyl-tRNA(Tyr)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_626 \"L-tyrosyl-tRNA(Tyr)\">\n",
      "warning: #metaid_M_627_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_628 \"L-valyl-tRNA(Val)\">\n",
      "warning: #metaid_M_629_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_630_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_631_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_632_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_634_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_635 \"long-chain acyl-CoAs\">\n",
      "warning: #metaid_M_636_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_637_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_638_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_639_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_640_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_641_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_642 \"malonyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_643 \"malonyl-[acp]\">\n",
      "warning: #metaid_M_644_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_645_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_646_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_647_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_648 \"mannan\">\n",
      "warning: #metaid_M_651_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_652_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_653_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_654_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_655_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_656_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_657_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_658_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_659_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_660_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_661_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_662_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_663_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_664_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_665_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_666_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_667_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_668_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_669_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_670_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_671_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_672_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_673_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_674_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_675_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_676_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_677_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_678 \"N-formylmethionyl-tRNA(fMet)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_679 \"N-tetradecanoylglycylpeptide\">\n",
      "warning: #metaid_M_680_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_681_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_682_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_683_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_684_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_685_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_686_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_687_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_688_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_689_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_690_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_691_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_692_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_693_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_694_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_695_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_696 \"Nalpha-acetylpeptide\">\n",
      "warning: #metaid_M_697_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_698_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_699_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_700_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_701_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_702_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_703_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_704_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_705_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_706_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_707_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_708_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_709_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_710_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_711_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_712_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_713_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_714_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_715_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_716_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_717_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_718_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_719_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_721 \"stearoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_722 \"stearoyl-[acp]\">\n",
      "warning: #metaid_M_723_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_724 \"octanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_725 \"octanoyl-[acp]\">\n",
      "warning: #metaid_M_726_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_727_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_728 \"oligopeptide\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_729 \"oligopeptide\">\n",
      "warning: #metaid_M_730_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_731_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_732_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_733_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_734_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_735_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_736_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_737_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_738_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_739_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_740_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_741_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_742 \"thioredoxin disulfide\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_743 \"thioredoxin disulfide\">\n",
      "warning: #metaid_M_744_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_745_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_747_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_748_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_749_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_750_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_751_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_752 \"peptide\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_753 \"peptide\">\n",
      "warning: #metaid_M_754_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_755_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_756_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_757 \"phosphatidate\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_758 \"phosphatidate\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_759 \"phosphatidyl-N-dimethylethanolamine\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_760 \"phosphatidyl-N-methylethanolamine\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_761 \"phosphatidylcholines\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_762 \"phosphatidylethanolamines\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_763 \"phosphatidylethanolamines\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_764 \"phosphatidylglycerol\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_765 \"phosphatidylglycerophosphate\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_766 \"phosphatidylserine\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_767 \"phosphatidylserine\">\n",
      "warning: #metaid_M_768_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_769_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_770_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_771_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_772_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_773_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_774_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_775_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_776_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_777_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_778_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_779_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_780_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_781_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_782_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_783_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_784_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_785_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_786_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_787_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_788_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_789_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_790_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_791_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_792_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_793_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_794_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_795_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_796_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_797_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_798_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_799_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_800 \"(R)-3-hydroxybutanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_801 \"(R)-3-hydroxybutanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_802 \"(R)-3-hydroxydecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_803 \"(R)-3-hydroxydecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_804 \"(R)-3-hydroxydodecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_805 \"(R)-3-hydroxydodecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_807 \"(R)-3-hydroxypalmitoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_808 \"(R)-3-hydroxypalmitoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_809 \"(R)-3-hydroxyhexanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_810 \"(R)-3-hydroxyhexanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_811 \"3-hydroxyoctadecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_812 \"3-hydroxyoctadecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_813 \"(R)-3-hydroxyoctanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_814 \"(R)-3-hydroxyoctanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_815 \"(R)-3-hydroxytetradecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_816 \"(R)-3-hydroxytetradecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_817 \"R-S-alanylglycine\">\n",
      "warning: #metaid_M_818_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_819 \"thioredoxin dithiol\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_820 \"thioredoxin dithiol\">\n",
      "warning: #metaid_M_821_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_822_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_823_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_824_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_825_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_826_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_827_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_828_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_829_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_830_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_831_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_832_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_833_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_834 \"S-aminomethyldihydrolipoylprotein\">\n",
      "warning: #metaid_M_835_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_836_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_837_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_838_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_839_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_840_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_841_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_842_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_843_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_844_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_845_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_846_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_847_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_848_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_849_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_850_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_851_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_852_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_853_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_854_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_855_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_856_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_857_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_858_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_859_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_860_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_861_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_862_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_863_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_864_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_865_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_866_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_867_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_870 \"tetradecanoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_871 \"tetradecanoyl-[acp]\">\n",
      "warning: #metaid_M_872_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_873_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_874_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_875_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_876_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_877_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_878_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_879_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_880_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_881_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_882_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_883_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_884_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_885_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_886_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_887_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_905_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_908 \"trans-dec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_909 \"trans-dec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_915 \"trans-dodec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_916 \"trans-dodec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_917 \"trans-hex-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_918 \"trans-hex-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_919 \"trans-hexadec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_920 \"trans-hexadec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_921 \"trans-oct-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_922 \"trans-oct-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_923 \"trans-octadec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_924 \"trans-octadec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_925 \"trans-tetradec-2-enoyl-[acp]\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_926 \"trans-tetradec-2-enoyl-[acp]\">\n",
      "warning: #metaid_M_927_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_928 \"triglycerides\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_929 \"transfer RNA\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_930 \"tRNA(Ala)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_931 \"tRNA(Arg)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_932 \"tRNA(Asp)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_933 \"tRNA(Asp)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_934 \"tRNA(Cys)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_935 \"tRNA(Cys)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_936 \"tRNA(Gly)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_937 \"tRNA(His)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_938 \"tRNA(Ile)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_939 \"tRNA(Ile)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_940 \"tRNA(Leu)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_941 \"tRNA(Leu)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_942 \"tRNA(Lys)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_943 \"tRNA(Lys)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_944 \"tRNA(Met)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_945 \"tRNA(Met)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_946 \"tRNA(Phe)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_947 \"tRNA(Phe)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_948 \"tRNA(Pro)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_949 \"tRNA(Pro)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_950 \"tRNA(Ser)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_951 \"tRNA(Thr)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_952 \"tRNA(Thr)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_953 \"tRNA(Trp)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_954 \"tRNA(Tyr)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_955 \"tRNA(Tyr)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_956 \"tRNA(Val)\">\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_957 \"transfer RNA\">\n",
      "warning: #metaid_M_958_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_959_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_961_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_962_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_963_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_964_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_965_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_966_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_967_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_968_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_969_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_970_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_971_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_972_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_973_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_974_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_975_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_976_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_977_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_978_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_979_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_980_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_981_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_982_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_983_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_984_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_985_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_986_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_987_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_988_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_989_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_990_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_991_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_992_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_993_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_994_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_995_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_996_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_997_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_998_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_999_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1000_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1001_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1003_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1004_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1005_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1006_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1008_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1010_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1014_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1015_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1016_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1017_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1018_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1019_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1020_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1021_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1022_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1023_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1024_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1025_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_1026 \"dipeptide\">\n",
      "warning: #metaid_M_1027_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1028_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1029_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1030_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1031_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1032_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1033_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1034_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1035_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1036_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1037_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1038_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1039_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1040_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1041_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1042_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1043_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1044_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1045_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1046_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1047_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1048_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1049_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1050_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1051_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1054_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1055_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1056_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1057_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1058_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1059_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1060_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1061_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1062_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1063_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1064_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1065_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1066_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1067_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1068_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1069_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1070_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1071_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1072_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1073_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1074_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1075_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1076_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1077_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1078_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1079_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_1080 \"oligopeptide\">\n",
      "warning: #metaid_M_1081_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1082_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_1083 \"peptide\">\n",
      "warning: #metaid_M_1084_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1085_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1086_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1087_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1088_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1089_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1090_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1091_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1092_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1093_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1094_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1095_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1096_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1097_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1098_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1099_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1100_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1101_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1102_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1103_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1104_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1105_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1106_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1107_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1108_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1109_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1110_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1111_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1112_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1113_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1114_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1115_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: #metaid_M_1116_inchi does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "warning: Adding exchange reaction EX_986 with default bounds for boundary metabolite: 986.\n",
      "warning: Adding exchange reaction EX_987 with default bounds for boundary metabolite: 987.\n",
      "warning: Adding exchange reaction EX_988 with default bounds for boundary metabolite: 988.\n",
      "warning: Adding exchange reaction EX_989 with default bounds for boundary metabolite: 989.\n",
      "warning: Adding exchange reaction EX_990 with default bounds for boundary metabolite: 990.\n",
      "warning: Adding exchange reaction EX_991 with default bounds for boundary metabolite: 991.\n",
      "warning: Adding exchange reaction EX_992 with default bounds for boundary metabolite: 992.\n",
      "warning: Adding exchange reaction EX_993 with default bounds for boundary metabolite: 993.\n",
      "warning: Adding exchange reaction EX_994 with default bounds for boundary metabolite: 994.\n",
      "warning: Adding exchange reaction EX_995 with default bounds for boundary metabolite: 995.\n",
      "warning: Adding exchange reaction EX_996 with default bounds for boundary metabolite: 996.\n",
      "warning: Adding exchange reaction EX_997 with default bounds for boundary metabolite: 997.\n",
      "warning: Adding exchange reaction EX_998 with default bounds for boundary metabolite: 998.\n",
      "warning: Adding exchange reaction EX_999 with default bounds for boundary metabolite: 999.\n",
      "warning: Adding exchange reaction EX_1000 with default bounds for boundary metabolite: 1000.\n",
      "warning: Adding exchange reaction EX_1001 with default bounds for boundary metabolite: 1001.\n",
      "warning: Adding exchange reaction EX_1002 with default bounds for boundary metabolite: 1002.\n",
      "warning: Adding exchange reaction EX_1003 with default bounds for boundary metabolite: 1003.\n",
      "warning: Adding exchange reaction EX_1004 with default bounds for boundary metabolite: 1004.\n",
      "warning: Adding exchange reaction EX_1005 with default bounds for boundary metabolite: 1005.\n",
      "warning: Adding exchange reaction EX_1006 with default bounds for boundary metabolite: 1006.\n",
      "warning: Adding exchange reaction EX_1007 with default bounds for boundary metabolite: 1007.\n",
      "warning: Adding exchange reaction EX_1008 with default bounds for boundary metabolite: 1008.\n",
      "warning: Adding exchange reaction EX_1009 with default bounds for boundary metabolite: 1009.\n",
      "warning: Adding exchange reaction EX_1010 with default bounds for boundary metabolite: 1010.\n",
      "warning: Adding exchange reaction EX_1011 with default bounds for boundary metabolite: 1011.\n",
      "warning: Adding exchange reaction EX_1012 with default bounds for boundary metabolite: 1012.\n",
      "warning: Adding exchange reaction EX_1013 with default bounds for boundary metabolite: 1013.\n",
      "warning: Adding exchange reaction EX_1014 with default bounds for boundary metabolite: 1014.\n",
      "warning: Adding exchange reaction EX_1015 with default bounds for boundary metabolite: 1015.\n",
      "warning: Adding exchange reaction EX_1016 with default bounds for boundary metabolite: 1016.\n",
      "warning: Adding exchange reaction EX_1017 with default bounds for boundary metabolite: 1017.\n",
      "warning: Adding exchange reaction EX_1018 with default bounds for boundary metabolite: 1018.\n",
      "warning: Adding exchange reaction EX_1019 with default bounds for boundary metabolite: 1019.\n",
      "warning: Adding exchange reaction EX_1020 with default bounds for boundary metabolite: 1020.\n",
      "warning: Adding exchange reaction EX_1021 with default bounds for boundary metabolite: 1021.\n",
      "warning: Adding exchange reaction EX_1022 with default bounds for boundary metabolite: 1022.\n",
      "warning: Adding exchange reaction EX_1023 with default bounds for boundary metabolite: 1023.\n",
      "warning: Adding exchange reaction EX_1024 with default bounds for boundary metabolite: 1024.\n",
      "warning: Adding exchange reaction EX_1025 with default bounds for boundary metabolite: 1025.\n",
      "warning: Adding exchange reaction EX_1026 with default bounds for boundary metabolite: 1026.\n",
      "warning: Adding exchange reaction EX_1027 with default bounds for boundary metabolite: 1027.\n",
      "warning: Adding exchange reaction EX_1028 with default bounds for boundary metabolite: 1028.\n",
      "warning: Adding exchange reaction EX_1029 with default bounds for boundary metabolite: 1029.\n",
      "warning: Adding exchange reaction EX_1030 with default bounds for boundary metabolite: 1030.\n",
      "warning: Adding exchange reaction EX_1031 with default bounds for boundary metabolite: 1031.\n",
      "warning: Adding exchange reaction EX_1032 with default bounds for boundary metabolite: 1032.\n",
      "warning: Adding exchange reaction EX_1033 with default bounds for boundary metabolite: 1033.\n",
      "warning: Adding exchange reaction EX_1034 with default bounds for boundary metabolite: 1034.\n",
      "warning: Adding exchange reaction EX_1035 with default bounds for boundary metabolite: 1035.\n",
      "warning: Adding exchange reaction EX_1036 with default bounds for boundary metabolite: 1036.\n",
      "warning: Adding exchange reaction EX_1037 with default bounds for boundary metabolite: 1037.\n",
      "warning: Adding exchange reaction EX_1038 with default bounds for boundary metabolite: 1038.\n",
      "warning: Adding exchange reaction EX_1039 with default bounds for boundary metabolite: 1039.\n",
      "warning: Adding exchange reaction EX_1040 with default bounds for boundary metabolite: 1040.\n",
      "warning: Adding exchange reaction EX_1041 with default bounds for boundary metabolite: 1041.\n",
      "warning: Adding exchange reaction EX_1042 with default bounds for boundary metabolite: 1042.\n",
      "warning: Adding exchange reaction EX_1043 with default bounds for boundary metabolite: 1043.\n",
      "warning: Adding exchange reaction EX_1044 with default bounds for boundary metabolite: 1044.\n",
      "warning: Adding exchange reaction EX_1045 with default bounds for boundary metabolite: 1045.\n",
      "warning: Adding exchange reaction EX_1046 with default bounds for boundary metabolite: 1046.\n",
      "warning: Adding exchange reaction EX_1047 with default bounds for boundary metabolite: 1047.\n",
      "warning: Adding exchange reaction EX_1048 with default bounds for boundary metabolite: 1048.\n",
      "warning: Adding exchange reaction EX_1049 with default bounds for boundary metabolite: 1049.\n",
      "warning: Adding exchange reaction EX_1050 with default bounds for boundary metabolite: 1050.\n",
      "warning: Adding exchange reaction EX_1051 with default bounds for boundary metabolite: 1051.\n",
      "warning: Adding exchange reaction EX_1052 with default bounds for boundary metabolite: 1052.\n",
      "warning: Adding exchange reaction EX_1053 with default bounds for boundary metabolite: 1053.\n",
      "warning: Adding exchange reaction EX_1054 with default bounds for boundary metabolite: 1054.\n",
      "warning: Adding exchange reaction EX_1055 with default bounds for boundary metabolite: 1055.\n",
      "warning: Adding exchange reaction EX_1056 with default bounds for boundary metabolite: 1056.\n",
      "warning: Adding exchange reaction EX_1057 with default bounds for boundary metabolite: 1057.\n",
      "warning: Adding exchange reaction EX_1058 with default bounds for boundary metabolite: 1058.\n",
      "warning: Adding exchange reaction EX_1059 with default bounds for boundary metabolite: 1059.\n",
      "warning: Adding exchange reaction EX_1060 with default bounds for boundary metabolite: 1060.\n",
      "warning: Adding exchange reaction EX_1061 with default bounds for boundary metabolite: 1061.\n",
      "warning: Adding exchange reaction EX_1062 with default bounds for boundary metabolite: 1062.\n",
      "warning: Adding exchange reaction EX_1063 with default bounds for boundary metabolite: 1063.\n",
      "warning: Adding exchange reaction EX_1064 with default bounds for boundary metabolite: 1064.\n",
      "warning: Adding exchange reaction EX_1065 with default bounds for boundary metabolite: 1065.\n",
      "warning: Adding exchange reaction EX_1066 with default bounds for boundary metabolite: 1066.\n",
      "warning: Adding exchange reaction EX_1067 with default bounds for boundary metabolite: 1067.\n",
      "warning: Adding exchange reaction EX_1068 with default bounds for boundary metabolite: 1068.\n",
      "warning: Adding exchange reaction EX_1069 with default bounds for boundary metabolite: 1069.\n",
      "warning: Adding exchange reaction EX_1070 with default bounds for boundary metabolite: 1070.\n",
      "warning: Adding exchange reaction EX_1071 with default bounds for boundary metabolite: 1071.\n",
      "warning: Adding exchange reaction EX_1072 with default bounds for boundary metabolite: 1072.\n",
      "warning: Adding exchange reaction EX_1073 with default bounds for boundary metabolite: 1073.\n",
      "warning: Adding exchange reaction EX_1074 with default bounds for boundary metabolite: 1074.\n",
      "warning: Adding exchange reaction EX_1075 with default bounds for boundary metabolite: 1075.\n",
      "warning: Adding exchange reaction EX_1076 with default bounds for boundary metabolite: 1076.\n",
      "warning: Adding exchange reaction EX_1077 with default bounds for boundary metabolite: 1077.\n",
      "warning: Adding exchange reaction EX_1078 with default bounds for boundary metabolite: 1078.\n",
      "warning: Adding exchange reaction EX_1079 with default bounds for boundary metabolite: 1079.\n",
      "warning: Adding exchange reaction EX_1080 with default bounds for boundary metabolite: 1080.\n",
      "warning: Adding exchange reaction EX_1081 with default bounds for boundary metabolite: 1081.\n",
      "warning: Adding exchange reaction EX_1082 with default bounds for boundary metabolite: 1082.\n",
      "warning: Adding exchange reaction EX_1083 with default bounds for boundary metabolite: 1083.\n",
      "warning: Adding exchange reaction EX_1084 with default bounds for boundary metabolite: 1084.\n",
      "warning: Adding exchange reaction EX_1085 with default bounds for boundary metabolite: 1085.\n",
      "warning: Adding exchange reaction EX_1086 with default bounds for boundary metabolite: 1086.\n",
      "warning: Adding exchange reaction EX_1087 with default bounds for boundary metabolite: 1087.\n",
      "warning: Adding exchange reaction EX_1088 with default bounds for boundary metabolite: 1088.\n",
      "warning: Adding exchange reaction EX_1089 with default bounds for boundary metabolite: 1089.\n",
      "warning: Adding exchange reaction EX_1090 with default bounds for boundary metabolite: 1090.\n",
      "warning: Adding exchange reaction EX_1091 with default bounds for boundary metabolite: 1091.\n",
      "warning: Adding exchange reaction EX_1092 with default bounds for boundary metabolite: 1092.\n",
      "warning: Adding exchange reaction EX_1093 with default bounds for boundary metabolite: 1093.\n",
      "warning: Adding exchange reaction EX_1094 with default bounds for boundary metabolite: 1094.\n",
      "warning: Adding exchange reaction EX_1095 with default bounds for boundary metabolite: 1095.\n",
      "warning: Adding exchange reaction EX_1096 with default bounds for boundary metabolite: 1096.\n",
      "warning: Adding exchange reaction EX_1097 with default bounds for boundary metabolite: 1097.\n",
      "warning: Adding exchange reaction EX_1098 with default bounds for boundary metabolite: 1098.\n",
      "warning: Adding exchange reaction EX_1099 with default bounds for boundary metabolite: 1099.\n",
      "warning: Adding exchange reaction EX_1100 with default bounds for boundary metabolite: 1100.\n",
      "warning: Adding exchange reaction EX_1101 with default bounds for boundary metabolite: 1101.\n",
      "warning: Adding exchange reaction EX_1102 with default bounds for boundary metabolite: 1102.\n",
      "warning: Adding exchange reaction EX_1103 with default bounds for boundary metabolite: 1103.\n",
      "warning: Adding exchange reaction EX_1104 with default bounds for boundary metabolite: 1104.\n",
      "warning: Adding exchange reaction EX_1105 with default bounds for boundary metabolite: 1105.\n",
      "warning: Adding exchange reaction EX_1106 with default bounds for boundary metabolite: 1106.\n",
      "warning: Adding exchange reaction EX_1107 with default bounds for boundary metabolite: 1107.\n",
      "warning: Adding exchange reaction EX_1108 with default bounds for boundary metabolite: 1108.\n",
      "warning: Adding exchange reaction EX_1109 with default bounds for boundary metabolite: 1109.\n",
      "warning: Adding exchange reaction EX_1110 with default bounds for boundary metabolite: 1110.\n",
      "warning: Adding exchange reaction EX_1111 with default bounds for boundary metabolite: 1111.\n",
      "warning: Adding exchange reaction EX_1112 with default bounds for boundary metabolite: 1112.\n",
      "warning: Adding exchange reaction EX_1113 with default bounds for boundary metabolite: 1113.\n",
      "warning: Adding exchange reaction EX_1114 with default bounds for boundary metabolite: 1114.\n",
      "warning: Adding exchange reaction EX_1115 with default bounds for boundary metabolite: 1115.\n",
      "warning: Adding exchange reaction EX_1116 with default bounds for boundary metabolite: 1116.\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAC1 \"ADP,ATP carrier protein 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAC1 \"ADP,ATP carrier protein 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAC3 \"ADP,ATP carrier protein 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAC3 \"ADP,ATP carrier protein 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAH1_1 \"Adenosine deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAH1_1 \"Adenosine deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAH1_2 \"Adenosine deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAH1_2 \"Adenosine deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAH1_3 \"Adenosine deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAH1_3 \"Adenosine deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAT1_1 \"Aspartate aminotransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAT1_1 \"Aspartate aminotransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAT1_2 \"Aspartate aminotransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAT1_2 \"Aspartate aminotransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAT2_1 \"Aspartate aminotransferase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAT2_1 \"Aspartate aminotransferase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AAT2_2 \"Aspartate aminotransferase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AAT2_2 \"Aspartate aminotransferase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ABZ1 \"Aminodeoxychorismate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ABZ1 \"Aminodeoxychorismate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACC1 \"Acetyl-CoA carboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACC1 \"Acetyl-CoA carboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACH1 \"Acetyl-CoA hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACH1 \"Acetyl-CoA hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACO1 \"Aconitate hydratase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACO1 \"Aconitate hydratase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACP1 \"Acyl carrier protein, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACP1 \"Acyl carrier protein, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACS1 \"Acetyl-coenzyme A synthetase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACS1 \"Acetyl-coenzyme A synthetase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACS2 \"Acetyl-coenzyme A synthetase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACS2 \"Acetyl-coenzyme A synthetase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE1 \"Phosphoribosylaminoimidazole-succinocarboxamide synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE1 \"Phosphoribosylaminoimidazole-succinocarboxamide synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE12 \"Adenylosuccinate synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE12 \"Adenylosuccinate synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE13_1 \"Adenylosuccinate lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE13_1 \"Adenylosuccinate lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE13_2 \"Adenylosuccinate lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE13_2 \"Adenylosuccinate lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE16_1 \"Phosphoribosylaminoimidazolecarboxamide formyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE16_1 \"Phosphoribosylaminoimidazolecarboxamide formyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE16_2 \"IMP cyclohydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE16_2 \"IMP cyclohydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE17_1 \"Phosphoribosylaminoimidazolecarboxamide formyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE17_1 \"Phosphoribosylaminoimidazolecarboxamide formyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE17_2 \"IMP cyclohydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE17_2 \"IMP cyclohydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE2 \"Phosphoribosylaminoimidazole carboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE2 \"Phosphoribosylaminoimidazole carboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE3_1 \"Methylenetetrahydrofolate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE3_1 \"Methylenetetrahydrofolate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE3_2 \"Formyltetrahydrofolate synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE3_2 \"Formyltetrahydrofolate synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE3_3 \"Methenyltetrahydrofolate cyclohydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE3_3 \"Methenyltetrahydrofolate cyclohydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE4 \"Amidophosphoribosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE4 \"Amidophosphoribosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE5_7_1 \"Phosphoribosylamine--glycine ligase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE5_7_1 \"Phosphoribosylamine--glycine ligase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE5_7_2 \"Phosphoribosylformylglycinamidine cyclo-ligase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE5_7_2 \"Phosphoribosylformylglycinamidine cyclo-ligase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE6 \"Phosphoribosylformylglycinamidine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE6 \"Phosphoribosylformylglycinamidine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADE8 \"Phosphoribosylglycinamide formyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADE8 \"Phosphoribosylglycinamide formyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADH1 \"Alcohol dehydrogenase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADH1 \"Alcohol dehydrogenase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADH2 \"Alcohol dehydrogenase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADH2 \"Alcohol dehydrogenase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADH3 \"Alcohol dehydrogenase 3, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADH3 \"Alcohol dehydrogenase 3, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADH4 \"Alcohol dehydrogenase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADH4 \"Alcohol dehydrogenase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADH5 \"Alcohol dehydrogenase 5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADH5 \"Alcohol dehydrogenase 5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADK1_1 \"Adenylate kinase cytosolic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADK1_1 \"Adenylate kinase cytosolic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADK1_2 \"Adenylate kinase cytosolic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADK1_2 \"Adenylate kinase cytosolic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADK1_3 \"Adenylate kinase cytosolic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADK1_3 \"Adenylate kinase cytosolic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADK2_1 \"Adenylate kinase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADK2_1 \"Adenylate kinase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADK2_2 \"Adenylate kinase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADK2_2 \"Adenylate kinase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADK2_3 \"Adenylate kinase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADK2_3 \"Adenylate kinase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADY2 \"Acetate transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADY2 \"Acetate transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGC1_1 \"Mitochondrial aspartate-glutamate transporter AGC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGC1_1 \"Mitochondrial aspartate-glutamate transporter AGC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGC1_2 \"Mitochondrial aspartate-glutamate transporter AGC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGC1_2 \"Mitochondrial aspartate-glutamate transporter AGC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_1 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_1 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_10 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_10 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_11 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_11 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_12 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_12 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_13 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_13 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_2 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_2 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_3 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_3 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_4 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_4 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_5 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_5 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_6 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_6 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_7 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_7 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_8 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_8 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP1_9 \"General amino acid permease AGP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP1_9 \"General amino acid permease AGP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP2 \"General amino acid permease AGP2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP2 \"General amino acid permease AGP2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP3_1 \"General amino acid permease AGP3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP3_1 \"General amino acid permease AGP3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP3_2 \"General amino acid permease AGP3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP3_2 \"General amino acid permease AGP3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AGP3_3 \"General amino acid permease AGP3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AGP3_3 \"General amino acid permease AGP3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALA1 \"Alanyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALA1 \"Alanyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALD2 \"Aldehyde dehydrogenase [NAD(P)+] 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALD2 \"Aldehyde dehydrogenase [NAD(P)+] 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALD3 \"Aldehyde dehydrogenase [NAD(P)+] 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALD3 \"Aldehyde dehydrogenase [NAD(P)+] 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALD4_1 \"Potassium-activated aldehyde dehydrogenase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALD4_1 \"Potassium-activated aldehyde dehydrogenase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALD4_2 \"Potassium-activated aldehyde dehydrogenase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALD4_2 \"Potassium-activated aldehyde dehydrogenase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALD5_1 \"Aldehyde dehydrogenase 5, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALD5_1 \"Aldehyde dehydrogenase 5, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALD5_2 \"Aldehyde dehydrogenase 5, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALD5_2 \"Aldehyde dehydrogenase 5, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALD6 \"Magnesium-activated aldehyde dehydrogenase, cytosolic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALD6 \"Magnesium-activated aldehyde dehydrogenase, cytosolic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALP1 \"Basic amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALP1 \"Basic amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AMD1_1 \"AMP deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AMD1_1 \"AMP deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AMD1_2 \"AMP deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AMD1_2 \"AMP deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AMD2_1 \"Probable amidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AMD2_1 \"Probable amidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AMD2_2 \"Probable amidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AMD2_2 \"Probable amidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AMD2_3 \"Probable amidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AMD2_3 \"Probable amidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_APA1_1 \"5',5'''-P-1,P-4-tetraphosphate phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_APA1_1 \"5',5'''-P-1,P-4-tetraphosphate phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_APA1_2 \"5',5'''-P-1,P-4-tetraphosphate phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_APA1_2 \"5',5'''-P-1,P-4-tetraphosphate phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_APA1_3 \"ADP-sulfurylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_APA1_3 \"ADP-sulfurylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_APA2 \"ATP adenylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_APA2 \"ATP adenylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_APT1 \"Adenine phosphoribosyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_APT1 \"Adenine phosphoribosyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_APT2 \"Adenine phosphoribosyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_APT2 \"Adenine phosphoribosyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARA1_1 \"D-arabinose dehydrogenase [NAD(P)+] heavy chain\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARA1_1 \"D-arabinose dehydrogenase [NAD(P)+] heavy chain\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARA1_2 \"D-arabinose dehydrogenase [NAD(P)+] heavy chain\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARA1_2 \"D-arabinose dehydrogenase [NAD(P)+] heavy chain\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARE1 \"Sterol O-acyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARE1 \"Sterol O-acyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARE2 \"Sterol O-acyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARE2 \"Sterol O-acyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARG1 \"Argininosuccinate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARG1 \"Argininosuccinate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARG3 \"Ornithine carbamoyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARG3 \"Ornithine carbamoyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARG4 \"Argininosuccinate lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARG4 \"Argininosuccinate lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARG5_1 \"Acetylglutamate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARG5_1 \"Acetylglutamate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARG5_2 \"N-acetyl-gamma-glutamyl-phosphate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARG5_2 \"N-acetyl-gamma-glutamyl-phosphate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARG8 \"Acetylornithine aminotransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARG8 \"Acetylornithine aminotransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO1_1 \"3-dehydroquinate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO1_1 \"3-dehydroquinate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO1_2 \"3-dehydroquinate dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO1_2 \"3-dehydroquinate dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO1_3 \"Shikimate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO1_3 \"Shikimate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO1_4 \"Shikimate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO1_4 \"Shikimate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO1_5 \"3-phosphoshikimate 1-carboxyvinyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO1_5 \"3-phosphoshikimate 1-carboxyvinyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO2 \"Chorismate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO2 \"Chorismate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO3 \"Phospho-2-keto-3-deoxyheptonate aldolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO3 \"Phospho-2-keto-3-deoxyheptonate aldolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO4 \"Phospho-2-keto-3-deoxyheptonate aldolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO4 \"Phospho-2-keto-3-deoxyheptonate aldolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO7 \"Chorismate mutase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO7 \"Chorismate mutase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO8 \"Aromatic amino acid aminotransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO8 \"Aromatic amino acid aminotransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO9_1 \"Aromatic amino acid aminotransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO9_1 \"Aromatic amino acid aminotransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARO9_2 \"Aromatic amino acid aminotransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARO9_2 \"Aromatic amino acid aminotransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASN1 \"Asparagine synthetase [glutamine-hydrolyzing] 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASN1 \"Asparagine synthetase [glutamine-hydrolyzing] 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASN2 \"Asparagine synthetase [glutamine-hydrolyzing] 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASN2 \"Asparagine synthetase [glutamine-hydrolyzing] 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASP1 \"L-asparaginase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASP1 \"L-asparaginase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASP31 \"L-asparaginase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASP31 \"L-asparaginase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASP32 \"L-asparaginase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASP32 \"L-asparaginase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASP33 \"L-asparaginase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASP33 \"L-asparaginase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASP34 \"L-asparaginase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASP34 \"L-asparaginase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ATH1 \"Vacuolar acid trehalase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ATH1 \"Vacuolar acid trehalase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ATP1 \"ATP synthase subunit alpha, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ATP1 \"ATP synthase subunit alpha, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUR1_1 \"Aureobasidin A resistance protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUR1_1 \"Aureobasidin A resistance protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUR1_2 \"Aureobasidin A resistance protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUR1_2 \"Aureobasidin A resistance protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUS1_1 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUS1_1 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUS1_2 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUS1_2 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUS1_3 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUS1_3 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUS1_4 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUS1_4 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUS1_5 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUS1_5 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUS1_6 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUS1_6 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUS1_7 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUS1_7 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AUS1_8 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AUS1_8 \"ATP-dependent permease AUS1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AYR1 \"NADPH-dependent 1-acyldihydroxyacetone phosphate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AYR1 \"NADPH-dependent 1-acyldihydroxyacetone phosphate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP2_1 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP2_1 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP2_2 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP2_2 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP2_3 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP2_3 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP2_4 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP2_4 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP2_5 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP2_5 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP2_6 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP2_6 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP2_7 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP2_7 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP2_8 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP2_8 \"Leu/Val/Ile amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP3_1 \"Valine amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP3_1 \"Valine amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP3_2 \"Valine amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP3_2 \"Valine amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP3_3 \"Valine amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP3_3 \"Valine amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP3_4 \"Valine amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP3_4 \"Valine amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP3_5 \"Valine amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP3_5 \"Valine amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP3_6 \"Valine amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP3_6 \"Valine amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP3_7 \"Valine amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP3_7 \"Valine amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAP3_8 \"Valine amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAP3_8 \"Valine amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAT1_1 \"Branched-chain-amino-acid aminotransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAT1_1 \"Branched-chain-amino-acid aminotransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAT1_2 \"Branched-chain-amino-acid aminotransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAT1_2 \"Branched-chain-amino-acid aminotransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAT2_1 \"Branched-chain-amino-acid aminotransferase, cytosolic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAT2_1 \"Branched-chain-amino-acid aminotransferase, cytosolic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAT2_2 \"Branched-chain-amino-acid aminotransferase, cytosolic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAT2_2 \"Branched-chain-amino-acid aminotransferase, cytosolic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BAT2_3 \"Branched-chain-amino-acid aminotransferase, cytosolic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BAT2_3 \"Branched-chain-amino-acid aminotransferase, cytosolic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BET2 \"Geranylgeranyl transferase type-2 subunit beta\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BET2 \"Geranylgeranyl transferase type-2 subunit beta\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BGL2 \"Glucan 1,3-beta-glucosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BGL2 \"Glucan 1,3-beta-glucosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BIO2 \"Biotin synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BIO2 \"Biotin synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BIO3 \"Adenosylmethionine-8-amino-7-oxononanoate aminotransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BIO3 \"Adenosylmethionine-8-amino-7-oxononanoate aminotransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BIO4 \"Dethiobiotin synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BIO4 \"Dethiobiotin synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BIO5 \"7-keto 8-aminopelargonic acid transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BIO5 \"7-keto 8-aminopelargonic acid transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BNA1 \"3-hydroxyanthranilate 3,4-dioxygenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BNA1 \"3-hydroxyanthranilate 3,4-dioxygenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BPH1 \"acetate transport via proton symport\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BPH1 \"acetate transport via proton symport\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CAN1_1 \"Arginine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CAN1_1 \"Arginine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CAN1_2 \"Arginine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CAN1_2 \"Arginine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CAR1 \"Arginase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CAR1 \"Arginase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CAR2 \"Ornithine aminotransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CAR2 \"Ornithine aminotransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CAT2 \"Carnitine O-acetyltransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CAT2 \"Carnitine O-acetyltransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDA1 \"Chitin deacetylase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDA1 \"Chitin deacetylase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDA2 \"Chitin deacetylase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDA2 \"Chitin deacetylase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDC19 \"Pyruvate kinase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDC19 \"Pyruvate kinase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDC21 \"Thymidylate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDC21 \"Thymidylate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDC60 \"Leucyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDC60 \"Leucyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDC8 \"Thymidylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDC8 \"Thymidylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDD1_1 \"Cytidine deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDD1_1 \"Cytidine deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDD1_2 \"Cytidine deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDD1_2 \"Cytidine deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDS1_1 \"Phosphatidate cytidylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDS1_1 \"Phosphatidate cytidylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CDS1_2 \"Phosphatidate cytidylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CDS1_2 \"Phosphatidate cytidylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CEM1_1 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CEM1_1 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CEM1_2 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CEM1_2 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CEM1_3 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CEM1_3 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CEM1_4 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CEM1_4 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CEM1_5 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CEM1_5 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CEM1_6 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CEM1_6 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CEM1_7 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CEM1_7 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CEM1_8 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CEM1_8 \"3-oxoacyl-[acyl-carrier-protein] synthase homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHA1_1 \"L-threonine dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHA1_1 \"L-threonine dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHA1_2 \"L-serine dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHA1_2 \"L-serine dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHO1_1 \"CDP-diacylglycerol--serine O-phosphatidyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHO1_1 \"CDP-diacylglycerol--serine O-phosphatidyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHO1_2 \"CDP-diacylglycerol--serine O-phosphatidyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHO1_2 \"CDP-diacylglycerol--serine O-phosphatidyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHO2 \"Phosphatidylethanolamine N-methyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHO2 \"Phosphatidylethanolamine N-methyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHS1 \"Chitin synthase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHS1 \"Chitin synthase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHS2 \"Chitin synthase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHS2 \"Chitin synthase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHS3 \"Chitin synthase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHS3 \"Chitin synthase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CIT1 \"Citrate synthase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CIT1 \"Citrate synthase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CIT2 \"Citrate synthase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CIT2 \"Citrate synthase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CIT3 \"Citrate synthase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CIT3 \"Citrate synthase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CKI1 \"Choline kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CKI1 \"Choline kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_COQ1 \"Hexaprenyl pyrophosphate synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_COQ1 \"Hexaprenyl pyrophosphate synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_COQ2 \"Para-hydroxybenzoate--polyprenyltransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_COQ2 \"Para-hydroxybenzoate--polyprenyltransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_COQ3 \"Hexaprenyldihydroxybenzoate methyltransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_COQ3 \"Hexaprenyldihydroxybenzoate methyltransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_COQ5 \"Ubiquinone biosynthesis methyltransferase COQ5, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_COQ5 \"Ubiquinone biosynthesis methyltransferase COQ5, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_COQ6 \"Ubiquinone biosynthesis monooxygenase COQ6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_COQ6 \"Ubiquinone biosynthesis monooxygenase COQ6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_COX1 \"Cytochrome c oxidase subunit 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_COX1 \"Cytochrome c oxidase subunit 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_COX10 \"Protoheme IX farnesyltransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_COX10 \"Protoheme IX farnesyltransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CPA2 \"Carbamoyl-phosphate synthase arginine-specific large chain\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CPA2 \"Carbamoyl-phosphate synthase arginine-specific large chain\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CPT1 \"Cholinephosphotransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CPT1 \"Cholinephosphotransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CRC1 \"Mitochondrial carnitine carrier\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CRC1 \"Mitochondrial carnitine carrier\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CRD1 \"Cardiolipin synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CRD1 \"Cardiolipin synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CSG2_1 \"Mannosyl phosphorylinositol ceramide synthase regulatory protein CSG2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CSG2_1 \"Mannosyl phosphorylinositol ceramide synthase regulatory protein CSG2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CSG2_2 \"Mannosyl phosphorylinositol ceramide synthase regulatory protein CSG2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CSG2_2 \"Mannosyl phosphorylinositol ceramide synthase regulatory protein CSG2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CTA1 \"Peroxisomal catalase A\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CTA1 \"Peroxisomal catalase A\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CTP1_1 \"Tricarboxylate transport protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CTP1_1 \"Tricarboxylate transport protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CTP1_2 \"Tricarboxylate transport protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CTP1_2 \"Tricarboxylate transport protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CTP1_3 \"Tricarboxylate transport protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CTP1_3 \"Tricarboxylate transport protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CTT1 \"Catalase T\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CTT1 \"Catalase T\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYB2 \"Cytochrome b2, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYB2 \"Cytochrome b2, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYR1 \"Adenylate cyclase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYR1 \"Adenylate cyclase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYS3 \"Cystathionine gamma-lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYS3 \"Cystathionine gamma-lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYS4 \"Cystathionine beta-synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYS4 \"Cystathionine beta-synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAK1 \"Dihydroxyacetone kinase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAK1 \"Dihydroxyacetone kinase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAK2 \"Dihydroxyacetone kinase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAK2 \"Dihydroxyacetone kinase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAL1 \"Allantoinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAL1 \"Allantoinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAL2 \"Allantoicase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAL2 \"Allantoicase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAL3 \"Ureidoglycolate hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAL3 \"Ureidoglycolate hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAL4 \"Allantoin permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAL4 \"Allantoin permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAL5 \"Allantoate permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAL5 \"Allantoate permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAL7 \"Malate synthase 2, glyoxysomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAL7 \"Malate synthase 2, glyoxysomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DCD1_1 \"Deoxycytidylate deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DCD1_1 \"Deoxycytidylate deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DCD1_2 \"Deoxycytidylate deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DCD1_2 \"Deoxycytidylate deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DCI1_1 \"Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DCI1_1 \"Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DCI1_2 \"Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DCI1_2 \"Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DCI1_3 \"Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DCI1_3 \"Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DCI1_4 \"Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DCI1_4 \"Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DED81 \"Asparaginyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DED81 \"Asparaginyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DEG1 \"tRNA pseudouridine synthase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DEG1 \"tRNA pseudouridine synthase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DFR1_1 \"Dihydrofolate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DFR1_1 \"Dihydrofolate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DFR1_2 \"Dihydrofolate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DFR1_2 \"Dihydrofolate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DGA1 \"Diacylglycerol O-acyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DGA1 \"Diacylglycerol O-acyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIA1 \"Uncharacterized seryl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIA1 \"Uncharacterized seryl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIC1_1 \"Mitochondrial dicarboxylate transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIC1_1 \"Mitochondrial dicarboxylate transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIC1_2 \"Mitochondrial dicarboxylate transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIC1_2 \"Mitochondrial dicarboxylate transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIC1_3 \"Mitochondrial dicarboxylate transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIC1_3 \"Mitochondrial dicarboxylate transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIM1 \"Dimethyladenosine transferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIM1 \"Dimethyladenosine transferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIP5_1 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIP5_1 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIP5_2 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIP5_2 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIP5_3 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIP5_3 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIP5_4 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIP5_4 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIP5_5 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIP5_5 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIP5_6 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIP5_6 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIP5_7 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIP5_7 \"Dicarboxylic amino acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DLD1 \"D-lactate dehydrogenase [cytochrome] 1, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DLD1 \"D-lactate dehydrogenase [cytochrome] 1, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DPH5 \"Diphthine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DPH5 \"Diphthine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DPL1_1 \"Sphingosine-1-phosphate lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DPL1_1 \"Sphingosine-1-phosphate lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DPL1_2 \"Sphingosine-1-phosphate lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DPL1_2 \"Sphingosine-1-phosphate lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DPM1 \"Dolichol-phosphate mannosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DPM1 \"Dolichol-phosphate mannosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DPP1 \"Diacylglycerol pyrophosphate phosphatase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DPP1 \"Diacylglycerol pyrophosphate phosphatase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DPS1 \"Aspartyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DPS1 \"Aspartyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DUR1_1 \"Urea carboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DUR1_1 \"Urea carboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DUR1_2 \"Allophanate hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DUR1_2 \"Allophanate hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DUR3 \"Urea active transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DUR3 \"Urea active transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DUT1 \"Deoxyuridine 5'-triphosphate nucleotidohydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DUT1 \"Deoxyuridine 5'-triphosphate nucleotidohydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DYS1 \"Deoxyhypusine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DYS1 \"Deoxyhypusine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ECI1_1 \"3,2-trans-enoyl-CoA isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ECI1_1 \"3,2-trans-enoyl-CoA isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ECI1_2 \"3,2-trans-enoyl-CoA isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ECI1_2 \"3,2-trans-enoyl-CoA isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ECI1_3 \"3,2-trans-enoyl-CoA isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ECI1_3 \"3,2-trans-enoyl-CoA isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ECM17 \"Sulfite reductase [NADPH] subunit beta\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ECM17 \"Sulfite reductase [NADPH] subunit beta\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ECM31 \"3-methyl-2-oxobutanoate hydroxymethyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ECM31 \"3-methyl-2-oxobutanoate hydroxymethyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ECM38 \"Gamma-glutamyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ECM38 \"Gamma-glutamyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ECM40_1 \"Amino-acid acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ECM40_1 \"Amino-acid acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ECM40_2 \"Glutamate N-acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ECM40_2 \"Glutamate N-acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_EKI1 \"Ethanolamine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_EKI1 \"Ethanolamine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO1_1 \"Elongation of fatty acids protein 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO1_1 \"Elongation of fatty acids protein 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO1_2 \"Elongation of fatty acids protein 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO1_2 \"Elongation of fatty acids protein 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO2_3 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO2_3 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO2_4 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO2_4 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO2_5 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO2_5 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO2_6 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO2_6 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO2_7 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO2_7 \"Elongation of fatty acids protein 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO3_3 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO3_3 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO3_4 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO3_4 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO3_5 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO3_5 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO3_6 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO3_6 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ELO3_7 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ELO3_7 \"Elongation of fatty acids protein 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ENO1 \"Enolase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ENO1 \"Enolase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ENO2 \"Enolase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ENO2 \"Enolase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_EPT1 \"Choline/ethanolaminephosphotransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_EPT1 \"Choline/ethanolaminephosphotransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG1 \"Squalene monooxygenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG1 \"Squalene monooxygenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG10 \"Acetyl-CoA acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG10 \"Acetyl-CoA acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG11 \"Cytochrome P450 51\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG11 \"Cytochrome P450 51\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG12_1 \"Mevalonate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG12_1 \"Mevalonate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG12_2 \"Mevalonate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG12_2 \"Mevalonate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG12_3 \"Mevalonate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG12_3 \"Mevalonate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG12_4 \"Mevalonate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG12_4 \"Mevalonate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG2 \"C-8 sterol isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG2 \"C-8 sterol isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG20_1 \"Dimethylallyltranstransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG20_1 \"Dimethylallyltranstransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG20_2 \"Geranyltranstransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG20_2 \"Geranyltranstransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG24 \"C-14 sterol reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG24 \"C-14 sterol reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG25_1 \"C-4 methylsterol oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG25_1 \"C-4 methylsterol oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG25_2 \"C-4 methylsterol oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG25_2 \"C-4 methylsterol oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG26_1 \"Sterol-4-alpha-carboxylate 3-dehydrogenase, decarboxylating\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG26_1 \"Sterol-4-alpha-carboxylate 3-dehydrogenase, decarboxylating\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG26_2 \"Sterol-4-alpha-carboxylate 3-dehydrogenase, decarboxylating\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG26_2 \"Sterol-4-alpha-carboxylate 3-dehydrogenase, decarboxylating\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG27_1 \"3-keto-steroid reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG27_1 \"3-keto-steroid reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG27_2 \"3-keto-steroid reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG27_2 \"3-keto-steroid reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG3 \"C-5 sterol desaturase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG3 \"C-5 sterol desaturase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG4 \"Delta(24(24(1)))-sterol reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG4 \"Delta(24(24(1)))-sterol reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG5 \"Cytochrome P450 61\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG5 \"Cytochrome P450 61\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG6 \"Sterol 24-C-methyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG6 \"Sterol 24-C-methyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG7 \"Lanosterol synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG7 \"Lanosterol synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG8 \"Phosphomevalonate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG8 \"Phosphomevalonate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG9_1 \"Squalene synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG9_1 \"Squalene synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG9_2 \"Squalene synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG9_2 \"Squalene synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERR1_1 \"Enolase-related protein 1/2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERR1_1 \"Enolase-related protein 1/2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERR1_2 \"Enolase-related protein 1/2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERR1_2 \"Enolase-related protein 1/2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERR2 \"Enolase-related protein 1/2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERR2 \"Enolase-related protein 1/2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETR1_1 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETR1_1 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETR1_2 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETR1_2 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETR1_3 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETR1_3 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETR1_4 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETR1_4 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETR1_5 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETR1_5 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETR1_6 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETR1_6 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETR1_7 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETR1_7 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETR1_8 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETR1_8 \"Enoyl-[acyl-carrier protein] reductase [NADPH, B-specific], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_EXG1 \"Glucan 1,3-beta-glucosidase I/II\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_EXG1 \"Glucan 1,3-beta-glucosidase I/II\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_EXG2 \"Glucan 1,3-beta-glucosidase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_EXG2 \"Glucan 1,3-beta-glucosidase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA1_1 \"Long-chain-fatty-acid--CoA ligase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA1_1 \"Long-chain-fatty-acid--CoA ligase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA1_2 \"Long-chain-fatty-acid--CoA ligase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA1_2 \"Long-chain-fatty-acid--CoA ligase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA1_3 \"Long-chain-fatty-acid--CoA ligase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA1_3 \"Long-chain-fatty-acid--CoA ligase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA2_1 \"Long-chain-fatty-acid--CoA ligase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA2_1 \"Long-chain-fatty-acid--CoA ligase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA2_2 \"Long-chain-fatty-acid--CoA ligase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA2_2 \"Long-chain-fatty-acid--CoA ligase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA3_1 \"Long-chain-fatty-acid--CoA ligase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA3_1 \"Long-chain-fatty-acid--CoA ligase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA3_2 \"Long-chain-fatty-acid--CoA ligase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA3_2 \"Long-chain-fatty-acid--CoA ligase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA4_1 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA4_1 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA4_2 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA4_2 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA4_3 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA4_3 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA4_4 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA4_4 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAA4_5 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAA4_5 \"Long-chain-fatty-acid--CoA ligase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAB1 \"1-phosphatidylinositol-3-phosphate 5-kinase FAB1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAB1 \"1-phosphatidylinositol-3-phosphate 5-kinase FAB1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAD1 \"FAD synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAD1 \"FAD synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_1 \"[Acyl-carrier-protein] malonyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_1 \"[Acyl-carrier-protein] malonyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_1_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_1_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_1_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_1_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_2_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_2_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_2_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_2_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_3_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_3_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_3_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_3_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_4_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_4_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_4_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_4_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_4_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_4_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_4_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_4_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_5_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_5_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_5_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_5_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_5_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_5_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_5_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_5_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_6_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_6_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_6_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_6_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_6_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_6_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_6_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_6_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_7_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_7_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_7_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_7_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_7_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_7_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_7_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_7_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_8_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_8_1 \"3-hydroxyacyl-[acyl-carrier-protein] dehydrase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_8_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_8_2 \"Enoyl-[acyl-carrier-protein] reductase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_8_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_8_c \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_8_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_8_f \"Acyl-[acyl-carrier-protein] hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS1_s \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS1_s \"[Acyl-carrier-protein] acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_1_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_1_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_1_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_1_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_2_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_2_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_2_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_2_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_3_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_3_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_3_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_3_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_4_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_4_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_4_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_4_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_5_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_5_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_5_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_5_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_6_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_6_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_6_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_6_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_7_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_7_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_7_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_7_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_8_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_8_1 \"3-oxoacyl-[acyl-carrier-protein] synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAS2_8_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAS2_8_2 \"3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAT1_1 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAT1_1 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAT1_2 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAT1_2 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAT1_3 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAT1_3 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAT1_4 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAT1_4 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAT1_5 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAT1_5 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAT1_6 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAT1_6 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FAT1_7 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FAT1_7 \"Very long-chain fatty acyl-CoA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FATP \"Model-specific reaction, used to affect ATP levels by converting ADT to ATP\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FATP \"Model-specific reaction, used to affect ATP levels by converting ADT to ATP\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FBA1 \"Fructose-bisphosphate aldolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FBA1 \"Fructose-bisphosphate aldolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FBP1 \"Fructose-1,6-bisphosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FBP1 \"Fructose-1,6-bisphosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FBP26 \"Fructose-2,6-bisphosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FBP26 \"Fructose-2,6-bisphosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY1 \"Cytosine deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY1 \"Cytosine deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY2_1 \"Purine-cytosine permease FCY2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY2_1 \"Purine-cytosine permease FCY2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY2_2 \"Purine-cytosine permease FCY2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY2_2 \"Purine-cytosine permease FCY2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY2_3 \"Purine-cytosine permease FCY2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY2_3 \"Purine-cytosine permease FCY2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY21_1 \"Purine-cytosine permease FCY21\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY21_1 \"Purine-cytosine permease FCY21\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY21_2 \"Purine-cytosine permease FCY21\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY21_2 \"Purine-cytosine permease FCY21\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY21_3 \"Purine-cytosine permease FCY21\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY21_3 \"Purine-cytosine permease FCY21\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY22_1 \"Purine-cytosine permease FCY22\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY22_1 \"Purine-cytosine permease FCY22\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY22_2 \"Purine-cytosine permease FCY22\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY22_2 \"Purine-cytosine permease FCY22\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCY22_3 \"Purine-cytosine permease FCY22\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCY22_3 \"Purine-cytosine permease FCY22\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FDH1 \"Formate dehydrogenase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FDH1 \"Formate dehydrogenase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FEN2 \"Pantothenate transporter FEN2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FEN2 \"Pantothenate transporter FEN2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FIG4 \"Polyphosphoinositide phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FIG4 \"Polyphosphoinositide phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FKS1 \"1,3-beta-D-glucan-UDP glucosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FKS1 \"1,3-beta-D-glucan-UDP glucosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FKS3 \"1,3-beta-glucan synthase component FKS3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FKS3 \"1,3-beta-glucan synthase component FKS3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FLX1 \"Mitochondrial FAD carrier protein FLX1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FLX1 \"Mitochondrial FAD carrier protein FLX1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FMN1_1 \"Riboflavin kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FMN1_1 \"Riboflavin kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FMN1_2 \"Riboflavin kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FMN1_2 \"Riboflavin kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FMT1 \"Methionyl-tRNA formyltransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FMT1 \"Methionyl-tRNA formyltransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FNADH \"Model-specific reaction, used to affect NADH levels by converting NAD+ to NADH\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FNADH \"Model-specific reaction, used to affect NADH levels by converting NAD+ to NADH\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FNADPH \"Model-specific reaction, used to affect NADPH levels by converting NADP+ to NADPH\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FNADPH \"Model-specific reaction, used to affect NADPH levels by converting NADP+ to NADPH\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOL1_1 \"Dihydroneopterin aldolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOL1_1 \"Dihydroneopterin aldolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOL1_2 \"2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOL1_2 \"2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOL1_3 \"Dihydropteroate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOL1_3 \"Dihydropteroate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOL1_4 \"Dihydropteroate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOL1_4 \"Dihydropteroate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOL2 \"GTP cyclohydrolase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOL2 \"GTP cyclohydrolase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOL3 \"Folylpolyglutamate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOL3 \"Folylpolyglutamate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX1_1 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX1_1 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX1_2 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX1_2 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX1_3 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX1_3 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX1_4 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX1_4 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX1_5 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX1_5 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX1_6 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX1_6 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX1_7 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX1_7 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX1_8 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX1_8 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX2_1 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX2_1 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX2_2 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX2_2 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX2_3 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX2_3 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX2_4 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX2_4 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX2_5 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX2_5 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX2_6 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX2_6 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX2_7 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX2_7 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FOX2_8 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FOX2_8 \"D-3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FPS1 \"Glycerol uptake/efflux facilitator protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FPS1 \"Glycerol uptake/efflux facilitator protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FRS1 \"Phenylalanyl-tRNA synthetase beta chain\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FRS1 \"Phenylalanyl-tRNA synthetase beta chain\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FRS2 \"Phenylalanyl-tRNA synthetase alpha chain\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FRS2 \"Phenylalanyl-tRNA synthetase alpha chain\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FSP2_1 \"Probable alpha-glucosidase FSP2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FSP2_1 \"Probable alpha-glucosidase FSP2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FSP2_2 \"Probable alpha-glucosidase FSP2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FSP2_2 \"Probable alpha-glucosidase FSP2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUI1_1 \"Uridine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUI1_1 \"Uridine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUI1_2 \"Uridine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUI1_2 \"Uridine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUI1_3 \"Uridine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUI1_3 \"Uridine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUM1_1 \"Fumarate hydratase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUM1_1 \"Fumarate hydratase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUM1_2 \"Fumarate hydratase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUM1_2 \"Fumarate hydratase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUN63 \"Putative inosine-5'-monophosphate dehydrogenase IMD1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUN63 \"Putative inosine-5'-monophosphate dehydrogenase IMD1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUR1 \"Uracil phosphoribosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUR1 \"Uracil phosphoribosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUR4 \"Uracil permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUR4 \"Uracil permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAD1 \"Glutamate decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAD1 \"Glutamate decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAL1 \"Galactokinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAL1 \"Galactokinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAL10 \"UDP-glucose 4-epimerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAL10 \"UDP-glucose 4-epimerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAL2_1 \"Galactose transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAL2_1 \"Galactose transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAL2_2 \"Galactose transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAL2_2 \"Galactose transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAL7_1 \"Galactose-1-phosphate uridylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAL7_1 \"Galactose-1-phosphate uridylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAL7_2 \"Galactose-1-phosphate uridylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAL7_2 \"Galactose-1-phosphate uridylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_1 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_1 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_10 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_10 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_11 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_11 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_12 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_12 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_13 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_13 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_14 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_14 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_15 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_15 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_16 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_16 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_17 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_17 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_18 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_18 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_19 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_19 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_2 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_2 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_20 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_20 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_21 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_21 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_3 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_3 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_4 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_4 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_5 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_5 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_6 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_6 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_7 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_7 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_8 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_8 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GAP1_9 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GAP1_9 \"General amino-acid permease GAP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GCV1_1 \"Aminomethyltransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GCV1_1 \"Aminomethyltransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GCV1_2 \"Aminomethyltransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GCV1_2 \"Aminomethyltransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GCV2 \"Glycine dehydrogenase [decarboxylating], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GCV2 \"Glycine dehydrogenase [decarboxylating], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GDE1 \"Glycerophosphodiester phosphodiesterase GDE1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GDE1 \"Glycerophosphodiester phosphodiesterase GDE1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GDH1 \"NADP-specific glutamate dehydrogenase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GDH1 \"NADP-specific glutamate dehydrogenase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GDH2 \"NAD-specific glutamate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GDH2 \"NAD-specific glutamate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GDH3 \"NADP-specific glutamate dehydrogenase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GDH3 \"NADP-specific glutamate dehydrogenase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GFA1 \"Glucosamine--fructose-6-phosphate aminotransferase [isomerizing]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GFA1 \"Glucosamine--fructose-6-phosphate aminotransferase [isomerizing]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GIT1_1 \"Glycerophosphoinositol plasma membrane permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GIT1_1 \"Glycerophosphoinositol plasma membrane permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GIT1_2 \"Glycerophosphoinositol plasma membrane permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GIT1_2 \"Glycerophosphoinositol plasma membrane permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLC3 \"1,4-alpha-glucan-branching enzyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLC3 \"1,4-alpha-glucan-branching enzyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLK1_1 \"Glucokinase GLK1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLK1_1 \"Glucokinase GLK1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLK1_2 \"Glucokinase GLK1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLK1_2 \"Glucokinase GLK1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLK1_3 \"Glucokinase GLK1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLK1_3 \"Glucokinase GLK1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLN1 \"Glutamine synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLN1 \"Glutamine synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLN4 \"Glutaminyl-tRNA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLN4 \"Glutaminyl-tRNA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLO1 \"Lactoylglutathione lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLO1 \"Lactoylglutathione lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLO2 \"Hydroxyacylglutathione hydrolase, cytoplasmic isozyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLO2 \"Hydroxyacylglutathione hydrolase, cytoplasmic isozyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLO4 \"Hydroxyacylglutathione hydrolase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLO4 \"Hydroxyacylglutathione hydrolase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLR1 \"Glutathione reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLR1 \"Glutathione reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLT1 \"Glutamate synthase [NADH]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLT1 \"Glutamate synthase [NADH]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLY1 \"Low specificity L-threonine aldolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLY1 \"Low specificity L-threonine aldolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNA1 \"Glucosamine 6-phosphate N-acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNA1 \"Glucosamine 6-phosphate N-acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GND1 \"6-phosphogluconate dehydrogenase, decarboxylating 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GND1 \"6-phosphogluconate dehydrogenase, decarboxylating 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GND2 \"6-phosphogluconate dehydrogenase, decarboxylating 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GND2 \"6-phosphogluconate dehydrogenase, decarboxylating 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNP1_1 \"High-affinity glutamine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNP1_1 \"High-affinity glutamine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNP1_2 \"High-affinity glutamine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNP1_2 \"High-affinity glutamine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNP1_3 \"High-affinity glutamine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNP1_3 \"High-affinity glutamine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNP1_4 \"High-affinity glutamine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNP1_4 \"High-affinity glutamine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNP1_5 \"High-affinity glutamine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNP1_5 \"High-affinity glutamine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNP1_6 \"High-affinity glutamine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNP1_6 \"High-affinity glutamine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNP1_7 \"High-affinity glutamine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNP1_7 \"High-affinity glutamine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPD1 \"Glycerol-3-phosphate dehydrogenase [NAD+] 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPD1 \"Glycerol-3-phosphate dehydrogenase [NAD+] 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPD2 \"Glycerol-3-phosphate dehydrogenase [NAD+] 2, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPD2 \"Glycerol-3-phosphate dehydrogenase [NAD+] 2, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPH1 \"Glycogen phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPH1 \"Glycogen phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPM1_1 \"Phosphoglycerate mutase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPM1_1 \"Phosphoglycerate mutase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPM1_2 \"Phosphoglycerate mutase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPM1_2 \"Phosphoglycerate mutase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPM2 \"Phosphoglycerate mutase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPM2 \"Phosphoglycerate mutase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPM3 \"Phosphoglycerate mutase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPM3 \"Phosphoglycerate mutase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPT2_1 \"Glycerol-3-phosphate O-acyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPT2_1 \"Glycerol-3-phosphate O-acyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPT2_2 \"Dihydroxyacetone phosphate acyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPT2_2 \"Dihydroxyacetone phosphate acyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPX1 \"Glutathione peroxidase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPX1 \"Glutathione peroxidase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GPX2 \"Glutathione peroxidase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GPX2 \"Glutathione peroxidase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GRS1 \"Glycyl-tRNA synthetase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GRS1 \"Glycyl-tRNA synthetase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GRS2 \"Glycyl-tRNA synthetase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GRS2 \"Glycyl-tRNA synthetase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GSC2 \"1,3-beta-glucan synthase component GSC2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GSC2 \"1,3-beta-glucan synthase component GSC2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GSH1 \"Glutamate--cysteine ligase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GSH1 \"Glutamate--cysteine ligase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GSH2 \"Glutathione synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GSH2 \"Glutathione synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GSY1 \"Glycogen [starch] synthase isoform 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GSY1 \"Glycogen [starch] synthase isoform 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GSY2 \"Glycogen [starch] synthase isoform 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GSY2 \"Glycogen [starch] synthase isoform 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GUA1 \"GMP synthase [glutamine-hydrolyzing]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GUA1 \"GMP synthase [glutamine-hydrolyzing]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GUK1_1 \"Guanylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GUK1_1 \"Guanylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GUK1_2 \"Guanylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GUK1_2 \"Guanylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GUK1_3 \"Guanylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GUK1_3 \"Guanylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GUT1 \"Glycerol kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GUT1 \"Glycerol kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GUT2 \"Glycerol-3-phosphate dehydrogenase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GUT2 \"Glycerol-3-phosphate dehydrogenase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEM1 \"5-aminolevulinate synthase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEM1 \"5-aminolevulinate synthase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEM12 \"Uroporphyrinogen decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEM12 \"Uroporphyrinogen decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEM13 \"Coproporphyrinogen III oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEM13 \"Coproporphyrinogen III oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEM14 \"Protoporphyrinogen oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEM14 \"Protoporphyrinogen oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEM15 \"Ferrochelatase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEM15 \"Ferrochelatase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEM2 \"Porphobilinogen synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEM2 \"Porphobilinogen synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEM3 \"Porphobilinogen deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEM3 \"Porphobilinogen deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEM4 \"Uroporphyrinogen-III synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEM4 \"Uroporphyrinogen-III synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HFA1 \"Acetyl-CoA carboxylase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HFA1 \"Acetyl-CoA carboxylase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIP1 \"Histidine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIP1 \"Histidine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS1 \"ATP phosphoribosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS1 \"ATP phosphoribosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS2 \"Histidinol-phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS2 \"Histidinol-phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS3 \"Imidazoleglycerol-phosphate dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS3 \"Imidazoleglycerol-phosphate dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS4_1 \"Phosphoribosyl-ATP pyrophosphohydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS4_1 \"Phosphoribosyl-ATP pyrophosphohydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS4_2 \"Phosphoribosyl-AMP cyclohydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS4_2 \"Phosphoribosyl-AMP cyclohydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS4_3 \"Histidinol dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS4_3 \"Histidinol dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS5 \"Histidinol-phosphate aminotransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS5 \"Histidinol-phosphate aminotransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS6 \"1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino] imidazole-4-carboxamide isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS6 \"1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino] imidazole-4-carboxamide isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HIS7 \"Imidazole glycerol phosphate synthase hisHF\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HIS7 \"Imidazole glycerol phosphate synthase hisHF\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HMG1 \"3-hydroxy-3-methylglutaryl-coenzyme A reductase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HMG1 \"3-hydroxy-3-methylglutaryl-coenzyme A reductase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HMG2 \"3-hydroxy-3-methylglutaryl-coenzyme A reductase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HMG2 \"3-hydroxy-3-methylglutaryl-coenzyme A reductase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HMGS \"Hydroxymethylglutaryl-CoA synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HMGS \"Hydroxymethylglutaryl-CoA synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HMT1 \"HNRNP arginine N-methyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HMT1 \"HNRNP arginine N-methyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HNM1 \"Choline transport protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HNM1 \"Choline transport protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HOM2 \"Aspartate-semialdehyde dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HOM2 \"Aspartate-semialdehyde dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HOM3 \"Aspartokinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HOM3 \"Aspartokinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HOM6_1 \"Homoserine dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HOM6_1 \"Homoserine dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HOM6_2 \"Homoserine dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HOM6_2 \"Homoserine dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HOR2 \"(DL)-glycerol-3-phosphatase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HOR2 \"(DL)-glycerol-3-phosphatase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HPT1_1 \"Hypoxanthine-guanine phosphoribosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HPT1_1 \"Hypoxanthine-guanine phosphoribosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HPT1_2 \"Hypoxanthine-guanine phosphoribosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HPT1_2 \"Hypoxanthine-guanine phosphoribosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTD2_1 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTD2_1 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTD2_2 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTD2_2 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTD2_3 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTD2_3 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTD2_4 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTD2_4 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTD2_5 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTD2_5 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTD2_6 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTD2_6 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTD2_7 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTD2_7 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTD2_8 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTD2_8 \"3-hydroxyacyl-thioester dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HTS1 \"Histidyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HTS1 \"Histidyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXK1_1 \"Hexokinase-1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXK1_1 \"Hexokinase-1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXK1_2 \"Hexokinase-1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXK1_2 \"Hexokinase-1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXK1_3 \"Hexokinase-1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXK1_3 \"Hexokinase-1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXK1_4 \"Hexokinase-1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXK1_4 \"Hexokinase-1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXK2_1 \"Hexokinase-2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXK2_1 \"Hexokinase-2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXK2_2 \"Hexokinase-2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXK2_2 \"Hexokinase-2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXK2_3 \"Hexokinase-2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXK2_3 \"Hexokinase-2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXK2_4 \"Hexokinase-2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXK2_4 \"Hexokinase-2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT1_1 \"Low-affinity glucose transporter HXT1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT1_1 \"Low-affinity glucose transporter HXT1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT1_2 \"Low-affinity glucose transporter HXT1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT1_2 \"Low-affinity glucose transporter HXT1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT1_3 \"Low-affinity glucose transporter HXT1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT1_3 \"Low-affinity glucose transporter HXT1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT10_1 \"Hexose transporter HXT10\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT10_1 \"Hexose transporter HXT10\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT10_2 \"Hexose transporter HXT10\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT10_2 \"Hexose transporter HXT10\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT10_3 \"Hexose transporter HXT10\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT10_3 \"Hexose transporter HXT10\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT10_4 \"Hexose transporter HXT10\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT10_4 \"Hexose transporter HXT10\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT11_1 \"Hexose transporter HXT11\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT11_1 \"Hexose transporter HXT11\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT11_2 \"Hexose transporter HXT11\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT11_2 \"Hexose transporter HXT11\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT11_3 \"Hexose transporter HXT11\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT11_3 \"Hexose transporter HXT11\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT11_4 \"Hexose transporter HXT11\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT11_4 \"Hexose transporter HXT11\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT11_5 \"Hexose transporter HXT11\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT11_5 \"Hexose transporter HXT11\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT13_1 \"Hexose transporter HXT13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT13_1 \"Hexose transporter HXT13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT13_2 \"Hexose transporter HXT13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT13_2 \"Hexose transporter HXT13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT13_3 \"Hexose transporter HXT13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT13_3 \"Hexose transporter HXT13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT14 \"Hexose transporter HXT14\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT14 \"Hexose transporter HXT14\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT15_1 \"Hexose transporter HXT15\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT15_1 \"Hexose transporter HXT15\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT15_2 \"Hexose transporter HXT15\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT15_2 \"Hexose transporter HXT15\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT15_3 \"Hexose transporter HXT15\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT15_3 \"Hexose transporter HXT15\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT16_1 \"Hexose transporter HXT16\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT16_1 \"Hexose transporter HXT16\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT16_2 \"Hexose transporter HXT16\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT16_2 \"Hexose transporter HXT16\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT16_3 \"Hexose transporter HXT16\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT16_3 \"Hexose transporter HXT16\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT17_1 \"Hexose transporter HXT17\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT17_1 \"Hexose transporter HXT17\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT17_2 \"Hexose transporter HXT17\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT17_2 \"Hexose transporter HXT17\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT17_3 \"Hexose transporter HXT17\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT17_3 \"Hexose transporter HXT17\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT2_1 \"High-affinity glucose transporter HXT2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT2_1 \"High-affinity glucose transporter HXT2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT2_2 \"High-affinity glucose transporter HXT2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT2_2 \"High-affinity glucose transporter HXT2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT2_3 \"High-affinity glucose transporter HXT2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT2_3 \"High-affinity glucose transporter HXT2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT3_1 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT3_1 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT3_2 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT3_2 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT3_3 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT3_3 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT4_1 \"Low-affinity glucose transporter HXT4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT4_1 \"Low-affinity glucose transporter HXT4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT4_2 \"Low-affinity glucose transporter HXT4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT4_2 \"Low-affinity glucose transporter HXT4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT4_3 \"Low-affinity glucose transporter HXT4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT4_3 \"Low-affinity glucose transporter HXT4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT4_4 \"Low-affinity glucose transporter HXT4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT4_4 \"Low-affinity glucose transporter HXT4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT5_1 \"Probable glucose transporter HXT5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT5_1 \"Probable glucose transporter HXT5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT5_2 \"Probable glucose transporter HXT5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT5_2 \"Probable glucose transporter HXT5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT5_3 \"Probable glucose transporter HXT5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT5_3 \"Probable glucose transporter HXT5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT6_1 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT6_1 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT6_2 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT6_2 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT6_3 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT6_3 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT7_1 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT7_1 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT7_2 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT7_2 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT7_3 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT7_3 \"High-affinity hexose transporter HXT6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT8_1 \"Hexose transporter HXT8\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT8_1 \"Hexose transporter HXT8\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT8_2 \"Hexose transporter HXT8\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT8_2 \"Hexose transporter HXT8\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT8_3 \"Hexose transporter HXT8\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT8_3 \"Hexose transporter HXT8\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT9_1 \"Hexose transporter HXT9\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT9_1 \"Hexose transporter HXT9\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT9_2 \"Hexose transporter HXT9\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT9_2 \"Hexose transporter HXT9\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT9_3 \"Hexose transporter HXT9\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT9_3 \"Hexose transporter HXT9\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HXT9_4 \"Hexose transporter HXT9\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HXT9_4 \"Hexose transporter HXT9\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HYR1 \"Peroxiredoxin HYR1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HYR1 \"Peroxiredoxin HYR1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ICL1 \"Isocitrate lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ICL1 \"Isocitrate lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ICL2 \"Mitochondrial 2-methylisocitrate lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ICL2 \"Mitochondrial 2-methylisocitrate lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IDH1 \"Isocitrate dehydrogenase [NAD] subunit 1, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IDH1 \"Isocitrate dehydrogenase [NAD] subunit 1, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IDI1 \"Isopentenyl-diphosphate Delta-isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IDI1 \"Isopentenyl-diphosphate Delta-isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IDP1_1 \"Isocitrate dehydrogenase [NADP], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IDP1_1 \"Isocitrate dehydrogenase [NADP], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IDP1_2 \"Isocitrate dehydrogenase [NADP], mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IDP1_2 \"Isocitrate dehydrogenase [NADP], mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IDP2_1 \"Isocitrate dehydrogenase [NADP], cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IDP2_1 \"Isocitrate dehydrogenase [NADP], cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IDP2_2 \"Isocitrate dehydrogenase [NADP], cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IDP2_2 \"Isocitrate dehydrogenase [NADP], cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IDP3_1 \"Isocitrate dehydrogenase [NADP]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IDP3_1 \"Isocitrate dehydrogenase [NADP]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IDP3_2 \"Isocitrate dehydrogenase [NADP]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IDP3_2 \"Isocitrate dehydrogenase [NADP]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IFA38_1 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IFA38_1 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IFA38_2 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IFA38_2 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IFA38_3 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IFA38_3 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IFA38_4 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IFA38_4 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IFA38_5 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IFA38_5 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IFA38_6 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IFA38_6 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IFA38_7 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IFA38_7 \"Microsomal beta-keto-reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILS1 \"Isoleucyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILS1 \"Isoleucyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV1 \"Threonine dehydratase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV1 \"Threonine dehydratase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV2_1 \"Acetolactate synthase catalytic subunit, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV2_1 \"Acetolactate synthase catalytic subunit, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV2_2 \"Acetolactate synthase catalytic subunit, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV2_2 \"Acetolactate synthase catalytic subunit, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV3_1 \"Dihydroxy-acid dehydratase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV3_1 \"Dihydroxy-acid dehydratase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV3_2 \"Dihydroxy-acid dehydratase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV3_2 \"Dihydroxy-acid dehydratase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV3_3 \"Dihydroxy-acid dehydratase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV3_3 \"Dihydroxy-acid dehydratase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV5_1 \"Ketol-acid reductoisomerase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV5_1 \"Ketol-acid reductoisomerase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV5_2 \"Ketol-acid reductoisomerase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV5_2 \"Ketol-acid reductoisomerase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILV5_3 \"Ketol-acid reductoisomerase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILV5_3 \"Ketol-acid reductoisomerase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IMD3 \"Probable inosine-5'-monophosphate dehydrogenase IMD3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IMD3 \"Probable inosine-5'-monophosphate dehydrogenase IMD3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IMD4 \"Probable inosine-5'-monophosphate dehydrogenase IMD4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IMD4 \"Probable inosine-5'-monophosphate dehydrogenase IMD4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INM1 \"Inositol monophosphatase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INM1 \"Inositol monophosphatase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INO1 \"Inositol-3-phosphate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INO1 \"Inositol-3-phosphate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP51 \"Phosphatidylinositol 4,5-bisphosphate 5-phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP51 \"Phosphatidylinositol 4,5-bisphosphate 5-phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP52_1 \"Inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP52_1 \"Inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP52_2 \"Inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP52_2 \"Inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP52_3 \"Inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP52_3 \"Inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP52_4 \"Inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP52_4 \"Inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP53_1 \"Inositol-1,4,5-trisphosphate 5-phosphatase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP53_1 \"Inositol-1,4,5-trisphosphate 5-phosphatase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP53_2 \"Inositol-1,4,5-trisphosphate 5-phosphatase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP53_2 \"Inositol-1,4,5-trisphosphate 5-phosphatase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP53_3 \"Inositol-1,4,5-trisphosphate 5-phosphatase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP53_3 \"Inositol-1,4,5-trisphosphate 5-phosphatase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP53_4 \"Inositol-1,4,5-trisphosphate 5-phosphatase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP53_4 \"Inositol-1,4,5-trisphosphate 5-phosphatase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INP54 \"Inositol-1,4,5-trisphosphate 5-phosphatase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INP54 \"Inositol-1,4,5-trisphosphate 5-phosphatase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IPK1 \"Inositol-pentakisphosphate 2-kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IPK1 \"Inositol-pentakisphosphate 2-kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IPP1 \"Inorganic pyrophosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IPP1 \"Inorganic pyrophosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IPT1_1 \"Inositolphosphotransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IPT1_1 \"Inositolphosphotransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_IPT1_2 \"Inositolphosphotransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_IPT1_2 \"Inositolphosphotransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ISC1_1 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ISC1_1 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ISC1_2_1 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ISC1_2_1 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ISC1_2_2 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ISC1_2_2 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ISC1_3_1 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ISC1_3_1 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ISC1_3_2 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ISC1_3_2 \"Inositol phosphosphingolipids phospholipase C\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ISM1 \"Isoleucyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ISM1 \"Isoleucyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ITR1 \"Myo-inositol transporter 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ITR1 \"Myo-inositol transporter 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ITR2 \"Myo-inositol transporter 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ITR2 \"Myo-inositol transporter 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_JEN1_1 \"Carboxylic acid transporter protein homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_JEN1_1 \"Carboxylic acid transporter protein homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_JEN1_2 \"Carboxylic acid transporter protein homolog\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_JEN1_2 \"Carboxylic acid transporter protein homolog\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KGD1 \"Alpha-ketoglutarate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KGD1 \"Alpha-ketoglutarate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KRE2 \"Glycolipid 2-alpha-mannosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KRE2 \"Glycolipid 2-alpha-mannosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KRS1 \"Lysyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KRS1 \"Lysyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KTR1 \"Alpha-1,2 mannosyltransferase KTR1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KTR1 \"Alpha-1,2 mannosyltransferase KTR1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KTR2 \"Probable mannosyltransferase KTR2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KTR2 \"Probable mannosyltransferase KTR2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KTR3 \"Probable mannosyltransferase KTR3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KTR3 \"Probable mannosyltransferase KTR3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KTR4 \"Probable mannosyltransferase KTR4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KTR4 \"Probable mannosyltransferase KTR4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KTR6 \"Mannosyltransferase KTR6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KTR6 \"Mannosyltransferase KTR6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LAC1_1 \"Sphingosine N-acyltransferase LAC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LAC1_1 \"Sphingosine N-acyltransferase LAC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LAC1_2 \"Sphingosine N-acyltransferase LAC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LAC1_2 \"Sphingosine N-acyltransferase LAC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LAG1_1 \"Sphingosine N-acyltransferase LAG1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LAG1_1 \"Sphingosine N-acyltransferase LAG1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LAG1_2 \"Sphingosine N-acyltransferase LAG1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LAG1_2 \"Sphingosine N-acyltransferase LAG1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LAT1 \"Dihydrolipoyllysine-residue acetyltransferase component of pyruvate dehydrogenase complex, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LAT1 \"Dihydrolipoyllysine-residue acetyltransferase component of pyruvate dehydrogenase complex, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LCB1 \"Serine palmitoyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LCB1 \"Serine palmitoyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LCB2 \"Serine palmitoyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LCB2 \"Serine palmitoyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LCB3_1 \"Sphingosine-1-phosphate phosphatase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LCB3_1 \"Sphingosine-1-phosphate phosphatase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LCB3_2 \"Sphingosine-1-phosphate phosphatase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LCB3_2 \"Sphingosine-1-phosphate phosphatase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LCB4_1 \"Sphingoid long chain base kinase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LCB4_1 \"Sphingoid long chain base kinase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LCB4_2 \"Sphingoid long chain base kinase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LCB4_2 \"Sphingoid long chain base kinase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LCB5_1 \"Sphingoid long chain base kinase 5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LCB5_1 \"Sphingoid long chain base kinase 5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LCB5_2 \"Sphingoid long chain base kinase 5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LCB5_2 \"Sphingoid long chain base kinase 5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LEU1_1 \"3-isopropylmalate dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LEU1_1 \"3-isopropylmalate dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LEU1_2 \"3-isopropylmalate dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LEU1_2 \"3-isopropylmalate dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LEU2 \"3-isopropylmalate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LEU2 \"3-isopropylmalate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LEU4 \"2-isopropylmalate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LEU4 \"2-isopropylmalate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LongAcyl \"Model-specific reaction, involved in sphingoglycolipid metabolism\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LongAcyl \"Model-specific reaction, involved in sphingoglycolipid metabolism\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LPD1 \"Dihydrolipoyl dehydrogenase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LPD1 \"Dihydrolipoyl dehydrogenase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LPP1 \"Lipid phosphate phosphatase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LPP1 \"Lipid phosphate phosphatase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LRO1 \"Phospholipid:diacylglycerol acyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LRO1 \"Phospholipid:diacylglycerol acyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LSB6 \"Phosphatidylinositol 4-kinase LSB6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LSB6 \"Phosphatidylinositol 4-kinase LSB6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LSC1 \"Succinyl-CoA ligase [ADP-forming] subunit alpha, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LSC1 \"Succinyl-CoA ligase [ADP-forming] subunit alpha, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LSC2 \"Succinyl-CoA ligase [ADP-forming] subunit beta, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LSC2 \"Succinyl-CoA ligase [ADP-forming] subunit beta, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYP1 \"Lysine-specific permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYP1 \"Lysine-specific permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS1 \"Saccharopine dehydrogenase [NAD+, L-lysine-forming]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS1 \"Saccharopine dehydrogenase [NAD+, L-lysine-forming]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS12 \"Homoisocitrate dehydrogenase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS12 \"Homoisocitrate dehydrogenase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS2_1 \"L-aminoadipate-semialdehyde dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS2_1 \"L-aminoadipate-semialdehyde dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS2_2 \"L-aminoadipate-semialdehyde dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS2_2 \"L-aminoadipate-semialdehyde dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS20_1 \"Homocitrate synthase, cytosolic isozyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS20_1 \"Homocitrate synthase, cytosolic isozyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS20_2 \"Homocitrate synthase, cytosolic isozyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS20_2 \"Homocitrate synthase, cytosolic isozyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS21 \"Homocitrate synthase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS21 \"Homocitrate synthase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS4 \"Homoaconitase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS4 \"Homoaconitase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYS9 \"Saccharopine dehydrogenase [NADP+, L-glutamate-forming]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYS9 \"Saccharopine dehydrogenase [NADP+, L-glutamate-forming]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MAE1 \"NAD-dependent malic enzyme, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MAE1 \"NAD-dependent malic enzyme, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MAL11 \"General alpha-glucoside permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MAL11 \"General alpha-glucoside permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MAL12 \"Alpha-glucosidase MAL12\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MAL12 \"Alpha-glucosidase MAL12\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MAL31 \"Maltose permease MAL31\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MAL31 \"Maltose permease MAL31\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MAL32 \"Alpha-glucosidase MAL32\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MAL32 \"Alpha-glucosidase MAL32\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MCT1 \"Malonyl CoA-acyl carrier protein transacylase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MCT1 \"Malonyl CoA-acyl carrier protein transacylase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MDH1 \"Malate dehydrogenase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MDH1 \"Malate dehydrogenase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MDH2 \"Malate dehydrogenase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MDH2 \"Malate dehydrogenase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MDH3 \"Malate dehydrogenase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MDH3 \"Malate dehydrogenase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MEP1 \"Ammonium transporter MEP1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MEP1 \"Ammonium transporter MEP1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MEP2 \"Ammonium transporter MEP2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MEP2 \"Ammonium transporter MEP2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MEP3 \"Ammonium transporter MEP3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MEP3 \"Ammonium transporter MEP3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MES1 \"Methionyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MES1 \"Methionyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET1 \"Uroporphyrinogen-III C-methyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET1 \"Uroporphyrinogen-III C-methyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET10 \"Sulfite reductase [NADPH] flavoprotein component\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET10 \"Sulfite reductase [NADPH] flavoprotein component\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET12 \"Methylenetetrahydrofolate reductase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET12 \"Methylenetetrahydrofolate reductase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET13 \"Methylenetetrahydrofolate reductase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET13 \"Methylenetetrahydrofolate reductase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET14 \"Adenylyl-sulfate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET14 \"Adenylyl-sulfate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET16 \"Phosphoadenosine phosphosulfate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET16 \"Phosphoadenosine phosphosulfate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET17_1 \"O-acetylhomoserine sulfhydrylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET17_1 \"O-acetylhomoserine sulfhydrylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET17_2 \"O-acetylserine sulfhydrylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET17_2 \"O-acetylserine sulfhydrylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET17_3 \"O-acetylserine sulfhydrylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET17_3 \"O-acetylserine sulfhydrylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET2 \"Homoserine O-acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET2 \"Homoserine O-acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET22 \"3'(2'),5'-bisphosphate nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET22 \"3'(2'),5'-bisphosphate nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET3 \"Sulfate adenylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET3 \"Sulfate adenylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET6 \"5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET6 \"5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MET7 \"Folylpolyglutamate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MET7 \"Folylpolyglutamate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MHT1 \"Homocysteine S-methyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MHT1 \"Homocysteine S-methyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MIR1 \"Mitochondrial phosphate carrier protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MIR1 \"Mitochondrial phosphate carrier protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MIS1_1 \"Methylenetetrahydrofolate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MIS1_1 \"Methylenetetrahydrofolate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MIS1_2 \"Formyltetrahydrofolate synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MIS1_2 \"Formyltetrahydrofolate synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MIS1_3 \"Methenyltetrahydrofolate cyclohydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MIS1_3 \"Methenyltetrahydrofolate cyclohydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MLS1 \"Malate synthase 1, glyoxysomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MLS1 \"Malate synthase 1, glyoxysomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MMP1 \"S-methylmethionine permease 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MMP1 \"S-methylmethionine permease 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MPH2 \"Alpha-glucosides permease MPH2/3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MPH2 \"Alpha-glucosides permease MPH2/3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MPH3 \"Alpha-glucosides permease MPH2/3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MPH3 \"Alpha-glucosides permease MPH2/3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSD1 \"Aspartyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSD1 \"Aspartyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSE1 \"Glutamyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSE1 \"Glutamyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSF1 \"Phenylalanyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSF1 \"Phenylalanyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSK1 \"Lysyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSK1 \"Lysyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSM1 \"Methionyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSM1 \"Methionyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSR1 \"Arginyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSR1 \"Arginyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSS4 \"Probable phosphatidylinositol-4-phosphate 5-kinase MSS4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSS4 \"Probable phosphatidylinositol-4-phosphate 5-kinase MSS4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MST1 \"Threonyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MST1 \"Threonyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSW1 \"Tryptophanyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSW1 \"Tryptophanyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MSY1 \"Tyrosyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MSY1 \"Tyrosyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MTD1 \"Methylenetetrahydrofolate dehydrogenase [NAD+]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MTD1 \"Methylenetetrahydrofolate dehydrogenase [NAD+]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MUP1 \"High-affinity methionine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MUP1 \"High-affinity methionine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MUP3 \"Low-affinity methionine permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MUP3 \"Low-affinity methionine permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MUQ1 \"Ethanolamine-phosphate cytidylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MUQ1 \"Ethanolamine-phosphate cytidylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MVD1 \"Diphosphomevalonate decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MVD1 \"Diphosphomevalonate decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NAM2 \"Leucyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NAM2 \"Leucyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NAT1 \"N-terminal acetyltransferase A complex subunit NAT1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NAT1 \"N-terminal acetyltransferase A complex subunit NAT1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NAT2 \"N-terminal acetyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NAT2 \"N-terminal acetyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NCP1 \"NADPH--cytochrome P450 reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NCP1 \"NADPH--cytochrome P450 reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NDH1 \"External NADH-ubiquinone oxidoreductase 1, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NDH1 \"External NADH-ubiquinone oxidoreductase 1, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NDH2 \"External NADH-ubiquinone oxidoreductase 2, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NDH2 \"External NADH-ubiquinone oxidoreductase 2, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NDI1 \"NADH-ubiquinone oxidoreductase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NDI1 \"NADH-ubiquinone oxidoreductase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NHA1 \"Na(+)/H(+) antiporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NHA1 \"Na(+)/H(+) antiporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NIT2_1 \"Probable hydrolase NIT2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NIT2_1 \"Probable hydrolase NIT2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NIT2_2 \"Probable hydrolase NIT2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NIT2_2 \"Probable hydrolase NIT2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NIT2_3 \"Probable hydrolase NIT2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NIT2_3 \"Probable hydrolase NIT2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NMT1 \"Glycylpeptide N-tetradecanoyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NMT1 \"Glycylpeptide N-tetradecanoyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NPT1_1 \"Nicotinate phosphoribosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NPT1_1 \"Nicotinate phosphoribosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NPT1_2 \"Nicotinate phosphoribosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NPT1_2 \"Nicotinate phosphoribosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NTE1 \"Lysophospholipase NTE1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NTE1 \"Lysophospholipase NTE1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NTH1 \"Neutral trehalase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NTH1 \"Neutral trehalase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NTH2 \"Probable trehalase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NTH2 \"Probable trehalase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAC1 \"Mitochondrial oxaloacetate transport protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAC1 \"Mitochondrial oxaloacetate transport protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAR1_1 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAR1_1 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAR1_2 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAR1_2 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAR1_3 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAR1_3 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAR1_4 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAR1_4 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAR1_5 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAR1_5 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAR1_6 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAR1_6 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAR1_7 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAR1_7 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OAR1_8 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OAR1_8 \"Mitochondrial 3-oxoacyl-[acyl-carrier-protein] reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ODC1 \"Mitochondrial 2-oxodicarboxylate carrier 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ODC1 \"Mitochondrial 2-oxodicarboxylate carrier 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ODC2 \"Mitochondrial 2-oxodicarboxylate carrier 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ODC2 \"Mitochondrial 2-oxodicarboxylate carrier 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OLE1_c14 \"Acyl-CoA desaturase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OLE1_c14 \"Acyl-CoA desaturase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OLE1_c16 \"Acyl-CoA desaturase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OLE1_c16 \"Acyl-CoA desaturase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OLE1_c18 \"Acyl-CoA desaturase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OLE1_c18 \"Acyl-CoA desaturase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OPI3_1 \"Methylene-fatty-acyl-phospholipid synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OPI3_1 \"Methylene-fatty-acyl-phospholipid synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OPI3_2 \"Methylene-fatty-acyl-phospholipid synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OPI3_2 \"Methylene-fatty-acyl-phospholipid synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ORT1 \"Mitochondrial ornithine carrier protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ORT1 \"Mitochondrial ornithine carrier protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OSM1 \"Fumarate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OSM1 \"Fumarate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PAD1 \"Phenylacrylic acid decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PAD1 \"Phenylacrylic acid decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PAN5 \"2-dehydropantoate 2-reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PAN5 \"2-dehydropantoate 2-reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PCK1 \"Phosphoenolpyruvate carboxykinase [ATP]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PCK1 \"Phosphoenolpyruvate carboxykinase [ATP]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PCM1_1 \"Phosphoacetylglucosamine mutase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PCM1_1 \"Phosphoacetylglucosamine mutase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PCM1_2 \"Phosphoacetylglucosamine mutase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PCM1_2 \"Phosphoacetylglucosamine mutase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PCT1 \"Choline-phosphate cytidylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PCT1 \"Choline-phosphate cytidylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDA1 \"Pyruvate dehydrogenase E1 component subunit alpha, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDA1 \"Pyruvate dehydrogenase E1 component subunit alpha, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDC1 \"Pyruvate decarboxylase isozyme 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDC1 \"Pyruvate decarboxylase isozyme 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDC5 \"Pyruvate decarboxylase isozyme 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDC5 \"Pyruvate decarboxylase isozyme 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDC6 \"Pyruvate decarboxylase isozyme 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDC6 \"Pyruvate decarboxylase isozyme 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDE1 \"3',5'-cyclic-nucleotide phosphodiesterase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDE1 \"3',5'-cyclic-nucleotide phosphodiesterase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDE2_1 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDE2_1 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDE2_2 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDE2_2 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDE2_3 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDE2_3 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDE2_4 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDE2_4 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDE2_5 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDE2_5 \"3',5'-cyclic-nucleotide phosphodiesterase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDX3_1 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDX3_1 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDX3_2 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDX3_2 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDX3_3 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDX3_3 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDX3_4 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDX3_4 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PDX3_5 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PDX3_5 \"Pyridoxamine 5'-phosphate oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PET9 \"ADP, ATP carrier protein 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PET9 \"ADP, ATP carrier protein 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PFK1_1 \"Phosphofructokinase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PFK1_1 \"Phosphofructokinase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PFK1_2 \"Phosphofructokinase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PFK1_2 \"Phosphofructokinase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PFK1_3 \"Phosphofructokinase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PFK1_3 \"Phosphofructokinase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PFK2 \"Phosphofructokinase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PFK2 \"Phosphofructokinase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PFK26 \"6-phosphofructo-2-kinase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PFK26 \"6-phosphofructo-2-kinase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PFK27 \"6-phosphofructo-2-kinase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PFK27 \"6-phosphofructo-2-kinase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGI1_1 \"Glucose-6-phosphate isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGI1_1 \"Glucose-6-phosphate isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGI1_2 \"Glucose-6-phosphate isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGI1_2 \"Glucose-6-phosphate isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGI1_3 \"Glucose-6-phosphate isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGI1_3 \"Glucose-6-phosphate isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGK1 \"Phosphoglycerate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGK1 \"Phosphoglycerate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGM1_1 \"Phosphoglucomutase-1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGM1_1 \"Phosphoglucomutase-1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGM1_2 \"Phosphoglucomutase-1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGM1_2 \"Phosphoglucomutase-1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGM2_1 \"Phosphoglucomutase-2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGM2_1 \"Phosphoglucomutase-2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGM2_2 \"Phosphoglucomutase-2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGM2_2 \"Phosphoglucomutase-2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PGS1 \"CDP-diacylglycerol--glycerol-3-phosphate 3-phosphatidyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PGS1 \"CDP-diacylglycerol--glycerol-3-phosphate 3-phosphatidyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PHA2 \"Prephenate dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PHA2 \"Prephenate dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PHO11 \"Acid phosphatase PHO11\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PHO11 \"Acid phosphatase PHO11\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PHO8 \"Alkaline phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PHO8 \"Alkaline phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PHO84 \"Inorganic phosphate transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PHO84 \"Inorganic phosphate transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PHO90 \"Low-affinity phosphate transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PHO90 \"Low-affinity phosphate transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PHO91 \"Low-affinity phosphate transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PHO91 \"Low-affinity phosphate transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PIK1 \"Phosphatidylinositol 4-kinase PIK1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PIK1 \"Phosphatidylinositol 4-kinase PIK1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PIS1 \"CDP-diacylglycerol--inositol 3-phosphatidyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PIS1 \"CDP-diacylglycerol--inositol 3-phosphatidyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PLB1 \"Lysophospholipase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PLB1 \"Lysophospholipase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PLB2 \"Lysophospholipase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PLB2 \"Lysophospholipase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PLB3 \"Lysophospholipase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PLB3 \"Lysophospholipase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PLC1 \"1-phosphatidylinositol-4,5-bisphosphate phosphodiesterase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PLC1 \"1-phosphatidylinositol-4,5-bisphosphate phosphodiesterase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMA1 \"Plasma membrane ATPase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMA1 \"Plasma membrane ATPase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMA2 \"Plasma membrane ATPase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMA2 \"Plasma membrane ATPase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMI40 \"Mannose-6-phosphate isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMI40 \"Mannose-6-phosphate isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMP1 \"Plasma membrane ATPase proteolipid 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMP1 \"Plasma membrane ATPase proteolipid 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMP2 \"Plasma membrane ATPase proteolipid 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMP2 \"Plasma membrane ATPase proteolipid 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMT1 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMT1 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMT2 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMT2 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMT3 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMT3 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMT4 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMT4 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMT5 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMT5 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PMT6 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 6\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PMT6 \"Dolichyl-phosphate-mannose--protein mannosyltransferase 6\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNC1_1 \"Nicotinamidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNC1_1 \"Nicotinamidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNC1_2 \"Nicotinamidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNC1_2 \"Nicotinamidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_1 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_1 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_10 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_10 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_11 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_11 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_2 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_2 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_3 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_3 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_4 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_4 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_5 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_5 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_6 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_6 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_7 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_7 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_8 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_8 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNP1_9 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNP1_9 \"Purine nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Pool_Acyl1 \"Model-specific reaction, pools acyl-CoAs\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Pool_Acyl1 \"Model-specific reaction, pools acyl-CoAs\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Pool_Acyl2 \"Model-specific reaction, pools acyl-CoAs\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Pool_Acyl2 \"Model-specific reaction, pools acyl-CoAs\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POS5_1 \"NADH kinase POS5, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POS5_1 \"NADH kinase POS5, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POS5_2 \"NADH kinase POS5, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POS5_2 \"NADH kinase POS5, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POT1_1 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POT1_1 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POT1_2 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POT1_2 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POT1_3 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POT1_3 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POT1_4 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POT1_4 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POT1_5 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POT1_5 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POT1_6 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POT1_6 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POT1_7 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POT1_7 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POT1_8 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POT1_8 \"3-ketoacyl-CoA thiolase, peroxisomal\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_1 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_1 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_10 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_10 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_11 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_11 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_12 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_12 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_2 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_2 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_3 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_3 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_4 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_4 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_5 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_5 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_6 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_6 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_7 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_7 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_8 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_8 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_POX1_9 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_POX1_9 \"Acyl-coenzyme A oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PPA2 \"Inorganic pyrophosphatase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PPA2 \"Inorganic pyrophosphatase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRO1 \"Glutamate 5-kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRO1 \"Glutamate 5-kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRO2_1 \"Gamma-glutamyl phosphate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRO2_1 \"Gamma-glutamyl phosphate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRO2_2 \"Gamma-glutamyl phosphate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRO2_2 \"Gamma-glutamyl phosphate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRO3_1 \"Pyrroline-5-carboxylate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRO3_1 \"Pyrroline-5-carboxylate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRO3_2 \"Pyrroline-5-carboxylate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRO3_2 \"Pyrroline-5-carboxylate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRO3_3 \"Pyrroline-5-carboxylate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRO3_3 \"Pyrroline-5-carboxylate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRS1 \"Ribose-phosphate pyrophosphokinase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRS1 \"Ribose-phosphate pyrophosphokinase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRS2 \"Ribose-phosphate pyrophosphokinase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRS2 \"Ribose-phosphate pyrophosphokinase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRS3 \"Ribose-phosphate pyrophosphokinase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRS3 \"Ribose-phosphate pyrophosphokinase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRS4 \"Ribose-phosphate pyrophosphokinase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRS4 \"Ribose-phosphate pyrophosphokinase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PRS5 \"Ribose-phosphate pyrophosphokinase 5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PRS5 \"Ribose-phosphate pyrophosphokinase 5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PSA1 \"Mannose-1-phosphate guanyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PSA1 \"Mannose-1-phosphate guanyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PSD1 \"Phosphatidylserine decarboxylase proenzyme 1, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PSD1 \"Phosphatidylserine decarboxylase proenzyme 1, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PSD2 \"Phosphatidylserine decarboxylase proenzyme 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PSD2 \"Phosphatidylserine decarboxylase proenzyme 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PTR2_1 \"Peptide transporter PTR2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PTR2_1 \"Peptide transporter PTR2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PTR2_2 \"Peptide transporter PTR2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PTR2_2 \"Peptide transporter PTR2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PTR2_3 \"Peptide transporter PTR2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PTR2_3 \"Peptide transporter PTR2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUR5 \"Inosine-5'-monophosphate dehydrogenase IMD2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUR5 \"Inosine-5'-monophosphate dehydrogenase IMD2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUS1 \"tRNA pseudouridine synthase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUS1 \"tRNA pseudouridine synthase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUS2 \"tRNA pseudouridine synthase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUS2 \"tRNA pseudouridine synthase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUS4 \"tRNA pseudouridine synthase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUS4 \"tRNA pseudouridine synthase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUT1 \"Proline oxidase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUT1 \"Proline oxidase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUT2 \"Delta-1-pyrroline-5-carboxylate dehydrogenase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUT2 \"Delta-1-pyrroline-5-carboxylate dehydrogenase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUT4_1 \"Proline-specific permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUT4_1 \"Proline-specific permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUT4_2 \"Proline-specific permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUT4_2 \"Proline-specific permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUT4_3 \"Proline-specific permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUT4_3 \"Proline-specific permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PUT4_4 \"Proline-specific permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PUT4_4 \"Proline-specific permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PYC1 \"Pyruvate carboxylase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PYC1 \"Pyruvate carboxylase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PYC2 \"Pyruvate carboxylase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PYC2 \"Pyruvate carboxylase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PYK2 \"Pyruvate kinase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PYK2 \"Pyruvate kinase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PYRD \"Dihydroorotate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PYRD \"Dihydroorotate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_QNS1_1 \"Glutamine-dependent NAD(+) synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_QNS1_1 \"Glutamine-dependent NAD(+) synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_QNS1_2 \"Glutamine-dependent NAD(+) synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_QNS1_2 \"Glutamine-dependent NAD(+) synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_QPT1_1 \"Nicotinate-nucleotide pyrophosphorylase [carboxylating]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_QPT1_1 \"Nicotinate-nucleotide pyrophosphorylase [carboxylating]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_QPT1_2 \"Nicotinate-nucleotide pyrophosphorylase [carboxylating]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_QPT1_2 \"Nicotinate-nucleotide pyrophosphorylase [carboxylating]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_QRI1 \"UDP-N-acetylglucosamine pyrophosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_QRI1 \"UDP-N-acetylglucosamine pyrophosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RAM1 \"Protein farnesyltransferase subunit beta\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RAM1 \"Protein farnesyltransferase subunit beta\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RBK1_1 \"Probable ribokinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RBK1_1 \"Probable ribokinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RBK1_2 \"Probable ribokinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RBK1_2 \"Probable ribokinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RER2 \"Dehydrodolichyl diphosphate synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RER2 \"Dehydrodolichyl diphosphate synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RHR2 \"(DL)-glycerol-3-phosphatase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RHR2 \"(DL)-glycerol-3-phosphatase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RIB1 \"GTP cyclohydrolase-2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RIB1 \"GTP cyclohydrolase-2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RIB5 \"Riboflavin synthase alpha chain\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RIB5 \"Riboflavin synthase alpha chain\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RIB7_1 \"5-amino-6-(5-phosphoribosylamino)uracil reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RIB7_1 \"5-amino-6-(5-phosphoribosylamino)uracil reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RIB7_2 \"5-amino-6-(5-phosphoribosylamino)uracil reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RIB7_2 \"5-amino-6-(5-phosphoribosylamino)uracil reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RIP1 \"Cytochrome b-c1 complex subunit Rieske, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RIP1 \"Cytochrome b-c1 complex subunit Rieske, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RKI1 \"Ribose-5-phosphate isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RKI1 \"Ribose-5-phosphate isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RMA1 \"Probable folylpolyglutamate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RMA1 \"Probable folylpolyglutamate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RNR1_1 \"Ribonucleoside-diphosphate reductase large chain 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RNR1_1 \"Ribonucleoside-diphosphate reductase large chain 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RNR1_2 \"Ribonucleoside-diphosphate reductase large chain 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RNR1_2 \"Ribonucleoside-diphosphate reductase large chain 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RNR1_3 \"Ribonucleoside-diphosphate reductase large chain 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RNR1_3 \"Ribonucleoside-diphosphate reductase large chain 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RNR1_4 \"Ribonucleoside-diphosphate reductase large chain 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RNR1_4 \"Ribonucleoside-diphosphate reductase large chain 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RNR3 \"Ribonucleoside-diphosphate reductase large chain 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RNR3 \"Ribonucleoside-diphosphate reductase large chain 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RPE1 \"Ribulose-phosphate 3-epimerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RPE1 \"Ribulose-phosphate 3-epimerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAC1_1 \"Phosphoinositide phosphatase SAC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAC1_1 \"Phosphoinositide phosphatase SAC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAC1_2 \"Phosphoinositide phosphatase SAC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAC1_2 \"Phosphoinositide phosphatase SAC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAC1_3 \"Phosphoinositide phosphatase SAC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAC1_3 \"Phosphoinositide phosphatase SAC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAH1 \"Adenosylhomocysteinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAH1 \"Adenosylhomocysteinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAM1 \"S-adenosylmethionine synthetase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAM1 \"S-adenosylmethionine synthetase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAM2 \"S-adenosylmethionine synthetase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAM2 \"S-adenosylmethionine synthetase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAM3 \"S-adenosylmethionine permease SAM3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAM3 \"S-adenosylmethionine permease SAM3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAM4 \"Homocysteine S-methyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAM4 \"Homocysteine S-methyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SCS7 \"Inositolphosphorylceramide-B C-26 hydroxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SCS7 \"Inositolphosphorylceramide-B C-26 hydroxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SCT1_1 \"Glycerol-3-phosphate O-acyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SCT1_1 \"Glycerol-3-phosphate O-acyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SCT1_2 \"Dihydroxyacetone phosphate acyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SCT1_2 \"Dihydroxyacetone phosphate acyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SDH3_1 \"Succinate dehydrogenase [ubiquinone] cytochrome b subunit, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SDH3_1 \"Succinate dehydrogenase [ubiquinone] cytochrome b subunit, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SDH3_2 \"Succinate dehydrogenase [ubiquinone] cytochrome b subunit, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SDH3_2 \"Succinate dehydrogenase [ubiquinone] cytochrome b subunit, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SEC53 \"Phosphomannomutase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SEC53 \"Phosphomannomutase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SEC59 \"Dolichol kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SEC59 \"Dolichol kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SER1_1 \"Phosphoserine aminotransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SER1_1 \"Phosphoserine aminotransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SER1_2 \"Phosphoserine aminotransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SER1_2 \"Phosphoserine aminotransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SER2 \"Phosphoserine phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SER2 \"Phosphoserine phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SER3 \"D-3-phosphoglycerate dehydrogenase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SER3 \"D-3-phosphoglycerate dehydrogenase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SER33 \"D-3-phosphoglycerate dehydrogenase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SER33 \"D-3-phosphoglycerate dehydrogenase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SES1 \"Seryl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SES1 \"Seryl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SFA1_1 \"S-(hydroxymethyl)glutathione dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SFA1_1 \"S-(hydroxymethyl)glutathione dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SFA1_2 \"S-(hydroxymethyl)glutathione dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SFA1_2 \"S-(hydroxymethyl)glutathione dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SFC1 \"Succinate/fumarate mitochondrial transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SFC1 \"Succinate/fumarate mitochondrial transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SHM1 \"Serine hydroxymethyltransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SHM1 \"Serine hydroxymethyltransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SHM2 \"Serine hydroxymethyltransferase, cytosolic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SHM2 \"Serine hydroxymethyltransferase, cytosolic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SLC1 \"Probable 1-acyl-sn-glycerol-3-phosphate acyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SLC1 \"Probable 1-acyl-sn-glycerol-3-phosphate acyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SOL1 \"Probable 6-phosphogluconolactonase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SOL1 \"Probable 6-phosphogluconolactonase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SOL2 \"Probable 6-phosphogluconolactonase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SOL2 \"Probable 6-phosphogluconolactonase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SOL3 \"Probable 6-phosphogluconolactonase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SOL3 \"Probable 6-phosphogluconolactonase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SOL4 \"Probable 6-phosphogluconolactonase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SOL4 \"Probable 6-phosphogluconolactonase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SOR1 \"Sorbitol dehydrogenase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SOR1 \"Sorbitol dehydrogenase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPE1 \"Ornithine decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPE1 \"Ornithine decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPE2 \"S-adenosylmethionine decarboxylase proenzyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPE2 \"S-adenosylmethionine decarboxylase proenzyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPE3 \"Spermidine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPE3 \"Spermidine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPE4 \"Spermine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPE4 \"Spermine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPO14 \"Phospholipase D1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPO14 \"Phospholipase D1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPO22 \"Sporulation-specific protein 22\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPO22 \"Sporulation-specific protein 22\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPR1 \"Glucan 1,3-beta-glucosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPR1 \"Glucan 1,3-beta-glucosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPS19_1 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPS19_1 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPS19_2 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPS19_2 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPS19_3 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPS19_3 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPS19_4 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPS19_4 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPS19_5 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPS19_5 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPS19_6 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPS19_6 \"Peroxisomal 2,4-dienoyl-CoA reductase SPS19\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SRT1 \"Cis-prenyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SRT1 \"Cis-prenyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_STL1_1 \"Sugar transporter STL1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_STL1_1 \"Sugar transporter STL1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_STL1_2 \"Sugar transporter STL1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_STL1_2 \"Sugar transporter STL1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_STL1_3 \"Sugar transporter STL1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_STL1_3 \"Sugar transporter STL1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_STL1_4 \"Sugar transporter STL1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_STL1_4 \"Sugar transporter STL1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_STT4 \"Phosphatidylinositol 4-kinase STT4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_STT4 \"Phosphatidylinositol 4-kinase STT4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUC2 \"Invertase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUC2 \"Invertase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUL1 \"Sulfate permease 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUL1 \"Sulfate permease 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUL2 \"Sulfate permease 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUL2 \"Sulfate permease 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUR1_1 \"Mannosyl phosphorylinositol ceramide synthase SUR1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUR1_1 \"Mannosyl phosphorylinositol ceramide synthase SUR1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUR1_2 \"Mannosyl phosphorylinositol ceramide synthase SUR1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUR1_2 \"Mannosyl phosphorylinositol ceramide synthase SUR1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUR2 \"Sphingolipid C4-hydroxylase SUR2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUR2 \"Sphingolipid C4-hydroxylase SUR2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAL1 \"Transaldolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAL1 \"Transaldolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT1_1 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT1_1 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT1_2 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT1_2 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT1_3 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT1_3 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT1_4 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT1_4 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT1_5 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT1_5 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT2_1 \"Tryptophan permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT2_1 \"Tryptophan permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT2_2 \"Tryptophan permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT2_2 \"Tryptophan permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT2_3 \"Tryptophan permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT2_3 \"Tryptophan permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT2_4 \"Tryptophan permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT2_4 \"Tryptophan permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT2_5 \"Tryptophan permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT2_5 \"Tryptophan permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TAT2_6 \"Tryptophan permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TAT2_6 \"Tryptophan permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TDH1 \"Glyceraldehyde-3-phosphate dehydrogenase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TDH1 \"Glyceraldehyde-3-phosphate dehydrogenase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TDH2 \"Glyceraldehyde-3-phosphate dehydrogenase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TDH2 \"Glyceraldehyde-3-phosphate dehydrogenase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TDH3 \"Glyceraldehyde-3-phosphate dehydrogenase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TDH3 \"Glyceraldehyde-3-phosphate dehydrogenase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TES1 \"Peroxisomal acyl-coenzyme A thioester hydrolase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TES1 \"Peroxisomal acyl-coenzyme A thioester hydrolase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TGL1 \"Sterol esterase TGL1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TGL1 \"Sterol esterase TGL1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TGL2 \"Lipase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TGL2 \"Lipase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TGL3 \"Lipase 3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TGL3 \"Lipase 3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TGL4 \"Lipase 4\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TGL4 \"Lipase 4\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TGL5 \"Lipase 5\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TGL5 \"Lipase 5\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI20_1 \"Phosphomethylpyrimidine kinase THI20\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI20_1 \"Phosphomethylpyrimidine kinase THI20\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI20_2 \"Phosphomethylpyrimidine kinase THI20\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI20_2 \"Phosphomethylpyrimidine kinase THI20\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI21 \"Phosphomethylpyrimidine kinase THI21\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI21 \"Phosphomethylpyrimidine kinase THI21\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI22 \"Thiamine biosynthesis protein THI22\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI22 \"Thiamine biosynthesis protein THI22\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI6_1 \"Hydroxyethylthiazole kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI6_1 \"Hydroxyethylthiazole kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI6_2 \"Thiamine-phosphate pyrophosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI6_2 \"Thiamine-phosphate pyrophosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI7 \"Thiamine transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI7 \"Thiamine transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI80_1 \"Thiamine pyrophosphokinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI80_1 \"Thiamine pyrophosphokinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THI80_2 \"Thiamine pyrophosphokinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THI80_2 \"Thiamine pyrophosphokinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THR1 \"Homoserine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THR1 \"Homoserine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THR4_1 \"Threonine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THR4_1 \"Threonine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THR4_2 \"Threonine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THR4_2 \"Threonine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THS1 \"Threonyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THS1 \"Threonyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TKL1_1 \"Transketolase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TKL1_1 \"Transketolase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TKL1_2 \"Transketolase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TKL1_2 \"Transketolase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TKL2_1 \"Transketolase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TKL2_1 \"Transketolase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TKL2_2 \"Transketolase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TKL2_2 \"Transketolase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TOR1 \"Phosphatidylinositol kinase homolog TOR1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TOR1 \"Phosphatidylinositol kinase homolog TOR1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TOR2 \"Phosphatidylinositol 4-kinase TOR2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TOR2 \"Phosphatidylinositol 4-kinase TOR2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TPC1 \"Mitochondrial thiamine pyrophosphate carrier 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TPC1 \"Mitochondrial thiamine pyrophosphate carrier 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TPI1 \"Triosephosphate isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TPI1 \"Triosephosphate isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TPN1 \"Vitamin B6 transporter TPN1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TPN1 \"Vitamin B6 transporter TPN1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TPS1 \"Trehalose-6-phosphate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TPS1 \"Trehalose-6-phosphate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TPS2 \"Trehalose-phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TPS2 \"Trehalose-phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TPS3 \"Trehalose synthase complex regulatory subunit TPS3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TPS3 \"Trehalose synthase complex regulatory subunit TPS3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRK1 \"High-affinity potassium transport protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRK1 \"High-affinity potassium transport protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRP1 \"N-(5'-phosphoribosyl)anthranilate isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRP1 \"N-(5'-phosphoribosyl)anthranilate isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRP2_1 \"Anthranilate synthase component 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRP2_1 \"Anthranilate synthase component 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRP2_2 \"Anthranilate synthase component 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRP2_2 \"Anthranilate synthase component 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRP3_1 \"Glutamine amidotransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRP3_1 \"Glutamine amidotransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRP3_2 \"Indole-3-glycerol phosphate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRP3_2 \"Indole-3-glycerol phosphate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRP3_3 \"Anthranilate synthase component 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRP3_3 \"Anthranilate synthase component 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRP4 \"Anthranilate phosphoribosyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRP4 \"Anthranilate phosphoribosyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRP5 \"Tryptophan synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRP5 \"Tryptophan synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRR1 \"Thioredoxin reductase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRR1 \"Thioredoxin reductase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRR2 \"Thioredoxin reductase 2, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRR2 \"Thioredoxin reductase 2, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSC10 \"3-ketodihydrosphingosine reductase TSC10\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSC10 \"3-ketodihydrosphingosine reductase TSC10\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSC13_1 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSC13_1 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSC13_2 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSC13_2 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSC13_3 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSC13_3 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSC13_4 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSC13_4 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSC13_5 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSC13_5 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSC13_6 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSC13_6 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSC13_7 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSC13_7 \"Enoyl reductase TSC13\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TSL1 \"Trehalose synthase complex regulatory subunit TSL1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TSL1 \"Trehalose synthase complex regulatory subunit TSL1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TYR1 \"Prephenate dehydrogenase [NADP+]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TYR1 \"Prephenate dehydrogenase [NADP+]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TYS1 \"Tyrosyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TYS1 \"Tyrosyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U1_ \"1-phosphofructokinase (fructose 1-phosphate kinase)\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U1_ \"1-phosphofructokinase (fructose 1-phosphate kinase)\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U100_ \"Quinolate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U100_ \"Quinolate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U101_ \"NADP phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U101_ \"NADP phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U102_ \"NAD+ nucleosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U102_ \"NAD+ nucleosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U103_ \"Purine-nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U103_ \"Purine-nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U104_ \"Purine-nucleoside phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U104_ \"Purine-nucleoside phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U105_ \"NADP phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U105_ \"NADP phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U106_ \"NAD+ nucleosidase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U106_ \"NAD+ nucleosidase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U107_ \"2-octaprenylphenol hydroxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U107_ \"2-octaprenylphenol hydroxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U108_ \"2-octaprenyl-6-methoxyphenol hydroxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U108_ \"2-octaprenyl-6-methoxyphenol hydroxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U109_ \"Transport of O2, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U109_ \"Transport of O2, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U110_ \"Transport of CO2, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U110_ \"Transport of CO2, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U111_ \"Transport of ethanol, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U111_ \"Transport of ethanol, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U112_ \"Transport of NH3, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U112_ \"Transport of NH3, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U113_ \"Transport of methane, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U113_ \"Transport of methane, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U114_ \"Transport of tetrahydrofolate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U114_ \"Transport of tetrahydrofolate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U115_ \"Transport of 5,10-methylenetetrahydrofolate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U115_ \"Transport of 5,10-methylenetetrahydrofolate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U116_ \"Transport of L-serine, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U116_ \"Transport of L-serine, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U117_ \"Transport of glycine, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U117_ \"Transport of glycine, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U118_ \"Transport of 3-isopropylmalate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U118_ \"Transport of 3-isopropylmalate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U119_ \"Transport of 4-methyl-2-oxopentanoate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U119_ \"Transport of 4-methyl-2-oxopentanoate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U120_ \"Transport of L-proline, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U120_ \"Transport of L-proline, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U121_ \"Transport of CMP, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U121_ \"Transport of CMP, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U122_ \"Transport of acetate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U122_ \"Transport of acetate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U123_ \"Transport of O-acetylcarnitine, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U123_ \"Transport of O-acetylcarnitine, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U124_ \"Transport of carnitine, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U124_ \"Transport of carnitine, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U125_ \"Transport of 2-acetolactate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U125_ \"Transport of 2-acetolactate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U126_ \"Transport of acetoacetate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U126_ \"Transport of acetoacetate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U127_ \"Transport of sulfate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U127_ \"Transport of sulfate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U128_ \"Transport of L-threonine, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U128_ \"Transport of L-threonine, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U129_ \"Transport of 2-oxoadipate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U129_ \"Transport of 2-oxoadipate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U130_ \"Transport of malonate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U130_ \"Transport of malonate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U131_ \"Transport of 2-isopropylmalate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U131_ \"Transport of 2-isopropylmalate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U132_ \"Transport of (R)-lactate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U132_ \"Transport of (R)-lactate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U133_ \"Transport of pyruvate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U133_ \"Transport of pyruvate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U135_ \"Transport of glutamate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U135_ \"Transport of glutamate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U136_ \"Transport of (R)-2-oxoisovalerate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U136_ \"Transport of (R)-2-oxoisovalerate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U137_ \"Transport of 3-methyl-2-oxobutanoate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U137_ \"Transport of 3-methyl-2-oxobutanoate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U137b_ \"Transport of 3-methyl-2-oxopentanoate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U137b_ \"Transport of 3-methyl-2-oxopentanoate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U138_ \"Transport of riboflavin, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U138_ \"Transport of riboflavin, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U139_ \"Transport of dethiobiotin, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U139_ \"Transport of dethiobiotin, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U140_ \"Transport of (S)-3-hydroxy-3-methylglutaryl-CoA, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U140_ \"Transport of (S)-3-hydroxy-3-methylglutaryl-CoA, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U141_ \"Transport of (R)-mevalonate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U141_ \"Transport of (R)-mevalonate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U142_ \"Transport of phosphatidate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U142_ \"Transport of phosphatidate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U143_ \"Transport of pantetheine 4'-phosphate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U143_ \"Transport of pantetheine 4'-phosphate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U144_ \"Transport of adenine, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U144_ \"Transport of adenine, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U145_ \"Transport of 5-phospho-alpha-D-ribose 1-diphosphate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U145_ \"Transport of 5-phospho-alpha-D-ribose 1-diphosphate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U146_ \"Transport of dihydrofolate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U146_ \"Transport of dihydrofolate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U147_ \"Transport of pyridine-2,3-dicarboxylate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U147_ \"Transport of pyridine-2,3-dicarboxylate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U149_ \"Transport of S-adenosyl-L-methionine, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U149_ \"Transport of S-adenosyl-L-methionine, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U15_ \"Thymidine (deoxyuridine) kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U15_ \"Thymidine (deoxyuridine) kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U150_ \"Transport of S-adenosyl-L-homocysteine, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U150_ \"Transport of S-adenosyl-L-homocysteine, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U151_ \"Transport of glycerone phosphate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U151_ \"Transport of glycerone phosphate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U152_ \"Transport of sn-glycerol 3-phosphate, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U152_ \"Transport of sn-glycerol 3-phosphate, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U154_ \"Transport of sucrose, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U154_ \"Transport of sucrose, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U155_ \"a-ketoglutarate/malate translocator, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U155_ \"a-ketoglutarate/malate translocator, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U156_ \"Transport of a-methylglucoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U156_ \"Transport of a-methylglucoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U157_ \"Transport of sorbose, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U157_ \"Transport of sorbose, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U158_ \"Transport of arabinose (low affinity), extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U158_ \"Transport of arabinose (low affinity), extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U159_ \"Transport of fucose, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U159_ \"Transport of fucose, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U16_ \"Thymidine (deoxyuridine) kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U16_ \"Thymidine (deoxyuridine) kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U161_ \"Transport of glucitol, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U161_ \"Transport of glucitol, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U162_ \"Transport of glucosamine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U162_ \"Transport of glucosamine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U163_ \"Transport of mannitol, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U163_ \"Transport of mannitol, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U164_ \"Transport of melibiose, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U164_ \"Transport of melibiose, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U165_ \"Transport of N-acetylglucosamine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U165_ \"Transport of N-acetylglucosamine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U166_ \"Transport of rhamnose, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U166_ \"Transport of rhamnose, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U167_ \"Transport of ribose, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U167_ \"Transport of ribose, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U168_ \"Transport of trehalose, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U168_ \"Transport of trehalose, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U17_ \"Cytidylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U17_ \"Cytidylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U170_ \"Transport of D-xylose, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U170_ \"Transport of D-xylose, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U171_ \"Transport of putrescine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U171_ \"Transport of putrescine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U172_ \"Transport of spermidine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U172_ \"Transport of spermidine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U173_ \"Transport of nicotinamide mononucleotide, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U173_ \"Transport of nicotinamide mononucleotide, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U174_ \"Transport of adenosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U174_ \"Transport of adenosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U175_ \"Transport of guanosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U175_ \"Transport of guanosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U176_ \"Transport of cytidine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U176_ \"Transport of cytidine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U177_ \"Transport of inosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U177_ \"Transport of inosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U178_ \"Transport of xanthosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U178_ \"Transport of xanthosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U179_ \"Transport of thymidine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U179_ \"Transport of thymidine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U18_ \"Cytidylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U18_ \"Cytidylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U180_ \"Transport of deoxyinosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U180_ \"Transport of deoxyinosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U181_ \"Transport of deoxyguanosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U181_ \"Transport of deoxyguanosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U182_ \"Transport of deoxyadenosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U182_ \"Transport of deoxyadenosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U183_ \"Transport of deoxycytidine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U183_ \"Transport of deoxycytidine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U184_ \"Transport of deoxyuridine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U184_ \"Transport of deoxyuridine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U185_ \"Transport of adenosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U185_ \"Transport of adenosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U186_ \"Transport of cytidine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U186_ \"Transport of cytidine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U187_ \"Transport of thymidine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U187_ \"Transport of thymidine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U188_ \"Transport of deoxyadenosine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U188_ \"Transport of deoxyadenosine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U189_ \"Transport of deoxycytidine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U189_ \"Transport of deoxycytidine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U19_ \"Adenylate kinase, dAMP kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U19_ \"Adenylate kinase, dAMP kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U190_ \"Transport of deoxyuridine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U190_ \"Transport of deoxyuridine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U191_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U191_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U192_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U192_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U193_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U193_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U194_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U194_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U195_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U195_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U196_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U196_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U197_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U197_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U198_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U198_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U199_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U199_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U2_ \"Formate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U2_ \"Formate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U20_ \"Inosine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U20_ \"Inosine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U200_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U200_ \"Transport of nucleosides and deoxynucleoside, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U201_ \"Transport of hypoxanthine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U201_ \"Transport of hypoxanthine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U202_ \"Transport of xanthine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U202_ \"Transport of xanthine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U203_ \"Transport of formate, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U203_ \"Transport of formate, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U204_ \"Transport of ethanol, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U204_ \"Transport of ethanol, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U205_ \"Transport of succinate, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U205_ \"Transport of succinate, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U206_ \"Transport of citrate, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U206_ \"Transport of citrate, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U207_ \"Transport of dicarboxylates, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U207_ \"Transport of dicarboxylates, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U208_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U208_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U2081_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U2081_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U209_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U209_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U21_ \"Guanosine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U21_ \"Guanosine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U210_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U210_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U211_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U211_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U212_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U212_ \"Transport of fatty acids, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U213_ \"Transport of a-ketoglutarate, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U213_ \"Transport of a-ketoglutarate, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U214_ \"ATP drain flux for constant maintanence requirements\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U214_ \"ATP drain flux for constant maintanence requirements\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U215_ \"Transport of glyceraldehyde, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U215_ \"Transport of glyceraldehyde, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U216_ \"Transport of acetaldehyde, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U216_ \"Transport of acetaldehyde, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U217_ \"Transport of methane, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U217_ \"Transport of methane, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U218_ \"Transport of adenosine 3',5'-bisphosphate, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U218_ \"Transport of adenosine 3',5'-bisphosphate, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U219_ \"Transport of dTTP, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U219_ \"Transport of dTTP, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U22_ \"Uridine phosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U22_ \"Uridine phosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U220_ \"Transport of thymine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U220_ \"Transport of thymine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U221_ \"Transport of D-glucosamine 6-phosphate, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U221_ \"Transport of D-glucosamine 6-phosphate, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U222_ \"Transport of 8-amino-7-oxononanoate, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U222_ \"Transport of 8-amino-7-oxononanoate, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U223_ \"Transport of 7,8-diaminononanoate, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U223_ \"Transport of 7,8-diaminononanoate, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U224_ \"Transport of oxidized glutathione, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U224_ \"Transport of oxidized glutathione, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U225_ \"Transport of spermine, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U225_ \"Transport of spermine, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U226_ \"Transport of pimelic acid, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U226_ \"Transport of pimelic acid, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U227_ \"Transport of oxygen, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U227_ \"Transport of oxygen, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U228_ \"Transport of carbon dioxide, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U228_ \"Transport of carbon dioxide, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U229_ \"Transport of riboflavin, extracellular\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U229_ \"Transport of riboflavin, extracellular\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U23_ \"CMP glycosylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U23_ \"CMP glycosylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U24_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U24_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U25_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U25_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U26_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U26_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U27_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U27_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U28_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U28_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U29_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U29_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U30_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U30_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U31_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U31_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U32_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U32_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U33_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U33_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U34_ \"5'-nucleotidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U34_ \"5'-nucleotidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U35_ \"Ribonucleoside-triphosphate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U35_ \"Ribonucleoside-triphosphate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U36_ \"Ribonucleoside-triphosphate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U36_ \"Ribonucleoside-triphosphate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U37_ \"Ribonucleoside-triphosphate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U37_ \"Ribonucleoside-triphosphate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U38_ \"Ribonucleoside-triphosphate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U38_ \"Ribonucleoside-triphosphate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U39_ \"Nucleoside triphosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U39_ \"Nucleoside triphosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U40_ \"Nucleoside triphosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U40_ \"Nucleoside triphosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U41_ \"Hypothetical enzyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U41_ \"Hypothetical enzyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U42_ \"Glutaminase A\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U42_ \"Glutaminase A\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U43_ \"Glutaminase B\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U43_ \"Glutaminase B\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U44_ \"Glucosamine-6-phosphate deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U44_ \"Glucosamine-6-phosphate deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U45_ \"Mannitol-1-phosphate 5-dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U45_ \"Mannitol-1-phosphate 5-dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U46_ \"Threonine dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U46_ \"Threonine dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U47_ \"Methionine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U47_ \"Methionine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U48_ \"Serine transacetylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U48_ \"Serine transacetylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U49_ \"2-methylcitrate dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U49_ \"2-methylcitrate dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U50_ \"Non-enzymatic reaction\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U50_ \"Non-enzymatic reaction\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U51_ \"2-aminoadipate transaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U51_ \"2-aminoadipate transaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U52_ \"Prephanate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U52_ \"Prephanate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U53_ \"Kynurenine formamidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U53_ \"Kynurenine formamidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U54_ \"Picolinic acid decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U54_ \"Picolinic acid decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U55_ \"Hypothetical enzyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U55_ \"Hypothetical enzyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U56_ \"Hypothetical enzyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U56_ \"Hypothetical enzyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U57_ \"4-hydroxyphenylpyruvate dioxygenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U57_ \"4-hydroxyphenylpyruvate dioxygenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U58_ \"Homogentisate 1,2-dioxygenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U58_ \"Homogentisate 1,2-dioxygenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U59_ \"Maleyl-acetoacetate isomerase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U59_ \"Maleyl-acetoacetate isomerase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U60_ \"Fumarylacetoacetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U60_ \"Fumarylacetoacetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U61_ \"Diamine transaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U61_ \"Diamine transaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U62_ \"Polyamine oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U62_ \"Polyamine oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U63_ \"Polyamine oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U63_ \"Polyamine oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U64_ \"Diamine transaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U64_ \"Diamine transaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U65_ \"Polyamine oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U65_ \"Polyamine oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U66_ \"Spontaneous conversion\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U66_ \"Spontaneous conversion\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U67_ \"Spontaneous conversion\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U67_ \"Spontaneous conversion\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U68_ \"Aldehyde dehydrogenase, mitochondrial 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U68_ \"Aldehyde dehydrogenase, mitochondrial 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U70_ \"ThiC protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U70_ \"ThiC protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U71_ \"Hypothetical enzyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U71_ \"Hypothetical enzyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U72_ \"ThiG protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U72_ \"ThiG protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U73_ \"ThiE protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U73_ \"ThiE protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U74_ \"ThiF protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U74_ \"ThiF protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U75_ \"ThiH protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U75_ \"ThiH protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U76_ \"Thiamin phosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U76_ \"Thiamin phosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U77_ \"(DL)-glycerol-3-phosphatase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U77_ \"(DL)-glycerol-3-phosphatase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U78_ \"Pyrimidine phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U78_ \"Pyrimidine phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U79_ \"3,4-dihydroxy-2-butanone-4-phosphate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U79_ \"3,4-dihydroxy-2-butanone-4-phosphate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U80_ \"FAD synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U80_ \"FAD synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U81_ \"Pyridoxine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U81_ \"Pyridoxine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U82_ \"Pyridoxine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U82_ \"Pyridoxine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U83_ \"Pyridoxine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U83_ \"Pyridoxine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U84_ \"Hypothetical enzyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U84_ \"Hypothetical enzyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U85_ \"Hypothetical enzyme\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U85_ \"Hypothetical enzyme\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U86_ \"8-amino-7-oxononanoate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U86_ \"8-amino-7-oxononanoate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U87_ \"Dihydroneopterin triphosphate pyrophosphorylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U87_ \"Dihydroneopterin triphosphate pyrophosphorylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U88_ \"Aminodeoxychorismate lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U88_ \"Aminodeoxychorismate lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U89_ \"Dihydrofolate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U89_ \"Dihydrofolate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U9_ \"Phosphatidylglycerol phosphate phosphatase A\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U9_ \"Phosphatidylglycerol phosphate phosphatase A\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U90_ \"5-formyltetrahydrofolate cyclo-ligase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U90_ \"5-formyltetrahydrofolate cyclo-ligase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U91_ \"5-formyltetrahydrofolate cyclo-ligase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U91_ \"5-formyltetrahydrofolate cyclo-ligase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U92_ \"Phosphopantothenate-cysteine ligase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U92_ \"Phosphopantothenate-cysteine ligase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U93_ \"Phosphopantothenate-cysteine decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U93_ \"Phosphopantothenate-cysteine decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U94_ \"Phospho-pantethiene adenylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U94_ \"Phospho-pantethiene adenylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U95_ \"Phospho-pantethiene adenylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U95_ \"Phospho-pantethiene adenylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U96_ \"Dephosphocoa kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U96_ \"Dephosphocoa kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U97_ \"Dephosphocoa kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U97_ \"Dephosphocoa kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U98_ \"Aspartate alpha-decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U98_ \"Aspartate alpha-decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_U99_ \"Aspartate oxidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_U99_ \"Aspartate oxidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_1_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_1_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_2_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_2_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_3_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_3_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_4_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_4_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_5_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_5_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_6_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_6_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_7_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_7_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_8_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_8_1 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UAcyl_8_2 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UAcyl_8_2 \"Acyl-[acyl-carrier-protein] transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uc10_ \"Fatty acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uc10_ \"Fatty acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uc12_ \"Fatty acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uc12_ \"Fatty acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uc24_ \"Fatty acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uc24_ \"Fatty acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uc26_ \"Fatty acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uc26_ \"Fatty acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uelo_1 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uelo_1 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uelo_2 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uelo_2 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uelo_3 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uelo_3 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uelo_4 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uelo_4 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uelo_5 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uelo_5 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uelo_6 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uelo_6 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_Uelo_7 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_Uelo_7 \"Model-specific reaction, involved in the elongation of fatty acids\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UGA1 \"4-aminobutyrate aminotransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UGA1 \"4-aminobutyrate aminotransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UGA4_1 \"GABA-specific permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UGA4_1 \"GABA-specific permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UGA4_2 \"GABA-specific permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UGA4_2 \"GABA-specific permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UGP1 \"UTP--glucose-1-phosphate uridylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UGP1 \"UTP--glucose-1-phosphate uridylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA1 \"Dihydroorotate dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA1 \"Dihydroorotate dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA10 \"Orotate phosphoribosyltransferase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA10 \"Orotate phosphoribosyltransferase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA2_1 \"Aspartate carbamoyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA2_1 \"Aspartate carbamoyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA2_2 \"Glutamine-dependent carbamoyl-phosphate synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA2_2 \"Glutamine-dependent carbamoyl-phosphate synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA3 \"Orotidine 5'-phosphate decarboxylase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA3 \"Orotidine 5'-phosphate decarboxylase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA4 \"Dihydroorotase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA4 \"Dihydroorotase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA5 \"Orotate phosphoribosyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA5 \"Orotate phosphoribosyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA6_1 \"Uridylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA6_1 \"Uridylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA6_2 \"Uridylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA6_2 \"Uridylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA6_3 \"Uridylate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA6_3 \"Uridylate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA7_1 \"CTP synthase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA7_1 \"CTP synthase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA7_2 \"CTP synthase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA7_2 \"CTP synthase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA8_1 \"CTP synthase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA8_1 \"CTP synthase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URA8_2 \"CTP synthase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URA8_2 \"CTP synthase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URH1_1 \"Uridine nucleosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URH1_1 \"Uridine nucleosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URH1_2 \"Uridine nucleosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URH1_2 \"Uridine nucleosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URK1_1 \"Uridine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URK1_1 \"Uridine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URK1_2 \"Uridine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URK1_2 \"Uridine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URK1_3 \"Uridine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URK1_3 \"Uridine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UTR1_1 \"NAD(+) kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UTR1_1 \"NAD(+) kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UTR1_2 \"NAD(+) kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UTR1_2 \"NAD(+) kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VAP1_1 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VAP1_1 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VAP1_2 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VAP1_2 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VAP1_3 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VAP1_3 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VAP1_4 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VAP1_4 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VAP1_5 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VAP1_5 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VAP1_6 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VAP1_6 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VAP1_7 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VAP1_7 \"Valine/tyrosine/tryptophan amino-acid permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VAS1 \"Valyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VAS1 \"Valyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VHT1 \"H(+)/biotin symporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VHT1 \"H(+)/biotin symporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VPS34 \"Phosphatidylinositol 3-kinase VPS34\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VPS34 \"Phosphatidylinositol 3-kinase VPS34\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_WRS1 \"Tryptophanyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_WRS1 \"Tryptophanyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_XKS1 \"Xylulose kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_XKS1 \"Xylulose kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_XPT1 \"Xanthine phosphoribosyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_XPT1 \"Xanthine phosphoribosyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YAR075W \"Putative inosine-5'-monophosphate dehydrogenase-like protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YAR075W \"Putative inosine-5'-monophosphate dehydrogenase-like protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YAT1 \"Putative mitochondrial carnitine O-acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YAT1 \"Putative mitochondrial carnitine O-acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YAT2 \"Carnitine acetyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YAT2 \"Carnitine acetyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBL098W \"Kynurenine 3-monooxygenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBL098W \"Kynurenine 3-monooxygenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR006W \"Succinate-semialdehyde dehydrogenase [NADP+]\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR006W \"Succinate-semialdehyde dehydrogenase [NADP+]\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR184W_1 \"Alpha-galactosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR184W_1 \"Alpha-galactosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR184W_2 \"Alpha-galactosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR184W_2 \"Alpha-galactosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR184W_3 \"Alpha-galactosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR184W_3 \"Alpha-galactosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR184W_4 \"Alpha-galactosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR184W_4 \"Alpha-galactosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR184W_5 \"Alpha-galactosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR184W_5 \"Alpha-galactosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR184W_6 \"Alpha-galactosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR184W_6 \"Alpha-galactosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR184W_7 \"Alpha-galactosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR184W_7 \"Alpha-galactosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YBR284W \"Uncharacterized deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YBR284W \"Uncharacterized deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YCR024C \"Asparaginyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YCR024C \"Asparaginyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YDC1_1 \"Alkaline ceramidase YDC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YDC1_1 \"Alkaline ceramidase YDC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YDC1_2 \"Alkaline ceramidase YDC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YDC1_2 \"Alkaline ceramidase YDC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YDL100C \"ATPase GET3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YDL100C \"ATPase GET3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YDR111C \"Probable alanine aminotransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YDR111C \"Probable alanine aminotransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YDR287W \"Inositol monophosphatase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YDR287W \"Inositol monophosphatase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YDR341C \"Arginyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YDR341C \"Arginyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YDR531W \"Pantothenate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YDR531W \"Pantothenate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YEA6 \"Putative mitochondrial NAD+ transporter\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YEA6 \"Putative mitochondrial NAD+ transporter\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YEH1 \"Sterol esterase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YEH1 \"Sterol esterase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YEH2 \"Sterol esterase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YEH2 \"Sterol esterase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YEL041W_1 \"ATP-NADH kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YEL041W_1 \"ATP-NADH kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YEL041W_2 \"ATP-NADH kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YEL041W_2 \"ATP-NADH kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YEL047C \"Fumarate reductase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YEL047C \"Fumarate reductase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YER053C \"Mitochondrial phosphate carrier protein 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YER053C \"Mitochondrial phosphate carrier protein 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YER087W \"Probable prolyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YER087W \"Probable prolyl-tRNA synthetase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YFL030W \"Alanine--glyoxylate aminotransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YFL030W \"Alanine--glyoxylate aminotransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YFR055W \"Putative cystathionine beta-lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YFR055W \"Putative cystathionine beta-lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGL186C_1 \"Vitamin B6 transporter TPN1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGL186C_1 \"Vitamin B6 transporter TPN1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGL186C_2 \"Vitamin B6 transporter TPN1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGL186C_2 \"Vitamin B6 transporter TPN1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGL186C_3 \"Vitamin B6 transporter TPN1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGL186C_3 \"Vitamin B6 transporter TPN1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGL245W \"Glutamyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGL245W \"Glutamyl-tRNA synthetase, cytoplasmic\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGR012W \"Putative cysteine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGR012W \"Putative cysteine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGR012W_1 \"Putative cysteine synthase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGR012W_1 \"Putative cysteine synthase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGR043C \"Putative transaldolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGR043C \"Putative transaldolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGR125W \"Uncharacterized sulfate permease\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGR125W \"Uncharacterized sulfate permease\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YGR287C \"Probable alpha-glucosidase FSP2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YGR287C \"Probable alpha-glucosidase FSP2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YHL012W \"Probable UTP--glucose-1-phosphate uridylyltransferase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YHL012W \"Probable UTP--glucose-1-phosphate uridylyltransferase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YHR020W \"Putative prolyl-tRNA synthetase YHR020W\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YHR020W \"Putative prolyl-tRNA synthetase YHR020W\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YIA6 \"NAD+ transporter, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YIA6 \"NAD+ transporter, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YIL145C \"Pantoate--beta-alanine ligase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YIL145C \"Pantoate--beta-alanine ligase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YIL167W \"L-serine dehydratase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YIL167W \"L-serine dehydratase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YIL172C \"Alpha-glucosidase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YIL172C \"Alpha-glucosidase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YJL068C \"S-formylglutathione hydrolase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YJL068C \"S-formylglutathione hydrolase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YJL070C \"Uncharacterized deaminase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YJL070C \"Uncharacterized deaminase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YJL200C \"Probable aconitate hydratase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YJL200C \"Probable aconitate hydratase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YJL216C \"Probable alpha-glucosidase YJL216C\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YJL216C \"Probable alpha-glucosidase YJL216C\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YJR078W \"Indoleamine 2,3-dioxygenase family protein\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YJR078W \"Indoleamine 2,3-dioxygenase family protein\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YJR105W \"Adenosine kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YJR105W \"Adenosine kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YLR089C \"Probable alanine aminotransferase, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YLR089C \"Probable alanine aminotransferase, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YLR231C_1 \"Kynureninase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YLR231C_1 \"Kynureninase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YLR231C_2 \"Kynureninase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YLR231C_2 \"Kynureninase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YLR328W_1 \"Nicotinamide-nucleotide adenylyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YLR328W_1 \"Nicotinamide-nucleotide adenylyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YLR328W_2 \"Nicotinamide-nucleotide adenylyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YLR328W_2 \"Nicotinamide-nucleotide adenylyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YLR328W_3 \"Nicotinamide-nucleotide adenylyltransferase 1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YLR328W_3 \"Nicotinamide-nucleotide adenylyltransferase 1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YML082W \"O-succinylhomoserine (thiol)-lyase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YML082W \"O-succinylhomoserine (thiol)-lyase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YMR1 \"Phosphoinositide 3-phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YMR1 \"Phosphoinositide 3-phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YMR293C \"Probable glutamyl-tRNA(Gln) amidotransferase subunit A, mitochondrial\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YMR293C \"Probable glutamyl-tRNA(Gln) amidotransferase subunit A, mitochondrial\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_1 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_1 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_2 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_2 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_3 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_3 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_4 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_4 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_5 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_5 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_6 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_6 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_7 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_7 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_8 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_8 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNK1_9 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNK1_9 \"Nucleoside diphosphate kinase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YNL247W \"Cysteinyl-tRNA synthetase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YNL247W \"Cysteinyl-tRNA synthetase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YOR071C \"Putative thiamine transporter YOR071C\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YOR071C \"Putative thiamine transporter YOR071C\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YOR192C \"Putative thiamine transporter YOR192C\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YOR192C \"Putative thiamine transporter YOR192C\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YPC1_1 \"Alkaline ceramidase YPC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YPC1_1 \"Alkaline ceramidase YPC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YPC1_2 \"Alkaline ceramidase YPC1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YPC1_2 \"Alkaline ceramidase YPC1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YPL275W \"Formate dehydrogenase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YPL275W \"Formate dehydrogenase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YPL276W \"Formate dehydrogenase 2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YPL276W \"Formate dehydrogenase 2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YSR3_1 \"Dihydrosphingosine 1-phosphate phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YSR3_1 \"Dihydrosphingosine 1-phosphate phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YSR3_2 \"Dihydrosphingosine 1-phosphate phosphatase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YSR3_2 \"Dihydrosphingosine 1-phosphate phosphatase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_YUR1 \"Probable mannosyltransferase YUR1\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_YUR1 \"Probable mannosyltransferase YUR1\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ZWF1 \"Glucose-6-phosphate 1-dehydrogenase\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ZWF1 \"Glucose-6-phosphate 1-dehydrogenase\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_44DIMZYMSTxtI \"Uptake of 4,4-dimethylzymosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_44DIMZYMSTxtI \"Uptake of 4,4-dimethylzymosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACALxtI \"Uptake of acetaldehyde\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACALxtI \"Uptake of acetaldehyde\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACxtI \"Uptake of acetate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACxtI \"Uptake of acetate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADNxtI \"Uptake of adenosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADNxtI \"Uptake of adenosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADxtI \"Uptake of adenine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADxtI \"Uptake of adenine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AKGxtI \"Uptake of 2-oxoglutarate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AKGxtI \"Uptake of 2-oxoglutarate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALAVxtI \"Uptake of 5-aminolevulinate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALAVxtI \"Uptake of 5-aminolevulinate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALAxtI \"Uptake of L-alanine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALAxtI \"Uptake of L-alanine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AMGxtI \"Uptake of methylglucoside\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AMGxtI \"Uptake of methylglucoside\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AONAxtI \"Uptake of 8-amino-7-oxononanoate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AONAxtI \"Uptake of 8-amino-7-oxononanoate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARABxtI \"Uptake of D-arabinose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARABxtI \"Uptake of D-arabinose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARGxtI \"Uptake of L-arginine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARGxtI \"Uptake of L-arginine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASNxtI \"Uptake of L-asparagine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASNxtI \"Uptake of L-asparagine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASPxtI \"Uptake of L-aspartate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASPxtI \"Uptake of L-aspartate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ATNxtI \"Uptake of allantoin\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ATNxtI \"Uptake of allantoin\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ATTxtI \"Uptake of allantoate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ATTxtI \"Uptake of allantoate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BMxtI \"Uptake of biomass\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BMxtI \"Uptake of biomass\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BTxtI \"Uptake of biotin\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BTxtI \"Uptake of biotin\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C100xtI \"Uptake of decanoic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C100xtI \"Uptake of decanoic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C120xtI \"Uptake of lauric acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C120xtI \"Uptake of lauric acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C140xtI \"Uptake of myristic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C140xtI \"Uptake of myristic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C141xtI \"Uptake of tetradecanoyl-9-ene acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C141xtI \"Uptake of tetradecanoyl-9-ene acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C160xtI \"Uptake of palmitic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C160xtI \"Uptake of palmitic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C161xtI \"Uptake of hexadecanoyl-9-ene acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C161xtI \"Uptake of hexadecanoyl-9-ene acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C180xtI \"Uptake of stearic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C180xtI \"Uptake of stearic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C181xtI \"Uptake of octadecanoyl-9-ene acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C181xtI \"Uptake of octadecanoyl-9-ene acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C24xtI \"Uptake of C24 acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C24xtI \"Uptake of C24 acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C26xtI \"Uptake of C26 acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C26xtI \"Uptake of C26 acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CARxtI \"Uptake of carnitinium\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CARxtI \"Uptake of carnitinium\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHOxtI \"Uptake of choline\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHOxtI \"Uptake of choline\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CITxtI \"Uptake of citrate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CITxtI \"Uptake of citrate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CO2xtI \"Uptake of CO2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CO2xtI \"Uptake of CO2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYSxtI \"Uptake of L-cysteine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYSxtI \"Uptake of L-cysteine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYTDxtI \"Uptake of cytidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYTDxtI \"Uptake of cytidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYTSxtI \"Uptake of cytosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYTSxtI \"Uptake of cytosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DANNAxtI \"Uptake of 7,8-diaminononanoate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DANNAxtI \"Uptake of 7,8-diaminononanoate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAxtI \"Uptake of deoxyadenosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAxtI \"Uptake of deoxyadenosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DCxtI \"Uptake of deoxycytidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DCxtI \"Uptake of deoxycytidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DGxtI \"Uptake of deoxyguanosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DGxtI \"Uptake of deoxyguanosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DINxtI \"Uptake of deoxyinosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DINxtI \"Uptake of deoxyinosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIPEPxtI \"Uptake of dipeptide\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIPEPxtI \"Uptake of dipeptide\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DTTPxtI \"Uptake of dTTP\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DTTPxtI \"Uptake of dTTP\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DTxtI \"Uptake of thymidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DTxtI \"Uptake of thymidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DUxtI \"Uptake of deoxyuridine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DUxtI \"Uptake of deoxyuridine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_EPISTxtI \"Uptake of episterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_EPISTxtI \"Uptake of episterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG572224xtI \"Uptake of ergosta-5,7,22,24(28)-tetraen-3beta-ol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG572224xtI \"Uptake of ergosta-5,7,22,24(28)-tetraen-3beta-ol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG722xtI \"Uptake of 5alpha-ergosta-7,22-dien-3beta-ol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG722xtI \"Uptake of 5alpha-ergosta-7,22-dien-3beta-ol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERGOSTxtI \"Uptake of ergosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERGOSTxtI \"Uptake of ergosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETHxtI \"Uptake of ethanol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETHxtI \"Uptake of ethanol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCOSTxtI \"Uptake of fecosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCOSTxtI \"Uptake of fecosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FORxtI \"Uptake of formate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FORxtI \"Uptake of formate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FRUxtI \"Uptake of D-fructose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FRUxtI \"Uptake of D-fructose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUCxtI \"Uptake of beta-D-fucose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUCxtI \"Uptake of beta-D-fucose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUMxtI \"Uptake of fumarate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUMxtI \"Uptake of fumarate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GA6PxtI \"Uptake of D-glucosamine 6-phosphate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GA6PxtI \"Uptake of D-glucosamine 6-phosphate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GABAxtI \"Uptake of gamma-aminobutyrate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GABAxtI \"Uptake of gamma-aminobutyrate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLACxtI \"Uptake of D-galactose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLACxtI \"Uptake of D-galactose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLALxtI \"Uptake of glycolaldehyde\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLALxtI \"Uptake of glycolaldehyde\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLAMxtI \"Uptake of glucosamine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLAMxtI \"Uptake of glucosamine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLCxtI \"Uptake of alpha-D-glucose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLCxtI \"Uptake of alpha-D-glucose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLNxtI \"Uptake of L-glutamine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLNxtI \"Uptake of L-glutamine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLTxtI \"Uptake of L-glucitol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLTxtI \"Uptake of L-glucitol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLUxtI \"Uptake of L-glutamate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLUxtI \"Uptake of L-glutamate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLxtI \"Uptake of glycerol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLxtI \"Uptake of glycerol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLYxtI \"Uptake of glycine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLYxtI \"Uptake of glycine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNxtI \"Uptake of guanine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNxtI \"Uptake of guanine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GROPCxtI \"Uptake of glycerophosphatidylcholine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GROPCxtI \"Uptake of glycerophosphatidylcholine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GROPIxtI \"Uptake of glycerophosphatidyl-D-myo-inositol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GROPIxtI \"Uptake of glycerophosphatidyl-D-myo-inositol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GSNxtI \"Uptake of guanosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GSNxtI \"Uptake of guanosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEXTI \"Uptake of H(+)\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEXTI \"Uptake of H(+)\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HISxtI \"Uptake of L-histidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HISxtI \"Uptake of L-histidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HYXNxtI \"Uptake of hypoxanthine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HYXNxtI \"Uptake of hypoxanthine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILExtI \"Uptake of L-isoleucine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILExtI \"Uptake of L-isoleucine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INSxtI \"Uptake of inosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INSxtI \"Uptake of inosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KxtI \"Uptake of potassium\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KxtI \"Uptake of potassium\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LACxtI \"Uptake of (R)-lactate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LACxtI \"Uptake of (R)-lactate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LANOSTxtI \"Uptake of lanosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LANOSTxtI \"Uptake of lanosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LEUxtI \"Uptake of L-leucine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LEUxtI \"Uptake of L-leucine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYSxtI \"Uptake of L-lysine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYSxtI \"Uptake of L-lysine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MALxtI \"Uptake of malate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MALxtI \"Uptake of malate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MANxtI \"Uptake of alpha-D-mannose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MANxtI \"Uptake of alpha-D-mannose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MELIxtI \"Uptake of melibiose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MELIxtI \"Uptake of melibiose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_METxtI \"Uptake of L-methionine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_METxtI \"Uptake of L-methionine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MIxtI \"Uptake of myo-inositol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MIxtI \"Uptake of myo-inositol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MLTxtI \"Uptake of maltose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MLTxtI \"Uptake of maltose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MMETxtI \"Uptake of S-methyl-L-methionine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MMETxtI \"Uptake of S-methyl-L-methionine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MNTxtI \"Uptake of D-mannitol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MNTxtI \"Uptake of D-mannitol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MTHNxtI \"Uptake of methane\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MTHNxtI \"Uptake of methane\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NAGxtI \"Uptake of N-acetyl-D-glucosamine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NAGxtI \"Uptake of N-acetyl-D-glucosamine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NAxtI \"Uptake of sodium\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NAxtI \"Uptake of sodium\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NH3xtI \"Uptake of NH3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NH3xtI \"Uptake of NH3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NMNxtI \"Uptake of nicotinamide mononucleotide\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NMNxtI \"Uptake of nicotinamide mononucleotide\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_O2xtI \"Uptake of oxygen\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_O2xtI \"Uptake of oxygen\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OGTxtI \"Uptake of oxidized glutathione\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OGTxtI \"Uptake of oxidized glutathione\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OPEPxtI \"Uptake of oligopeptide\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OPEPxtI \"Uptake of oligopeptide\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ORNxtI \"Uptake of L-ornithine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ORNxtI \"Uptake of L-ornithine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PAPxtI \"Uptake of adenosine 3',5'-bisphosphate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PAPxtI \"Uptake of adenosine 3',5'-bisphosphate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PEPTxtI \"Uptake of peptide\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PEPTxtI \"Uptake of peptide\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PHExtI \"Uptake of L-phenylalanine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PHExtI \"Uptake of L-phenylalanine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PIMExtI \"Uptake of pimelic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PIMExtI \"Uptake of pimelic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PIxtI \"Uptake of phosphate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PIxtI \"Uptake of phosphate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNTOxtI \"Uptake of (R)-pantothenate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNTOxtI \"Uptake of (R)-pantothenate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PROxtI \"Uptake of L-proline\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PROxtI \"Uptake of L-proline\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PTRSCxtI \"Uptake of putrescine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PTRSCxtI \"Uptake of putrescine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PYRxtI \"Uptake of pyruvate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PYRxtI \"Uptake of pyruvate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RFLAVxtI \"Uptake of riboflavin\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RFLAVxtI \"Uptake of riboflavin\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RIBxtI \"Uptake of D-ribose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RIBxtI \"Uptake of D-ribose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RMNxtI \"Uptake of D-rhamnose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RMNxtI \"Uptake of D-rhamnose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAMxtI \"Uptake of S-adenosyl-L-methionine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAMxtI \"Uptake of S-adenosyl-L-methionine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SERxtI \"Uptake of L-serine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SERxtI \"Uptake of L-serine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SLFxtI \"Uptake of sulfate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SLFxtI \"Uptake of sulfate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SORxtI \"Uptake of sorbose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SORxtI \"Uptake of sorbose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPRMDxtI \"Uptake of spermidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPRMDxtI \"Uptake of spermidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPRMxtI \"Uptake of spermine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPRMxtI \"Uptake of spermine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUCCxtI \"Uptake of succinate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUCCxtI \"Uptake of succinate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUCxtI \"Uptake of sucrose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUCxtI \"Uptake of sucrose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THMxtI \"Uptake of thiamin\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THMxtI \"Uptake of thiamin\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THRxtI \"Uptake of L-threonine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THRxtI \"Uptake of L-threonine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THYxtI \"Uptake of thymine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THYxtI \"Uptake of thymine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRExtI \"Uptake of alpha,alpha-trehalose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRExtI \"Uptake of alpha,alpha-trehalose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRPxtI \"Uptake of L-tryptophan\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRPxtI \"Uptake of L-tryptophan\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TYRxtI \"Uptake of L-tyrosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TYRxtI \"Uptake of L-tyrosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URAxtI \"Uptake of uracil\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URAxtI \"Uptake of uracil\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UREAxtI \"Uptake of urea\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UREAxtI \"Uptake of urea\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URIxtI \"Uptake of uridine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URIxtI \"Uptake of uridine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VALxtI \"Uptake of L-valine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VALxtI \"Uptake of L-valine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VB6xtI \"Uptake of pyridoxine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VB6xtI \"Uptake of pyridoxine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_XANxtI \"Uptake of xanthine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_XANxtI \"Uptake of xanthine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_XTSINExtI \"Uptake of xanthosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_XTSINExtI \"Uptake of xanthosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_XYLxtI \"Uptake of D-xylose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_XYLxtI \"Uptake of D-xylose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ZYMSTxtI \"Uptake of zymosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ZYMSTxtI \"Uptake of zymosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_44DIMZYMSTxtO \"Excretion of 4,4-dimethylzymosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_44DIMZYMSTxtO \"Excretion of 4,4-dimethylzymosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACALxtO \"Excretion of acetaldehyde\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACALxtO \"Excretion of acetaldehyde\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ACxtO \"Excretion of acetate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ACxtO \"Excretion of acetate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADNxtO \"Excretion of adenosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADNxtO \"Excretion of adenosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ADxtO \"Excretion of adenine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ADxtO \"Excretion of adenine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AKGxtO \"Excretion of 2-oxoglutarate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AKGxtO \"Excretion of 2-oxoglutarate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALAVxtO \"Excretion of 5-aminolevulinate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALAVxtO \"Excretion of 5-aminolevulinate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ALAxtO \"Excretion of L-alanine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ALAxtO \"Excretion of L-alanine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AMGxtO \"Excretion of methylglucoside\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AMGxtO \"Excretion of methylglucoside\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_AONAxtO \"Excretion of 8-amino-7-oxononanoate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_AONAxtO \"Excretion of 8-amino-7-oxononanoate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARABxtO \"Excretion of D-arabinose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARABxtO \"Excretion of D-arabinose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ARGxtO \"Excretion of L-arginine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ARGxtO \"Excretion of L-arginine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASNxtO \"Excretion of L-asparagine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASNxtO \"Excretion of L-asparagine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ASPxtO \"Excretion of L-aspartate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ASPxtO \"Excretion of L-aspartate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ATNxtO \"Excretion of allantoin\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ATNxtO \"Excretion of allantoin\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ATTxtO \"Excretion of allantoate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ATTxtO \"Excretion of allantoate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BMxtO \"Excretion of biomass\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BMxtO \"Excretion of biomass\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_BTxtO \"Excretion of biotin\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_BTxtO \"Excretion of biotin\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C100xtO \"Excretion of decanoic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C100xtO \"Excretion of decanoic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C120xtO \"Excretion of lauric acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C120xtO \"Excretion of lauric acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C140xtO \"Excretion of myristic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C140xtO \"Excretion of myristic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C141xtO \"Excretion of tetradecanoyl-9-ene acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C141xtO \"Excretion of tetradecanoyl-9-ene acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C160xtO \"Excretion of palmitic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C160xtO \"Excretion of palmitic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C161xtO \"Excretion of hexadecanoyl-9-ene acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C161xtO \"Excretion of hexadecanoyl-9-ene acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C180xtO \"Excretion of stearic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C180xtO \"Excretion of stearic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C181xtO \"Excretion of octadecanoyl-9-ene acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C181xtO \"Excretion of octadecanoyl-9-ene acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C24xtO \"Excretion of C24 acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C24xtO \"Excretion of C24 acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_C26xtO \"Excretion of C26 acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_C26xtO \"Excretion of C26 acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CARxtO \"Excretion of carnitinium\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CARxtO \"Excretion of carnitinium\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CHOxtO \"Excretion of choline\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CHOxtO \"Excretion of choline\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CITxtO \"Excretion of citrate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CITxtO \"Excretion of citrate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CO2xtO \"Excretion of CO2\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CO2xtO \"Excretion of CO2\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYSxtO \"Excretion of L-cysteine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYSxtO \"Excretion of L-cysteine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYTDxtO \"Excretion of cytidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYTDxtO \"Excretion of cytidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CYTSxtO \"Excretion of cytosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CYTSxtO \"Excretion of cytosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DANNAxtO \"Excretion of 7,8-diaminononanoate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DANNAxtO \"Excretion of 7,8-diaminononanoate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DAxtO \"Excretion of deoxyadenosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DAxtO \"Excretion of deoxyadenosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DCxtO \"Excretion of deoxycytidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DCxtO \"Excretion of deoxycytidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DGxtO \"Excretion of deoxyguanosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DGxtO \"Excretion of deoxyguanosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DINxtO \"Excretion of deoxyinosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DINxtO \"Excretion of deoxyinosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DIPEPxtO \"Excretion of dipeptide\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DIPEPxtO \"Excretion of dipeptide\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DTTPxtO \"Excretion of dTTP\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DTTPxtO \"Excretion of dTTP\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DTxtO \"Excretion of thymidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DTxtO \"Excretion of thymidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_DUxtO \"Excretion of deoxyuridine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_DUxtO \"Excretion of deoxyuridine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_EPISTxtO \"Excretion of episterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_EPISTxtO \"Excretion of episterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG572224xtO \"Excretion of ergosta-5,7,22,24(28)-tetraen-3beta-ol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG572224xtO \"Excretion of ergosta-5,7,22,24(28)-tetraen-3beta-ol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERG722xtO \"Excretion of 5alpha-ergosta-7,22-dien-3beta-ol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERG722xtO \"Excretion of 5alpha-ergosta-7,22-dien-3beta-ol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ERGOSTxtO \"Excretion of ergosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ERGOSTxtO \"Excretion of ergosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ETHxtO \"Excretion of ethanol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ETHxtO \"Excretion of ethanol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FCOSTxtO \"Excretion of fecosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FCOSTxtO \"Excretion of fecosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FORxtO \"Excretion of formate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FORxtO \"Excretion of formate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FRUxtO \"Excretion of D-fructose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FRUxtO \"Excretion of D-fructose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUCxtO \"Excretion of beta-D-fucose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUCxtO \"Excretion of beta-D-fucose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_FUMxtO \"Excretion of fumarate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_FUMxtO \"Excretion of fumarate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GA6PxtO \"Excretion of D-glucosamine 6-phosphate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GA6PxtO \"Excretion of D-glucosamine 6-phosphate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GABAxtO \"Excretion of gamma-aminobutyrate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GABAxtO \"Excretion of gamma-aminobutyrate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLACxtO \"Excretion of D-galactose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLACxtO \"Excretion of D-galactose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLALxtO \"Excretion of glycolaldehyde\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLALxtO \"Excretion of glycolaldehyde\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLAMxtO \"Excretion of glucosamine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLAMxtO \"Excretion of glucosamine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLCxtO \"Excretion of alpha-D-glucose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLCxtO \"Excretion of alpha-D-glucose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLNxtO \"Excretion of L-glutamine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLNxtO \"Excretion of L-glutamine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLTxtO \"Excretion of L-glucitol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLTxtO \"Excretion of L-glucitol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLUxtO \"Excretion of L-glutamate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLUxtO \"Excretion of L-glutamate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLxtO \"Excretion of glycerol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLxtO \"Excretion of glycerol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GLYxtO \"Excretion of glycine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GLYxtO \"Excretion of glycine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GNxtO \"Excretion of guanine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GNxtO \"Excretion of guanine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GROPCxtO \"Excretion of glycerophosphatidylcholine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GROPCxtO \"Excretion of glycerophosphatidylcholine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GROPIxtO \"Excretion of glycerophosphatidyl-D-myo-inositol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GROPIxtO \"Excretion of glycerophosphatidyl-D-myo-inositol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GSNxtO \"Excretion of guanosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GSNxtO \"Excretion of guanosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HEXTO \"Excretion of H(+)\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HEXTO \"Excretion of H(+)\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HISxtO \"Excretion of L-histidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HISxtO \"Excretion of L-histidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_HYXNxtO \"Excretion of hypoxanthine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_HYXNxtO \"Excretion of hypoxanthine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ILExtO \"Excretion of L-isoleucine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ILExtO \"Excretion of L-isoleucine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_INSxtO \"Excretion of inosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_INSxtO \"Excretion of inosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_KxtO \"Excretion of potassium\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_KxtO \"Excretion of potassium\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LACxtO \"Excretion of (R)-lactate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LACxtO \"Excretion of (R)-lactate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LANOSTxtO \"Excretion of lanosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LANOSTxtO \"Excretion of lanosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LEUxtO \"Excretion of L-leucine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LEUxtO \"Excretion of L-leucine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_LYSxtO \"Excretion of L-lysine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_LYSxtO \"Excretion of L-lysine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MALxtO \"Excretion of malate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MALxtO \"Excretion of malate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MANxtO \"Excretion of alpha-D-mannose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MANxtO \"Excretion of alpha-D-mannose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MELIxtO \"Excretion of melibiose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MELIxtO \"Excretion of melibiose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_METxtO \"Excretion of L-methionine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_METxtO \"Excretion of L-methionine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MIxtO \"Excretion of myo-inositol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MIxtO \"Excretion of myo-inositol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MLTxtO \"Excretion of maltose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MLTxtO \"Excretion of maltose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MMETxtO \"Excretion of S-methyl-L-methionine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MMETxtO \"Excretion of S-methyl-L-methionine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MNTxtO \"Excretion of D-mannitol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MNTxtO \"Excretion of D-mannitol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_MTHNxtO \"Excretion of methane\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_MTHNxtO \"Excretion of methane\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NAGxtO \"Excretion of N-acetyl-D-glucosamine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NAGxtO \"Excretion of N-acetyl-D-glucosamine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NAxtO \"Excretion of sodium\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NAxtO \"Excretion of sodium\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NH3xtO \"Excretion of NH3\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NH3xtO \"Excretion of NH3\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NMNxtO \"Excretion of nicotinamide mononucleotide\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NMNxtO \"Excretion of nicotinamide mononucleotide\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_O2xtO \"Excretion of oxygen\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_O2xtO \"Excretion of oxygen\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OGTxtO \"Excretion of oxidized glutathione\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OGTxtO \"Excretion of oxidized glutathione\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_OPEPxtO \"Excretion of oligopeptide\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_OPEPxtO \"Excretion of oligopeptide\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ORNxtO \"Excretion of L-ornithine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ORNxtO \"Excretion of L-ornithine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PAPxtO \"Excretion of adenosine 3',5'-bisphosphate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PAPxtO \"Excretion of adenosine 3',5'-bisphosphate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PEPTxtO \"Excretion of peptide\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PEPTxtO \"Excretion of peptide\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PHExtO \"Excretion of L-phenylalanine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PHExtO \"Excretion of L-phenylalanine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PIMExtO \"Excretion of pimelic acid\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PIMExtO \"Excretion of pimelic acid\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PIxtO \"Excretion of phosphate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PIxtO \"Excretion of phosphate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PNTOxtO \"Excretion of (R)-pantothenate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PNTOxtO \"Excretion of (R)-pantothenate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PROxtO \"Excretion of L-proline\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PROxtO \"Excretion of L-proline\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PTRSCxtO \"Excretion of putrescine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PTRSCxtO \"Excretion of putrescine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_PYRxtO \"Excretion of pyruvate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_PYRxtO \"Excretion of pyruvate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RFLAVxtO \"Excretion of riboflavin\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RFLAVxtO \"Excretion of riboflavin\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RIBxtO \"Excretion of D-ribose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RIBxtO \"Excretion of D-ribose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_RMNxtO \"Excretion of D-rhamnose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_RMNxtO \"Excretion of D-rhamnose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SAMxtO \"Excretion of S-adenosyl-L-methionine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SAMxtO \"Excretion of S-adenosyl-L-methionine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SERxtO \"Excretion of L-serine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SERxtO \"Excretion of L-serine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SLFxtO \"Excretion of sulfate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SLFxtO \"Excretion of sulfate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SORxtO \"Excretion of sorbose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SORxtO \"Excretion of sorbose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPRMDxtO \"Excretion of spermidine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPRMDxtO \"Excretion of spermidine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SPRMxtO \"Excretion of spermine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SPRMxtO \"Excretion of spermine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUCCxtO \"Excretion of succinate\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUCCxtO \"Excretion of succinate\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_SUCxtO \"Excretion of sucrose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_SUCxtO \"Excretion of sucrose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THMxtO \"Excretion of thiamin\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THMxtO \"Excretion of thiamin\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THRxtO \"Excretion of L-threonine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THRxtO \"Excretion of L-threonine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_THYxtO \"Excretion of thymine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_THYxtO \"Excretion of thymine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRExtO \"Excretion of alpha,alpha-trehalose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRExtO \"Excretion of alpha,alpha-trehalose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TRPxtO \"Excretion of L-tryptophan\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TRPxtO \"Excretion of L-tryptophan\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_TYRxtO \"Excretion of L-tyrosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_TYRxtO \"Excretion of L-tyrosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URAxtO \"Excretion of uracil\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URAxtO \"Excretion of uracil\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_UREAxtO \"Excretion of urea\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_UREAxtO \"Excretion of urea\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_URIxtO \"Excretion of uridine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_URIxtO \"Excretion of uridine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VALxtO \"Excretion of L-valine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VALxtO \"Excretion of L-valine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_VB6xtO \"Excretion of pyridoxine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_VB6xtO \"Excretion of pyridoxine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_XANxtO \"Excretion of xanthine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_XANxtO \"Excretion of xanthine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_XTSINExtO \"Excretion of xanthosine\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_XTSINExtO \"Excretion of xanthosine\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_XYLxtO \"Excretion of D-xylose\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_XYLxtO \"Excretion of D-xylose\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_ZYMSTxtO \"Excretion of zymosterol\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_ZYMSTxtO \"Excretion of zymosterol\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CLIPIDS \"Model-specific reaction, used to group lipid formation for biomass production (carbon limited)\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CLIPIDS \"Model-specific reaction, used to group lipid formation for biomass production (carbon limited)\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_CBIOMASS \"Biomass production, carbon limited\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_CBIOMASS \"Biomass production, carbon limited\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NLIPIDS \"Model-specific reaction, used to group lipid formation for biomass production (nitrogen limited)\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NLIPIDS \"Model-specific reaction, used to group lipid formation for biomass production (nitrogen limited)\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_NBIOMASS \"Biomass production, nitrogen limited\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_NBIOMASS \"Biomass production, nitrogen limited\">'\n",
      "warning: Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_GROWTH \"Growth\">'\n",
      "warning: Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_GROWTH \"Growth\">'\n",
      "error: No objective coefficients in model. Unclear what should be optimized\n",
      "warning: Missing flux bounds on reactions set to default bounds.As best practise and to avoid confusion flux bounds should be set explicitly on all reactions.\n",
      "The current solver interface glpk doesn't support setting the optimality tolerance.\n",
      "The current solver interface glpk doesn't support setting the optimality tolerance.\n",
      "The current solver interface glpk doesn't support setting the optimality tolerance.\n",
      "The current solver interface glpk doesn't support setting the optimality tolerance.\n",
      "Writing diff report to 'index.html'.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "15212 lines were written\n",
      "Writing problem data to 'C:\\Users\\pauli\\AppData\\Local\\Temp\\tmpc0xjteka.glpk'...\n",
      "20822 lines were written\n",
      "Writing problem data to 'C:\\Users\\pauli\\AppData\\Local\\Temp\\tmpvpv88f97.glpk'...\n",
      "13191 lines were written\n",
      "Writing problem data to 'C:\\Users\\pauli\\AppData\\Local\\Temp\\tmp6ikxbhwx.glpk'...\n",
      "20574 lines were written\n",
      "Writing problem data to 'C:\\Users\\pauli\\AppData\\Local\\Temp\\tmpc5sfzmn3.glpk'...\n",
      "16729 lines were written\n",
      "Writing problem data to 'C:\\Users\\pauli\\AppData\\Local\\Temp\\tmptw0m_4ud.glpk'...\n",
      "49634 lines were written\n",
      "\u001b[1m============================= test session starts =============================\u001b[0m\n",
      "platform win32 -- Python 3.11.4, pytest-7.4.0, pluggy-1.0.0\n",
      "rootdir: C:\\Users\\pauli\n",
      "plugins: anyio-3.5.0\n",
      "collected 146 items / 1 skipped\n",
      "\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py \u001b[31mF\u001b[0m\u001b[31m [  0%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[1m============================= test session starts =============================\u001b[0m\n",
      "platform win32 -- Python 3.11.4, pytest-7.4.0, pluggy-1.0.0\n",
      "rootdir: C:\\Users\\pauli\n",
      "plugins: anyio-3.5.0\n",
      "collected 164 items / 1 skipped\n",
      "\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py \u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[33m [  0%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[1m============================= test session starts =============================\u001b[0m\n",
      "platform win32 -- Python 3.11.4, pytest-7.4.0, pluggy-1.0.0\n",
      "rootdir: C:\\Users\\pauli\n",
      "plugins: anyio-3.5.0\n",
      "collected 155 items / 1 skipped\n",
      "\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py \u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m [  0%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[1m============================= test session starts =============================\u001b[0m\n",
      "platform win32 -- Python 3.11.4, pytest-7.4.0, pluggy-1.0.0\n",
      "rootdir: C:\\Users\\pauli\n",
      "plugins: anyio-3.5.0\n",
      "collected 146 items / 1 skipped\n",
      "\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py \u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m [  0%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[1m============================= test session starts =============================\u001b[0m\n",
      "platform win32 -- Python 3.11.4, pytest-7.4.0, pluggy-1.0.0\n",
      "rootdir: C:\\Users\\pauli\n",
      "plugins: anyio-3.5.0\n",
      "collected 146 items / 1 skipped\n",
      "\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py \u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m [  0%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m         [ 44%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py \u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m [ 45%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31m         [ 39%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py \u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31m [ 40%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31m         [ 41%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py \u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31m [ 42%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31m         [ 44%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py \u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m [ 45%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31m         [ 44%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py \u001b[32m.\u001b[0m\u001b[31m [ 45%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[1m============================= test session starts =============================\u001b[0m\n",
      "platform win32 -- Python 3.11.4, pytest-7.4.0, pluggy-1.0.0\n",
      "rootdir: C:\\Users\\pauli\n",
      "plugins: anyio-3.5.0\n",
      "collected 146 items / 1 skipped\n",
      "\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py \u001b[32m.\u001b[0m\u001b[33m [  0%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                   [ 60%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py \u001b[31mF\u001b[0m\u001b[31m [ 60%]\n",
      "\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31m                                                                [ 67%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py \u001b[32m.\u001b[0m\u001b[31m         [ 44%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py \u001b[32m.\u001b[0m\u001b[31m [ 45%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                   [ 56%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py \u001b[32m.\u001b[0m\u001b[31m [ 57%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                   [ 60%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py \u001b[32m.\u001b[0m\u001b[31m [ 60%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m [ 67%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                   [ 60%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py \u001b[32m.\u001b[0m\u001b[31m [ 60%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                   [ 53%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py \u001b[32m.\u001b[0m\u001b[31m [ 54%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                       [ 69%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py \u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                                [ 67%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py \u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m [ 69%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31m [ 67%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                                [ 67%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py \u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31m [ 67%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                             [ 86%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_essentiality.py \u001b[33ms\u001b[0m\u001b[31m [ 87%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 87%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_growth.py \u001b[33ms\u001b[0m\u001b[31m [ 88%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 88%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py \u001b[32m.\u001b[0m\u001b[31m [ 89%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                      [ 91%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbml.py \u001b[32m.\u001b[0m\u001b[31m [ 91%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                        [ 92%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py \u001b[32m.\u001b[0m\u001b[31m [ 93%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                               [100%]\u001b[0m\n",
      "\n",
      "\u001b[33m============================== warnings summary ===============================\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: anyio\n",
      "    self._mark_plugins_for_rewrite(hook)\n",
      "\n",
      "-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html\n",
      "\u001b[36m\u001b[1m=========================== short test summary info ===========================\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_presence\u001b[0m - AssertionError: A total of 1061 metabolites (100.00%) lack any form of anno...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_presence\u001b[0m - AssertionError: A total of 1266 reactions (100.00%) lack any form of annota...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_presence\u001b[0m - AssertionError: A total of 750 genes (100.00%) lack any form of         ann...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[pubchem.compound]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[kegg.compound]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[seed.compound]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchikey]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchi]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[chebi]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[hmdb]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[metanetx.chemical]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[bigg.metabolite]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1061 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[rhea]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[kegg.reaction]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[seed.reaction]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[metanetx.reaction]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[bigg.reaction]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[ec-code]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[brenda]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1266 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[refseq]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for refseq:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[uniprot]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for uniprot:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ecogene]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ecogene:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[kegg.genes]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for kegg....\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigi]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ncbigi:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigene]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ncbig...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbiprotein]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ncbip...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ccds]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for ccds:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[hprd]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for hprd:\n",
      "\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                   [ 60%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py \u001b[32m.\u001b[0m\u001b[31m [ 60%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                             [ 86%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_essentiality.py \u001b[33ms\u001b[0m\u001b[31m [ 87%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 87%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_growth.py \u001b[33ms\u001b[0m\u001b[31m [ 88%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 88%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py \u001b[32m.\u001b[0m\u001b[31m [ 89%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                              [ 70%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py \u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                      [ 91%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbml.py \u001b[32m.\u001b[0m\u001b[31m [ 91%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                        [ 92%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py \u001b[32m.\u001b[0m\u001b[31m [ 93%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31m                                                               [100%]\u001b[0m\u001b[32m.\u001b[0m\n",
      "\n",
      "\u001b[33m============================== warnings summary ===============================\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: anyio\n",
      "    self._mark_plugins_for_rewrite(hook)\n",
      "\n",
      "-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html\n",
      "\u001b[36m\u001b[1m=========================== short test summary info ===========================\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_presence\u001b[0m - AssertionError: A total of 671 metabolites (100.00%) lack any form of annot...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_presence\u001b[0m - AssertionError: A total of 1195 reactions (100.00%) lack any form of annota...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_presence\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[pubchem.compound]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[kegg.compound]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[seed.compound]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchikey]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchi]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[chebi]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[hmdb]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[reactome]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[metanetx.chemical]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[bigg.metabolite]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 671 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[rhea]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[kegg.reaction]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[seed.reaction]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[metanetx.reaction]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[bigg.reaction]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[ec-code]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[brenda]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1195 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[refseq]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[uniprot]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ecogene]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[kegg.genes]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigi]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigene]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbiprotein]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ccds]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[hprd]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[asap]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[pubchem.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the pubchem.compoun...\n",
      "\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                             [ 86%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_essentiality.py \u001b[33ms\u001b[0m\u001b[31m [ 87%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 87%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_growth.py \u001b[33ms\u001b[0m\u001b[31m [ 88%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 88%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py \u001b[32m.\u001b[0m\u001b[31m [ 89%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31m [ 71%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                      [ 91%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbml.py \u001b[32m.\u001b[0m\u001b[31m [ 91%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                        [ 92%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py \u001b[32m.\u001b[0m\u001b[31m [ 93%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                               [100%]\u001b[0m\n",
      "\n",
      "\u001b[33m============================== warnings summary ===============================\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: anyio\n",
      "    self._mark_plugins_for_rewrite(hook)\n",
      "\n",
      "-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html\n",
      "\u001b[36m\u001b[1m=========================== short test summary info ===========================\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_presence\u001b[0m - AssertionError: A total of 1228 metabolites (100.00%) lack any form of anno...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_presence\u001b[0m - AssertionError: A total of 1577 reactions (100.00%) lack any form of annota...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_presence\u001b[0m - AssertionError: A total of 905 genes (100.00%) lack any form of         ann...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[pubchem.compound]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[kegg.compound]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[seed.compound]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchikey]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchi]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[chebi]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[hmdb]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[metanetx.chemical]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[bigg.metabolite]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1228 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[rhea]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[kegg.reaction]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[seed.reaction]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[metanetx.reaction]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[bigg.reaction]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[ec-code]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[brenda]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1577 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[refseq]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for refseq:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[uniprot]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for uniprot:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ecogene]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for ecogene:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[kegg.genes]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for kegg....\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigi]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for ncbigi:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigene]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for ncbig...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbiprotein]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for ncbip...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ccds]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for ccds:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[hprd]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for hprd:\n",
      "\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                                [ 67%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py \u001b[31mF\u001b[0m\u001b[31m [ 67%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                             [ 87%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_essentiality.py \u001b[33ms\u001b[0m\u001b[31m [ 88%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 88%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_growth.py \u001b[33ms\u001b[0m\u001b[31m [ 89%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 89%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py \u001b[32m.\u001b[0m\u001b[31m [ 89%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                      [ 91%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbml.py \u001b[32m.\u001b[0m\u001b[31m [ 92%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                        [ 92%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py \u001b[32m.\u001b[0m\u001b[31m [ 93%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                               [100%]\u001b[0m\n",
      "\n",
      "\u001b[33m============================== warnings summary ===============================\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: anyio\n",
      "    self._mark_plugins_for_rewrite(hook)\n",
      "\n",
      "-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html\n",
      "\u001b[36m\u001b[1m=========================== short test summary info ===========================\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_presence\u001b[0m - AssertionError: A total of 796 metabolites (100.00%) lack any form of annot...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_presence\u001b[0m - AssertionError: A total of 1379 reactions (100.00%) lack any form of annota...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_presence\u001b[0m - AssertionError: A total of 619 genes (100.00%) lack any form of         ann...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[pubchem.compound]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[kegg.compound]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[seed.compound]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchikey]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchi]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[chebi]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[hmdb]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[reactome]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[metanetx.chemical]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[bigg.metabolite]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 796 metabolites (100.00%) lack annotation for...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[rhea]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[kegg.reaction]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[seed.reaction]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[metanetx.reaction]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[bigg.reaction]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[ec-code]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[brenda]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1379 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[refseq]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for refseq:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[uniprot]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for uniprot:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ecogene]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for ecogene:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[kegg.genes]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for kegg....\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigi]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for ncbigi:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigene]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for ncbig...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbiprotein]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for ncbip...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ccds]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for ccds:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[hprd]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for hprd:\n",
      "\u001b[31mF\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                             [ 88%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_essentiality.py \u001b[33ms\u001b[0m\u001b[31m [ 89%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 89%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_growth.py \u001b[33ms\u001b[0m\u001b[31m [ 89%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 89%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py \u001b[32m.\u001b[0m\u001b[31m [ 90%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                      [ 92%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbml.py \u001b[31mF\u001b[0m\u001b[31m [ 92%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                                        [ 93%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py \u001b[31mF\u001b[0m\u001b[31m [ 93%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                                               [100%]\u001b[0m\n",
      "\n",
      "\u001b[33m============================== warnings summary ===============================\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: anyio\n",
      "    self._mark_plugins_for_rewrite(hook)\n",
      "\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_basic.py: 5 warnings\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_biomass.py: 3 warnings\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_consistency.py: 1 warning\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_sbo.py: 2 warnings\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\cobra\\core\\metabolite.py:132: UserWarning: invalid formula (has parenthesis) in 'C2H4NO2R(C2H2NOR)n'\n",
      "    warn(f\"invalid formula (has parenthesis) in '{self.formula}'\")\n",
      "\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_biomass.py::test_biomass_consistency[CBIOMASS]\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_biomass.py::test_biomass_consistency[NBIOMASS]\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_consistency.py::test_reaction_mass_balance\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\cobra\\core\\metabolite.py:132: UserWarning: invalid formula (has parenthesis) in '(C6H10O5)n'\n",
      "    warn(f\"invalid formula (has parenthesis) in '{self.formula}'\")\n",
      "\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_consistency.py::test_reaction_mass_balance\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\cobra\\core\\metabolite.py:132: UserWarning: invalid formula (has parenthesis) in 'C20H37O4P(C5H8)n'\n",
      "    warn(f\"invalid formula (has parenthesis) in '{self.formula}'\")\n",
      "\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_consistency.py::test_reaction_mass_balance\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\cobra\\core\\metabolite.py:132: UserWarning: invalid formula (has parenthesis) in '(C8H13NO5)n'\n",
      "    warn(f\"invalid formula (has parenthesis) in '{self.formula}'\")\n",
      "\n",
      "anaconda3/Lib/site-packages/memote/suite/tests/test_consistency.py::test_reaction_mass_balance\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\cobra\\core\\metabolite.py:132: UserWarning: invalid formula (has parenthesis) in 'C23H38N7O17P3S(CH2)n'\n",
      "    warn(f\"invalid formula (has parenthesis) in '{self.formula}'\")\n",
      "\n",
      "-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html\n",
      "\u001b[36m\u001b[1m=========================== short test summary info ===========================\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_presence\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[pubchem.compound]\u001b[0m - AssertionError: The following 1813 metabolites (99.40%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[kegg.compound]\u001b[0m - AssertionError: The following 1712 metabolites (93.86%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[seed.compound]\u001b[0m - AssertionError: The following 1824 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchikey]\u001b[0m - AssertionError: The following 1824 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchi]\u001b[0m - AssertionError: The following 1824 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[chebi]\u001b[0m - AssertionError: The following 926 metabolites (50.77%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[hmdb]\u001b[0m - AssertionError: The following 1824 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1824 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[metanetx.chemical]\u001b[0m - AssertionError: The following 1824 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[bigg.metabolite]\u001b[0m - AssertionError: The following 1824 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1824 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[rhea]\u001b[0m - AssertionError: The following 1837 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[kegg.reaction]\u001b[0m - AssertionError: The following 1837 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[seed.reaction]\u001b[0m - AssertionError: The following 1837 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[metanetx.reaction]\u001b[0m - AssertionError: The following 1837 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[bigg.reaction]\u001b[0m - AssertionError: The following 1837 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[reactome]\u001b[0m - AssertionError: The following 1837 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[ec-code]\u001b[0m - AssertionError: The following 839 reactions (45.67%) lack annotation for ec...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[brenda]\u001b[0m - AssertionError: The following 1837 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 1837 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[refseq]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[uniprot]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ecogene]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[kegg.genes]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigi]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigene]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbiprotein]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ccds]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[hprd]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[asap]\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[seed.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the seed.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchikey]\u001b[0m - AssertionError: There are no metabolite annotations for the inchikey database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchi]\u001b[0m - AssertionError: There are no metabolite annotations for the inchi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[hmdb]\u001b[0m - AssertionError: There are no metabolite annotations for the hmdb database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no metabolite annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[metanetx.chemical]\u001b[0m - AssertionError: There are no metabolite annotations for the metanetx.chemical\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[bigg.metabolite]\u001b[0m - AssertionError: There are no metabolite annotations for the bigg.metabolite...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no metabolite annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[rhea]\u001b[0m - AssertionError: There are no reaction annotations for the rhea database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[kegg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the kegg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[seed.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the seed.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[metanetx.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the metanetx.reaction...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[bigg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the bigg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no reaction annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[brenda]\u001b[0m - AssertionError: There are no reaction annotations for the brenda database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no reaction annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[refseq]\u001b[0m - AssertionError: There are no gene annotations for the refseq database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[uniprot]\u001b[0m - AssertionError: There are no gene annotations for the uniprot database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ecogene]\u001b[0m - AssertionError: There are no gene annotations for the ecogene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[kegg.genes]\u001b[0m - AssertionError: There are no gene annotations for the kegg.genes database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigi]\u001b[0m - AssertionError: There are no gene annotations for the ncbigi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigene]\u001b[0m - AssertionError: There are no gene annotations for the ncbigene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbiprotein]\u001b[0m - AssertionError: There are no gene annotations for the ncbiprotein database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ccds]\u001b[0m - AssertionError: There are no gene annotations for the ccds database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[hprd]\u001b[0m - AssertionError: There are no gene annotations for the hprd database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[asap]\u001b[0m - AssertionError: There are no gene annotations for the asap database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_genes_presence\u001b[0m - AssertionError: 0 genes are defined in the model.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_metabolites_formula_presence\u001b[0m - AssertionError: There are a total of 1644         metabolites (90.13%) with...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_metabolites_charge_presence\u001b[0m - AssertionError: There are a total of 1824         metabolites (100.00%) wit...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_gene_protein_reaction_rule_presence\u001b[0m - AssertionError: There are a total of 1706 reactions (92.87%) without GPR:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_ngam_presence\u001b[0m - AssertionError: A total of 0 NGAM reactions could be identified:\n",
      "\u001b[31mF\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31m                                             [ 86%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_essentiality.py \u001b[33ms\u001b[0m\u001b[31m [ 87%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 87%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_growth.py \u001b[33ms\u001b[0m\u001b[31m [ 88%]\n",
      "\u001b[0m\u001b[31m                                                                         [ 88%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_matrix.py \u001b[32m.\u001b[0m\u001b[31m [ 89%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                      [ 91%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbml.py \u001b[32m.\u001b[0m\u001b[31m [ 91%]\n",
      "\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                                        [ 92%]\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py \u001b[31mF\u001b[0m\u001b[31m [ 93%]\n",
      "\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[31mF\u001b[0m\u001b[33ms\u001b[0m\u001b[31mF\u001b[0m\u001b[32m.\u001b[0m\u001b[31m                                                               [100%]\u001b[0m\n",
      "\n",
      "\u001b[33m============================== warnings summary ===============================\u001b[0m\n",
      "..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204\n",
      "  C:\\Users\\pauli\\anaconda3\\Lib\\site-packages\\_pytest\\config\\__init__.py:1204: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: anyio\n",
      "    self._mark_plugins_for_rewrite(hook)\n",
      "\n",
      "-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html\n",
      "\u001b[36m\u001b[1m=========================== short test summary info ===========================\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[pubchem.compound]\u001b[0m - AssertionError: The following 2742 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[kegg.compound]\u001b[0m - AssertionError: The following 1066 metabolites (38.88%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[seed.compound]\u001b[0m - AssertionError: The following 2742 metabolites (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchikey]\u001b[0m - AssertionError: The following 2742 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[inchi]\u001b[0m - AssertionError: The following 2742 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[chebi]\u001b[0m - AssertionError: The following 400 metabolites (14.59%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[hmdb]\u001b[0m - AssertionError: The following 2742 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[reactome]\u001b[0m - AssertionError: The following 2742 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[metanetx.chemical]\u001b[0m - AssertionError: The following 547 metabolites (19.95%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[bigg.metabolite]\u001b[0m - AssertionError: The following 1022 metabolites (37.27%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 2742 metabolites (100.00%) lack annotation fo...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[rhea]\u001b[0m - AssertionError: The following 4058 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[kegg.reaction]\u001b[0m - AssertionError: The following 3182 reactions (78.41%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[seed.reaction]\u001b[0m - AssertionError: The following 4058 reactions (100.00%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[metanetx.reaction]\u001b[0m - AssertionError: The following 2406 reactions (59.29%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[bigg.reaction]\u001b[0m - AssertionError: The following 2649 reactions (65.28%) lack annotation for\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[reactome]\u001b[0m - AssertionError: The following 4058 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[ec-code]\u001b[0m - AssertionError: The following 1668 reactions (41.10%) lack annotation for e...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[brenda]\u001b[0m - AssertionError: The following 4058 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_overview[biocyc]\u001b[0m - AssertionError: The following 4058 reactions (100.00%) lack annotation for ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[refseq]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for refseq:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[uniprot]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for unip...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ecogene]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for ecog...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[kegg.genes]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for kegg...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigi]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for ncbigi:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbigene]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for ncbi...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ncbiprotein]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for ncbi...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[ccds]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for ccds:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[hprd]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for hprd:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[asap]\u001b[0m - AssertionError: The following 1161 genes (100.00%) lack annotation for asap:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[pubchem.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the pubchem.compoun...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[seed.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the seed.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[asap]\u001b[0m - AssertionError: The following 750 genes (100.00%) lack annotation for asap:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[pubchem.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the pubchem.compoun...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[kegg.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the kegg.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[seed.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the seed.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchikey]\u001b[0m - AssertionError: There are no metabolite annotations for the inchikey database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchi]\u001b[0m - AssertionError: There are no metabolite annotations for the inchi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[chebi]\u001b[0m - AssertionError: There are no metabolite annotations for the chebi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[hmdb]\u001b[0m - AssertionError: There are no metabolite annotations for the hmdb database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no metabolite annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[metanetx.chemical]\u001b[0m - AssertionError: There are no metabolite annotations for the metanetx.chemical\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[bigg.metabolite]\u001b[0m - AssertionError: There are no metabolite annotations for the bigg.metabolite...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no metabolite annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[rhea]\u001b[0m - AssertionError: There are no reaction annotations for the rhea database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[kegg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the kegg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[seed.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the seed.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[metanetx.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the metanetx.reaction...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[bigg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the bigg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no reaction annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[ec-code]\u001b[0m - AssertionError: There are no reaction annotations for the ec-code database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[brenda]\u001b[0m - AssertionError: There are no reaction annotations for the brenda database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no reaction annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[refseq]\u001b[0m - AssertionError: There are no gene annotations for the refseq database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[uniprot]\u001b[0m - AssertionError: There are no gene annotations for the uniprot database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ecogene]\u001b[0m - AssertionError: There are no gene annotations for the ecogene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[kegg.genes]\u001b[0m - AssertionError: There are no gene annotations for the kegg.genes database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigi]\u001b[0m - AssertionError: There are no gene annotations for the ncbigi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigene]\u001b[0m - AssertionError: There are no gene annotations for the ncbigene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbiprotein]\u001b[0m - AssertionError: There are no gene annotations for the ncbiprotein database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ccds]\u001b[0m - AssertionError: There are no gene annotations for the ccds database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[hprd]\u001b[0m - AssertionError: There are no gene annotations for the hprd database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[asap]\u001b[0m - AssertionError: There are no gene annotations for the asap database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_gene_protein_reaction_rule_presence\u001b[0m - AssertionError: There are a total of 340 reactions (26.86%) without GPR:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_ngam_presence\u001b[0m - RuntimeError: It was not possible to uniquely identify a single metabolite ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_transport_reaction_gpr_presence\u001b[0m - AssertionError: There are a total of 212 transport reactions (71.38% of all\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_reactions_with_identical_genes\u001b[0m - AssertionError: Based only on equal genes there are 132 different groups of\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_medium_metabolites\u001b[0m - TypeError: unsupported operand type(s) for /: 'int' and 'list'\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_biomass_consistency[biomass_SC4_bal]\u001b[0m - AssertionError: The component molar mass of the biomass reaction biomass_SC...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_gam_in_biomass[biomass_SC4_bal]\u001b[0m - AssertionError: No, biomass_SC4_bal does not contain a term for growth-asso...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_fast_growth_default[biomass_SC4_bal]\u001b[0m - AssertionError: Using the biomass reaction biomass_SC4_bal and when the mod...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_essential_precursors_not_in_biomass[biomass_SC4_bal]\u001b[0m - AssertionError: biomass_SC4_bal lacks a total of 35 essential metabolites\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_charge_balance\u001b[0m - AssertionError: A total of 20 (1.74%) reactions are charge unbalanced with at\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_mass_balance\u001b[0m - AssertionError: A total of 31 (2.70%) reactions are mass unbalanced with at...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_blocked_reactions\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_stoichiometrically_balanced_cycles\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_orphans\u001b[0m - AssertionError: A total of 71 (6.69%) metabolites are not produced by any r...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_deadends\u001b[0m - AssertionError: A total of 85 (8.01%) metabolites are not consumed by any r...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_produced_with_open_bounds\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_consumed_with_open_bounds\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_reactions_unbounded_flux_default_condition\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_biomass_specific_sbo_presence\u001b[0m - AssertionError: A total of 1 biomass reactions (100.00% of all biomass reac...\n",
      "\u001b[31m============ \u001b[31m\u001b[1m82 failed\u001b[0m, \u001b[32m44 passed\u001b[0m, \u001b[33m21 skipped\u001b[0m, \u001b[33m1 warning\u001b[0m\u001b[31m in 37.48s\u001b[0m\u001b[31m ============\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[kegg.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the kegg.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[seed.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the seed.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchikey]\u001b[0m - AssertionError: There are no metabolite annotations for the inchikey database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchi]\u001b[0m - AssertionError: There are no metabolite annotations for the inchi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[chebi]\u001b[0m - AssertionError: There are no metabolite annotations for the chebi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[hmdb]\u001b[0m - AssertionError: There are no metabolite annotations for the hmdb database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no metabolite annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[metanetx.chemical]\u001b[0m - AssertionError: There are no metabolite annotations for the metanetx.chemical\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[bigg.metabolite]\u001b[0m - AssertionError: There are no metabolite annotations for the bigg.metabolite...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no metabolite annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[rhea]\u001b[0m - AssertionError: There are no reaction annotations for the rhea database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[kegg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the kegg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[seed.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the seed.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[metanetx.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the metanetx.reaction...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[bigg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the bigg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no reaction annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[ec-code]\u001b[0m - AssertionError: There are no reaction annotations for the ec-code database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[brenda]\u001b[0m - AssertionError: There are no reaction annotations for the brenda database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no reaction annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[refseq]\u001b[0m - AssertionError: There are no gene annotations for the refseq database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[uniprot]\u001b[0m - AssertionError: There are no gene annotations for the uniprot database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ecogene]\u001b[0m - AssertionError: There are no gene annotations for the ecogene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[kegg.genes]\u001b[0m - AssertionError: There are no gene annotations for the kegg.genes database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigi]\u001b[0m - AssertionError: There are no gene annotations for the ncbigi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigene]\u001b[0m - AssertionError: There are no gene annotations for the ncbigene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbiprotein]\u001b[0m - AssertionError: There are no gene annotations for the ncbiprotein database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ccds]\u001b[0m - AssertionError: There are no gene annotations for the ccds database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[hprd]\u001b[0m - AssertionError: There are no gene annotations for the hprd database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[asap]\u001b[0m - AssertionError: There are no gene annotations for the asap database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_genes_presence\u001b[0m - AssertionError: 0 genes are defined in the model.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_metabolites_formula_presence\u001b[0m - AssertionError: There are a total of 671         metabolites (100.00%) with...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_gene_protein_reaction_rule_presence\u001b[0m - AssertionError: There are a total of 1095 reactions (91.63%) without GPR:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_ngam_presence\u001b[0m - ValueError: not enough values to unpack (expected 2, got 1)\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_metabolic_coverage\u001b[0m - ValueError: The model contains no reactions or genes.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_compartments_presence\u001b[0m - AssertionError: A total of 1 compartments are defined in the model: default\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_protein_complex_presence\u001b[0m - AssertionError: A total of 0 reactions are catalyzed by complexes defined\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_transport_reactions\u001b[0m - AssertionError: A total of 0 (0.00%) transport reactions are defined in the\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_constrained_transport_reactions\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_transport_reaction_gpr_presence\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_reversible_oxygen_reactions\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_unique_metabolites\u001b[0m - AssertionError: Not counting the same entities in other compartments, there...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_medium_metabolites\u001b[0m - TypeError: unsupported operand type(s) for /: 'int' and 'list'\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_biomass_presence\u001b[0m - AssertionError: In this model 0 the following biomass reactions were\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_stoichiometric_consistency\u001b[0m - AssertionError: This model's stoichiometry inconsistent\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_unconserved_metabolites\u001b[0m - AssertionError: This model contains 650 unconserved metabolites: PHExt, ADx...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_inconsistent_min_stoichiometry\u001b[0m - TypeError: sequence item 0: expected str instance, list found\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_mass_balance\u001b[0m - AssertionError: A total of 1095 (100.00%) reactions are mass unbalanced wit...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_blocked_reactions\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_stoichiometrically_balanced_cycles\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_orphans\u001b[0m - AssertionError: A total of 2 (0.30%) metabolites are not produced by any re...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_deadends\u001b[0m - AssertionError: A total of 10 (1.49%) metabolites are not consumed by any r...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_produced_with_open_bounds\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_consumed_with_open_bounds\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_reactions_unbounded_flux_default_condition\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31m============ \u001b[31m\u001b[1m88 failed\u001b[0m, \u001b[32m25 passed\u001b[0m, \u001b[33m34 skipped\u001b[0m, \u001b[33m1 warning\u001b[0m\u001b[31m in 42.12s\u001b[0m\u001b[31m ============\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[asap]\u001b[0m - AssertionError: The following 905 genes (100.00%) lack annotation for asap:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[pubchem.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the pubchem.compoun...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[kegg.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the kegg.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[seed.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the seed.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchikey]\u001b[0m - AssertionError: There are no metabolite annotations for the inchikey database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchi]\u001b[0m - AssertionError: There are no metabolite annotations for the inchi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[chebi]\u001b[0m - AssertionError: There are no metabolite annotations for the chebi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[hmdb]\u001b[0m - AssertionError: There are no metabolite annotations for the hmdb database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no metabolite annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[metanetx.chemical]\u001b[0m - AssertionError: There are no metabolite annotations for the metanetx.chemical\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[bigg.metabolite]\u001b[0m - AssertionError: There are no metabolite annotations for the bigg.metabolite...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no metabolite annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[rhea]\u001b[0m - AssertionError: There are no reaction annotations for the rhea database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[kegg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the kegg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[seed.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the seed.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[metanetx.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the metanetx.reaction...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[bigg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the bigg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no reaction annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[ec-code]\u001b[0m - AssertionError: There are no reaction annotations for the ec-code database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[brenda]\u001b[0m - AssertionError: There are no reaction annotations for the brenda database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no reaction annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[refseq]\u001b[0m - AssertionError: There are no gene annotations for the refseq database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[uniprot]\u001b[0m - AssertionError: There are no gene annotations for the uniprot database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ecogene]\u001b[0m - AssertionError: There are no gene annotations for the ecogene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[kegg.genes]\u001b[0m - AssertionError: There are no gene annotations for the kegg.genes database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigi]\u001b[0m - AssertionError: There are no gene annotations for the ncbigi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigene]\u001b[0m - AssertionError: There are no gene annotations for the ncbigene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbiprotein]\u001b[0m - AssertionError: There are no gene annotations for the ncbiprotein database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ccds]\u001b[0m - AssertionError: There are no gene annotations for the ccds database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[hprd]\u001b[0m - AssertionError: There are no gene annotations for the hprd database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[asap]\u001b[0m - AssertionError: There are no gene annotations for the asap database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_metabolites_formula_presence\u001b[0m - AssertionError: There are a total of 1228         metabolites (100.00%) wit...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_gene_protein_reaction_rule_presence\u001b[0m - AssertionError: There are a total of 370 reactions (23.46%) without GPR:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_ngam_presence\u001b[0m - RuntimeError: It was not possible to uniquely identify a single metabolite ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_transport_reactions\u001b[0m - AssertionError: A total of 0 (0.00%) transport reactions are defined in the\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_constrained_transport_reactions\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_transport_reaction_gpr_presence\u001b[0m - ZeroDivisionError: division by zero\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_unique_metabolites\u001b[0m - AssertionError: Not counting the same entities in other compartments, there...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_reactions_with_identical_genes\u001b[0m - AssertionError: Based only on equal genes there are 178 different groups of\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_medium_metabolites\u001b[0m - TypeError: unsupported operand type(s) for /: 'int' and 'list'\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_biomass_consistency[biomass_SC5_notrace]\u001b[0m - AssertionError: The component molar mass of the biomass reaction biomass_SC...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_gam_in_biomass[biomass_SC5_notrace]\u001b[0m - AssertionError: No, biomass_SC5_notrace does not contain a term for growth-...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_fast_growth_default[biomass_SC5_notrace]\u001b[0m - AssertionError: Using the biomass reaction biomass_SC5_notrace and when the...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_essential_precursors_not_in_biomass[biomass_SC5_notrace]\u001b[0m - AssertionError: biomass_SC5_notrace lacks a total of 35 essential metabolites\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_charge_balance\u001b[0m - AssertionError: A total of 21 (1.49%) reactions are charge unbalanced with at\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_mass_balance\u001b[0m - AssertionError: A total of 1412 (100.00%) reactions are mass unbalanced wit...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_blocked_reactions\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_stoichiometrically_balanced_cycles\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_orphans\u001b[0m - AssertionError: A total of 70 (5.70%) metabolites are not produced by any r...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_deadends\u001b[0m - AssertionError: A total of 96 (7.82%) metabolites are not consumed by any r...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_produced_with_open_bounds\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_consumed_with_open_bounds\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_reactions_unbounded_flux_default_condition\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_biomass_specific_sbo_presence\u001b[0m - AssertionError: A total of 1 biomass reactions (100.00% of all biomass reac...\n",
      "\u001b[31m============ \u001b[31m\u001b[1m86 failed\u001b[0m, \u001b[32m39 passed\u001b[0m, \u001b[33m22 skipped\u001b[0m, \u001b[33m1 warning\u001b[0m\u001b[31m in 47.05s\u001b[0m\u001b[31m ============\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchikey]\u001b[0m - AssertionError: There are no metabolite annotations for the inchikey database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchi]\u001b[0m - AssertionError: There are no metabolite annotations for the inchi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[hmdb]\u001b[0m - AssertionError: There are no metabolite annotations for the hmdb database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no metabolite annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no metabolite annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[rhea]\u001b[0m - AssertionError: There are no reaction annotations for the rhea database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[seed.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the seed.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no reaction annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[brenda]\u001b[0m - AssertionError: There are no reaction annotations for the brenda database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no reaction annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[refseq]\u001b[0m - AssertionError: There are no gene annotations for the refseq database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[uniprot]\u001b[0m - AssertionError: There are no gene annotations for the uniprot database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ecogene]\u001b[0m - AssertionError: There are no gene annotations for the ecogene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[kegg.genes]\u001b[0m - AssertionError: There are no gene annotations for the kegg.genes database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigi]\u001b[0m - AssertionError: There are no gene annotations for the ncbigi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigene]\u001b[0m - AssertionError: There are no gene annotations for the ncbigene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbiprotein]\u001b[0m - AssertionError: There are no gene annotations for the ncbiprotein database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ccds]\u001b[0m - AssertionError: There are no gene annotations for the ccds database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[hprd]\u001b[0m - AssertionError: There are no gene annotations for the hprd database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[asap]\u001b[0m - AssertionError: There are no gene annotations for the asap database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_id_namespace_consistency\u001b[0m - AssertionError: 2742 metabolite identifiers (100.00%) deviate from the larg...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_metabolites_formula_presence\u001b[0m - AssertionError: There are a total of 27         metabolites (0.98%) without...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_gene_protein_reaction_rule_presence\u001b[0m - AssertionError: There are a total of 1164 reactions (28.68%) without GPR:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_transport_reaction_gpr_presence\u001b[0m - AssertionError: There are a total of 983 transport reactions (68.65% of all\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_unique_metabolites\u001b[0m - AssertionError: Not counting the same entities in other compartments, there...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_reactions_with_partially_identical_annotations\u001b[0m - AssertionError: Based on annotations there are 253 different groups of over...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_reactions_with_identical_genes\u001b[0m - AssertionError: Based only on equal genes there are 285 different groups of\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_medium_metabolites\u001b[0m - TypeError: unsupported operand type(s) for /: 'int' and 'list'\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_biomass_consistency[r_4041]\u001b[0m - AssertionError: The component molar mass of the biomass reaction r_4041 sum...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_fast_growth_default[r_4041]\u001b[0m - AssertionError: Using the biomass reaction r_4041 and when the model is sim...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_biomass.py::\u001b[1mtest_essential_precursors_not_in_biomass[r_4041]\u001b[0m - AssertionError: r_4041 lacks a total of 30 essential metabolites         (4...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_stoichiometric_consistency\u001b[0m - AssertionError: This model's stoichiometry inconsistent\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_unconserved_metabolites\u001b[0m - AssertionError: This model contains 1905 unconserved metabolites: s_0027[m],\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_inconsistent_min_stoichiometry\u001b[0m - TypeError: sequence item 0: expected str instance, list found\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_charge_balance\u001b[0m - AssertionError: A total of 58 (1.53%) reactions are charge unbalanced with at\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_reaction_mass_balance\u001b[0m - AssertionError: A total of 234 (6.16%) reactions are mass unbalanced with a...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_blocked_reactions\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_stoichiometrically_balanced_cycles\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_orphans\u001b[0m - AssertionError: A total of 120 (4.38%) metabolites are not produced by any ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_deadends\u001b[0m - AssertionError: A total of 143 (5.22%) metabolites are not consumed by any ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_produced_with_open_bounds\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_metabolites_not_consumed_with_open_bounds\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_consistency.py::\u001b[1mtest_find_reactions_unbounded_flux_default_condition\u001b[0m - AssertionError: daemonic processes are not allowed to have children\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_metabolite_sbo_presence\u001b[0m - AssertionError: A total of 0 metabolites (0.00%) lack annotation with any type\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_reaction_sbo_presence\u001b[0m - AssertionError: A total of 0 reactions (0.00%) lack annotation with any type\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_gene_sbo_presence\u001b[0m - AssertionError: A total of 0 genes (0.00%) lack annotation with any type of\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_metabolic_reaction_specific_sbo_presence\u001b[0m - AssertionError: A total of 28 metabolic reactions (1.18% of all purely\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_transport_reaction_specific_sbo_presence\u001b[0m - AssertionError: A total of 340 metabolic reactions (23.74% of all transport\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_metabolite_specific_sbo_presence\u001b[0m - AssertionError: A total of 52 transport reactions (1.90% of all metabolites)\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_gene_specific_sbo_presence\u001b[0m - AssertionError: A total of 0 genes (0.00% of all genes) lack             an...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_exchange_specific_sbo_presence\u001b[0m - AssertionError: A total of 0 exchange reactions (0.00% of all exchange\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_sbo.py::\u001b[1mtest_sink_specific_sbo_presence\u001b[0m - AssertionError: A total of 0 genes (0.00% of all sink reactions) lack\n",
      "\u001b[31m====== \u001b[31m\u001b[1m84 failed\u001b[0m, \u001b[32m43 passed\u001b[0m, \u001b[33m20 skipped\u001b[0m, \u001b[33m1 warning\u001b[0m\u001b[31m in 1019.11s (0:16:59)\u001b[0m\u001b[31m ======\u001b[0m\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_overview[asap]\u001b[0m - AssertionError: The following 619 genes (100.00%) lack annotation for asap:\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[pubchem.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the pubchem.compoun...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[kegg.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the kegg.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[seed.compound]\u001b[0m - AssertionError: There are no metabolite annotations for the seed.compound d...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchikey]\u001b[0m - AssertionError: There are no metabolite annotations for the inchikey database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[inchi]\u001b[0m - AssertionError: There are no metabolite annotations for the inchi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[chebi]\u001b[0m - AssertionError: There are no metabolite annotations for the chebi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[hmdb]\u001b[0m - AssertionError: There are no metabolite annotations for the hmdb database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no metabolite annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[metanetx.chemical]\u001b[0m - AssertionError: There are no metabolite annotations for the metanetx.chemical\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[bigg.metabolite]\u001b[0m - AssertionError: There are no metabolite annotations for the bigg.metabolite...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_metabolite_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no metabolite annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[rhea]\u001b[0m - AssertionError: There are no reaction annotations for the rhea database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[kegg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the kegg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[seed.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the seed.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[metanetx.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the metanetx.reaction...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[bigg.reaction]\u001b[0m - AssertionError: There are no reaction annotations for the bigg.reaction dat...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[reactome]\u001b[0m - AssertionError: There are no reaction annotations for the reactome database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[ec-code]\u001b[0m - AssertionError: There are no reaction annotations for the ec-code database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[brenda]\u001b[0m - AssertionError: There are no reaction annotations for the brenda database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_reaction_annotation_wrong_ids[biocyc]\u001b[0m - AssertionError: There are no reaction annotations for the biocyc database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[refseq]\u001b[0m - AssertionError: There are no gene annotations for the refseq database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[uniprot]\u001b[0m - AssertionError: There are no gene annotations for the uniprot database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ecogene]\u001b[0m - AssertionError: There are no gene annotations for the ecogene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[kegg.genes]\u001b[0m - AssertionError: There are no gene annotations for the kegg.genes database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigi]\u001b[0m - AssertionError: There are no gene annotations for the ncbigi database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbigene]\u001b[0m - AssertionError: There are no gene annotations for the ncbigene database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ncbiprotein]\u001b[0m - AssertionError: There are no gene annotations for the ncbiprotein database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[ccds]\u001b[0m - AssertionError: There are no gene annotations for the ccds database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[hprd]\u001b[0m - AssertionError: There are no gene annotations for the hprd database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_annotation.py::\u001b[1mtest_gene_product_annotation_wrong_ids[asap]\u001b[0m - AssertionError: There are no gene annotations for the asap database.\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_metabolites_formula_presence\u001b[0m - AssertionError: There are a total of 796         metabolites (100.00%) with...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_gene_protein_reaction_rule_presence\u001b[0m - AssertionError: There are a total of 225 reactions (16.32%) without GPR:   ...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_ngam_presence\u001b[0m - RuntimeError: It was not possible to identify any metabolite in compartment...\n",
      "\u001b[31mFAILED\u001b[0m ..\\..\\..\\..\\..\\anaconda3\\Lib\\site-packages\\memote\\suite\\tests\\test_basic.py::\u001b[1mtest_find_transport_reactions\u001b[0m - AssertionError: A total of 0 (0.00%) transport reactions are defined in the\n"
     ]
    }
   ],
   "source": [
    "!memote report diff iFF708.xml iIN800.xml iLL672.xml iMM904.xml iND750.xml yeast8.xml"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e6caafb4",
   "metadata": {},
   "source": [
    "| *Saccharomyces cerevisiae* GSM model | Total score(%) | Total reactions | Total metabolites | Total Genes | Metabolic coverage | Consistency score (%) | Annotation-metabolites score  (%) | Annotation-reactions score (%) | \n",
    "|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:| \n",
    "|iFF708| 15 | 1379 | 796 | 619 | 2.23 | 0 | 0 | 0 |\n",
    "|iIN800 | 23 | 1837 | 1824 | 0 | 1.00 | 0 | 100 | 100 |\n",
    "|iLL672 | 15 | 1195 | 671 | 0 | 1.00 | 0 | 0 | 0 |\n",
    "|iMM904 | 31 | 1577 | 1228 | 905 | 1.74 | 100 | 0 | 0 |\n",
    "|iND750 | 37 | 1266 | 1061 | 750 | 1.69 | 100 | 0 | 0 |\n",
    "|yeast8| 62 | 4058 |  2742 | 1161 | 3.50 | 0 | 100 | 100 |"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0e084262",
   "metadata": {},
   "source": [
    "Number of protein genes:        6017\n",
    "Number of RNA genes:             420"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "a5e04af6",
   "metadata": {},
   "outputs": [],
   "source": [
    "from cobra.io import read_sbml_model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "0ef43d4c",
   "metadata": {},
   "outputs": [],
   "source": [
    "model = read_sbml_model('yeast8.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "e63779ac",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>yeastGEM_v8__46__7__46__1</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>196c9410290</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>2806</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>4131</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>1163</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>106</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>1.0*r_2111 - 1.0*r_2111_reverse_58b69</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>cell envelope, cytoplasm, extracellular, mitochondrion, nucleus, peroxisome, endoplasmic reticulum, Golgi, lipid particle, vacuole, endoplasmic reticulum membrane, vacuolar membrane, Golgi membrane, mitochondrial membrane</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model yeastGEM_v8__46__7__46__1 at 0x196c9410290>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "0debcdad",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Metabolite identifier</strong></td><td>s_1566</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Name</strong></td><td>xylitol</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x196c941c1d0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Formula</strong></td><td>C5H12O5</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartment</strong></td><td>c</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>In 4 reaction(s)</strong></td><td>\n",
       "                    r_1092, r_2105, r_4490, r_1093\n",
       "                    </td>\n",
       "            </tr>\n",
       "        </table>"
      ],
      "text/plain": [
       "<Metabolite s_1566 at 0x196c941c1d0>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.metabolites.get_by_id('s_1566')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "7338d401",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'r_1654': 1000.0,\n",
       " 'r_1714': 1.0,\n",
       " 'r_1832': 1000.0,\n",
       " 'r_1861': 1000.0,\n",
       " 'r_1992': 1000.0,\n",
       " 'r_2005': 1000.0,\n",
       " 'r_2020': 1000.0,\n",
       " 'r_2049': 1000.0,\n",
       " 'r_2060': 1000.0,\n",
       " 'r_2100': 1000.0,\n",
       " 'r_4593': 1000.0,\n",
       " 'r_4594': 1000.0,\n",
       " 'r_4595': 1000.0,\n",
       " 'r_4596': 1000.0,\n",
       " 'r_4597': 1000.0,\n",
       " 'r_4600': 1000.0}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.medium"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "8c94b4f1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "r_0005    0.061324\n",
       "r_0006    0.020490\n",
       "r_0007    0.006239\n",
       "r_0013    0.000016\n",
       "r_0014    0.000082\n",
       "            ...   \n",
       "r_4599    0.081928\n",
       "r_4600   -0.000018\n",
       "r_4702   -0.000637\n",
       "r_4703   -0.000637\n",
       "r_4749   -0.000016\n",
       "Name: fluxes, Length: 628, dtype: float64"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "solution = model.optimize()\n",
    "solution.fluxes[solution.fluxes != 0.]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "9522cecd",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "r_1092 r_1092: s_1198 + s_1566 --> s_0580 + s_0794 + s_1203 xylitol dehydrogenase (D-xyulose-forming)\n",
      "r_2105 r_2105: s_1567 <=> s_1566 xylitol transport\n",
      "r_4490 r_4490: s_0794 + s_1212 + s_4180 <=> s_1207 + s_1566 Xylitol:NADP+ 4-oxidoreductase (L-xylulose-forming)\n",
      "r_1093 r_1093: s_0578 + s_0794 + s_1212 --> s_1207 + s_1566 xylose reductase\n"
     ]
    }
   ],
   "source": [
    "for reaction in model.metabolites.s_1566.reactions:\n",
    "    print(reaction.id, reaction, reaction.name)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "r_1093 - xylose reductase\n",
    "r_1719 - D-xylose transport\n",
    "r_1092 - xylitol dehydrogenase \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "c3744488",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.081928158523638\n"
     ]
    }
   ],
   "source": [
    "model.optimize()                               \n",
    "print(model.objective.value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "e44d7422",
   "metadata": {},
   "outputs": [],
   "source": [
    "with model:\n",
    "    model.objective = model.reactions.r_1092  # Assuming XYLR is the reaction for xylitol production\n",
    "    xylitol_production = model.optimize().objective_value\n",
    "    print('Maximum theoretical productivity of xylitol:', xylitol_production, 'mmol/gDW*h')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "acb3ae9d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.081928158523638\n"
     ]
    }
   ],
   "source": [
    "xylose_reductase_reaction = model.reactions.get_by_id('r_1093')\n",
    "\n",
    "# To simulate overexpression, you might increase the upper bound of the reaction.\n",
    "# If you want to remove the upper limit to simulate unlimited enzyme capacity, you can set it to a very high number or to None (which COBRApy interprets as infinity).\n",
    "xylose_reductase_reaction.upper_bound = 1000  # This sets the upper bound to a high number to simulate overexpression\n",
    "\n",
    "# Optionally, if you want to ensure that the reaction only goes in the forward direction, you can set the lower bound to 0.\n",
    "xylose_reductase_reaction.lower_bound = 0  # This ensures the reaction can only go in the forward direction\n",
    "\n",
    "# After modifying the reaction bounds, you can re-optimize the model\n",
    "solution = model.optimize()\n",
    "print(model.objective.value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
